<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.5.57">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>3&nbsp; Evidence for Evolution – Evolution 303</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<link href="./HowNaturalSelectionWorks.html" rel="next">
<link href="./HistoricaPerspective.html" rel="prev">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


</head>

<body class="nav-sidebar floating">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" role="button" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
        <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./IntroBasicConcepts.html">Introduction: Basic Concepts</a></li><li class="breadcrumb-item"><a href="./EvidenceForEvolution.html"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Evidence for Evolution</span></a></li></ol></nav>
        <a class="flex-grow-1" role="navigation" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
        </a>
      <button type="button" class="btn quarto-search-button" aria-label="Search" onclick="window.quartoOpenSearch();">
        <i class="bi bi-search"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal quarto-sidebar-collapse-item sidebar-navigation floating overflow-auto">
    <div class="pt-lg-2 mt-2 text-left sidebar-header">
    <div class="sidebar-title mb-0 py-0">
      <a href="./">Evolution 303</a> 
    </div>
      </div>
        <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
        </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Evolution 303</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./Preface.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Preface - Evolution 303</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="./IntroBasicConcepts.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Introduction: Basic Concepts</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./WhatEvolutionIs.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">What evolution is</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./HistoricaPerspective.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">A brief history of evolutionary thought</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./EvidenceForEvolution.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Evidence for Evolution</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./HowNaturalSelectionWorks.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">How Natural Selection Works</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="./EvolTreesAndPhylogenetics.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Evolutionary Trees and Phylogenetics</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-2" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./UnderstandingEvolTrees.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Understanding Evolutionary Trees</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./EstimatingEvoTrees.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Estimating evolutionary trees</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="./EvolGenetics.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Evolutionary Genetics</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-3" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./GenesMutationGeneticVariation.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Genes, mutation, and genetic variation</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./IntroEvolGenetics.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">Introduction: Evolutionary genetics</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./MechEvolDrift.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">Genetic Drift</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./MechEvolSelection.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">10</span>&nbsp; <span class="chapter-title">Mechanisms that drive evolution: natural selection</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./MechEvolMutation.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">11</span>&nbsp; <span class="chapter-title">Mutation</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./MechEvolMigration.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">12</span>&nbsp; <span class="chapter-title">Migration and other factors</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./EvolPhenotype.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">13</span>&nbsp; <span class="chapter-title">The Evolution of Quantitative Traits</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-4" role="navigation" aria-expanded="true">
 <span class="menu-text">In Light Of Evolution</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-4" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-4" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./KinAndSexualSelection.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">14</span>&nbsp; <span class="chapter-title">Social Behavior and Sexual Selection</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./Adaptations.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">15</span>&nbsp; <span class="chapter-title">Adaptation and Phenotypic Plasticity</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./SpeciesSpeciation.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">16</span>&nbsp; <span class="chapter-title">Species and Speciation</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./EvolutionGenomicLevel.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">17</span>&nbsp; <span class="chapter-title">Evolution at the genomic level</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./EvoDevo.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">18</span>&nbsp; <span class="chapter-title">EvoDevo</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-5" role="navigation" aria-expanded="true">
 <span class="menu-text">Evolution At The Macro Level</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-5" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-5" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./EvoBiogeogrphy.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">19</span>&nbsp; <span class="chapter-title">Evolutionary Biogeography</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./DivLife.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">20</span>&nbsp; <span class="chapter-title">Diversification and patterns of life through time</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-6" role="navigation" aria-expanded="true">
 <span class="menu-text">What About Us</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-6" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-6" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./HumanEvol.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">21</span>&nbsp; <span class="chapter-title">Human Evolution</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./EvolMed.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">22</span>&nbsp; <span class="chapter-title">Evolutionary Medicine</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./EvolMedPartI.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">23</span>&nbsp; <span class="chapter-title">Evolutionary Medicine I: Aging and Diseases of Civilization</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./EvolMedPartII.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">24</span>&nbsp; <span class="chapter-title">Evolutionary Medicine II: Evolving Pathogens</span></span></a>
  </div>
</li>
      </ul>
  </li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" class="quarto-sidebar-collapse-item" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#an-introduction-to-the-evidence" id="toc-an-introduction-to-the-evidence" class="nav-link active" data-scroll-target="#an-introduction-to-the-evidence"><span class="header-section-number">3.1</span> An introduction to the evidence</a></li>
  <li><a href="#the-evidence" id="toc-the-evidence" class="nav-link" data-scroll-target="#the-evidence"><span class="header-section-number">3.2</span> The Evidence</a></li>
  <li><a href="#sec-microevol" id="toc-sec-microevol" class="nav-link" data-scroll-target="#sec-microevol"><span class="header-section-number">3.3</span> Microevolution</a>
  <ul class="collapse">
  <li><a href="#natural-populations" id="toc-natural-populations" class="nav-link" data-scroll-target="#natural-populations"><strong>Natural Populations</strong></a></li>
  <li><a href="#artificial-selection" id="toc-artificial-selection" class="nav-link" data-scroll-target="#artificial-selection"><strong>Artificial selection</strong></a></li>
  <li><a href="#experimental-populations" id="toc-experimental-populations" class="nav-link" data-scroll-target="#experimental-populations"><strong>Experimental Populations</strong></a></li>
  <li><a href="#a-case-study-rapid-change-and-sticklebacks" id="toc-a-case-study-rapid-change-and-sticklebacks" class="nav-link" data-scroll-target="#a-case-study-rapid-change-and-sticklebacks"><strong>A case study: rapid change and sticklebacks</strong></a></li>
  </ul></li>
  <li><a href="#sec-spec" id="toc-sec-spec" class="nav-link" data-scroll-target="#sec-spec"><span class="header-section-number">3.4</span> Speciation</a>
  <ul class="collapse">
  <li><a href="#ring-species" id="toc-ring-species" class="nav-link" data-scroll-target="#ring-species"><strong>Ring Species</strong></a></li>
  <li><a href="#catching-speciation-in-action" id="toc-catching-speciation-in-action" class="nav-link" data-scroll-target="#catching-speciation-in-action"><strong>Catching Speciation in Action</strong></a></li>
  </ul></li>
  <li><a href="#sec-macroevol" id="toc-sec-macroevol" class="nav-link" data-scroll-target="#sec-macroevol"><span class="header-section-number">3.5</span> Macroevolution</a>
  <ul class="collapse">
  <li><a href="#geographic-and-temporal-patterns-of-succession" id="toc-geographic-and-temporal-patterns-of-succession" class="nav-link" data-scroll-target="#geographic-and-temporal-patterns-of-succession"><strong>Geographic and Temporal Patterns of Succession</strong></a></li>
  <li><a href="#transitional-fossils" id="toc-transitional-fossils" class="nav-link" data-scroll-target="#transitional-fossils"><strong>Transitional Fossils</strong></a></li>
  </ul></li>
  <li><a href="#sec-comanc" id="toc-sec-comanc" class="nav-link" data-scroll-target="#sec-comanc"><span class="header-section-number">3.6</span> Common Ancestry and Homology</a>
  <ul class="collapse">
  <li><a href="#shared-flaws" id="toc-shared-flaws" class="nav-link" data-scroll-target="#shared-flaws"><strong>Shared Flaws</strong></a></li>
  <li><a href="#vestigial-organs" id="toc-vestigial-organs" class="nav-link" data-scroll-target="#vestigial-organs"><strong>Vestigial Organs</strong></a></li>
  <li><a href="#molecular-and-genomic-evidence-of-shared-ancestry" id="toc-molecular-and-genomic-evidence-of-shared-ancestry" class="nav-link" data-scroll-target="#molecular-and-genomic-evidence-of-shared-ancestry"><strong>Molecular and genomic evidence of shared ancestry</strong></a></li>
  <li><a href="#why-homologies-matter" id="toc-why-homologies-matter" class="nav-link" data-scroll-target="#why-homologies-matter"><strong>Why Homologies Matter</strong></a></li>
  </ul></li>
  <li><a href="#sec-biogeo" id="toc-sec-biogeo" class="nav-link" data-scroll-target="#sec-biogeo"><span class="header-section-number">3.7</span> Biogeography</a>
  <ul class="collapse">
  <li><a href="#continental-scale" id="toc-continental-scale" class="nav-link" data-scroll-target="#continental-scale"><span class="header-section-number">3.7.1</span> Continental scale</a></li>
  <li><a href="#smaller-scale-microgeographic-patterns-of-similarity-in-island-chains" id="toc-smaller-scale-microgeographic-patterns-of-similarity-in-island-chains" class="nav-link" data-scroll-target="#smaller-scale-microgeographic-patterns-of-similarity-in-island-chains"><span class="header-section-number">3.7.2</span> Smaller scale, microgeographic patterns of similarity in island chains</a></li>
  <li><a href="#summarizing-biogeographic-evidence" id="toc-summarizing-biogeographic-evidence" class="nav-link" data-scroll-target="#summarizing-biogeographic-evidence"><span class="header-section-number">3.7.3</span> Summarizing biogeographic evidence</a></li>
  </ul></li>
  <li><a href="#sec-deeptime" id="toc-sec-deeptime" class="nav-link" data-scroll-target="#sec-deeptime"><span class="header-section-number">3.8</span> Deep Time</a></li>
  <li><a href="#sec-tiktaalik" id="toc-sec-tiktaalik" class="nav-link" data-scroll-target="#sec-tiktaalik"><span class="header-section-number">3.9</span> Correspondence of Different Lines of Evidence</a></li>
  <li><a href="#absence-of-evidence" id="toc-absence-of-evidence" class="nav-link" data-scroll-target="#absence-of-evidence"><span class="header-section-number">3.10</span> Absence of Evidence…</a></li>
  <li><a href="#in-closing" id="toc-in-closing" class="nav-link" data-scroll-target="#in-closing"><span class="header-section-number">3.11</span> In closing</a></li>
  <li><a href="#reflection-questions" id="toc-reflection-questions" class="nav-link" data-scroll-target="#reflection-questions"><span class="header-section-number">3.12</span> Reflection Questions</a></li>
  <li><a href="#references" id="toc-references" class="nav-link" data-scroll-target="#references"><span class="header-section-number">3.13</span> References</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default"><nav class="quarto-page-breadcrumbs quarto-title-breadcrumbs d-none d-lg-block" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./IntroBasicConcepts.html">Introduction: Basic Concepts</a></li><li class="breadcrumb-item"><a href="./EvidenceForEvolution.html"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Evidence for Evolution</span></a></li></ol></nav>
<div class="quarto-title">
<h1 class="title"><span id="sec-evidence" class="quarto-section-identifier"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Evidence for Evolution</span></span></h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<p><em>DRAFT 20250113</em></p>
<p>In the previous chapter we discussed what evolution is, how it is effectively an emergent property of any system that satisfies a set of four criteria (population, variation, heritable similarity, sorting process), that evolution is the only valid scientific explanation that explains the diversity of life on this planet, and that evolution is more than a scientific theory, that it is in fact a paradigm.</p>
<p>So what is this overwhelming evidence for evolution? Why are we so convinced, as scientists, that evolution is indeed a fact, one of the few in the biological sciences? In this chapter we will briefly describe the incredibly varied lines of evidence for evolution. We will not go much into detail into any one piece of evidence, but we will present some specific examples to help make the case. Throughout the rest of the course we will discussing theory, processes, and data that will fill in a lot of the detail. In other words, this entire rest of this course can serve as evidence. And even then, this will not be an exhaustive description of all the evidence for evolution. To attempt to do that would be a Quixotic effort. Nonetheless, in this chapter we will review the main categories of evidence and provide a few favorite examples to help make them clear.</p>
<section id="an-introduction-to-the-evidence" class="level2" data-number="3.1">
<h2 data-number="3.1" class="anchored" data-anchor-id="an-introduction-to-the-evidence"><span class="header-section-number">3.1</span> An introduction to the evidence</h2>
<p>When Darwin’s ideas of evolution and natural selection were first proposed, they were indeed an hypothesis. Now they are considered “facts”, but there was a time when like any good scientific hypothesis, they required testing through the development of testable predictions that can be falsified with hard empirical data. Generally speaking, if we consider Darwin’s idea of descent with modification as a reasonable description (if not a definition) of evolution, then we find that there are five predictions that can be tested through experimentation and hard data:</p>
<ol type="1">
<li>Species change through time (microevolution).</li>
<li>Lineages split to form new species (speciation).</li>
<li>Novel forms are derived from earlier forms (macroevolution).</li>
<li>Species are not independent but connected by descent from a common ancestor (common ancestry and homology).</li>
<li>Earth and life on Earth are old (deep time).</li>
</ol>
<p>Though there may be numerous other predictions that come from Darwin’s hypothesis of common descent, the above list makes for a good starting point for organizing our discussions, a road map so to speak to summarize the evidence demonstrating evolution as the explanation for the diversification of life on earth.</p>
<p>But before we do this, we need to discuss a very important distinction of types of evidence, and that is a prediction versus a retrodiction.</p>
<blockquote class="blockquote">
<p>• <strong>Prediction</strong> - if evolution is correct, then we could predict/anticipate some observed outcome or pattern. This is fundamentally consistent with what we consider to be a prediction in say a basic laboratory experiment, <em>e.g.</em> in chemistry if I mix chemical A with B the outcome will be C. In biology things get little more complicated, actually alot more complicated (which is what makes biolgy so interesting in my opinion), and can not always be tested in a laboratory setting. Nonetheless, we can use observations in nature to test predictions. For example, if common ancestry is correct, then we should find that species that have split more recently from each other should be more genetically similar to each other than they are to species that they are more distantly related to. So for example, we should find that humans and chimpanzees tend to be more genetically similar to each other than either are to say whales. A simple straightforward prediction derived from common ancestry.</p>
</blockquote>
<blockquote class="blockquote">
<p>• <strong>Retrodiction</strong> - This is strange sounding term that basically says that the observation may not necessarily be directly predicted by evolution, but that explaining the observed patterns make most sense in light of evolution. Some examples we will see below will be more retrodictions than based on predictions, for example biogeographic distributions, developmental patterns (comparative development), and vestigial structures, but nonetheless provide compelling evidence for evolution.</p>
</blockquote>
<p>Before continuing with the evidence, I would like to point you to an excellent book by Jerry Coyne, one of the leading figures in modern evolutionary biology, entitled “<a href="https://www.amazon.com/Why-Evolution-True-Jerry-Coyne/dp/0143116649">Why Evolution Is True</a>”. It is definitely worth the read. The only reason I do not assign it for my course is that we simply do not have the time to go over his book in the detail it deserves and get to the rest of what we need to discuss in one semester. Give it a look though.</p>
<p>This being our modern world, there are also a number of YouTube videos where Dr.&nbsp;Coyne presents his take on the evidence for evolution. The following is only one of many, but deserves a careful listen (HINT: start the video at the 3:35 mark, and feel free to stop it at the 51:05 mark, where he transitions into religion and evolution).</p>
<div class="quarto-video ratio ratio-16x9"><iframe data-external="1" src="https://www.youtube.com/embed/eEVXzeU3BIA" title="Why Evolution is True (2012 Jerry Coyne Lecture)" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe></div>
</section>
<section id="the-evidence" class="level2" data-number="3.2">
<h2 data-number="3.2" class="anchored" data-anchor-id="the-evidence"><span class="header-section-number">3.2</span> The Evidence</h2>
<div class="callout callout-style-default callout-important callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Categories of evidence for evolution
</div>
</div>
<div class="callout-body-container callout-body">
<ul>
<li><p><a href="#sec-microevol" class="quarto-xref"><span>Section 3.3</span></a></p></li>
<li><p><a href="#sec-spec" class="quarto-xref"><span>Section 3.4</span></a></p></li>
<li><p><a href="#sec-macroevol" class="quarto-xref"><span>Section 3.5</span></a></p></li>
<li><p><a href="#sec-comanc" class="quarto-xref"><span>Section 3.6</span></a></p></li>
<li><p><a href="#sec-biogeo" class="quarto-xref"><span>Section 3.7</span></a></p></li>
<li><p><a href="#sec-deeptime" class="quarto-xref"><span>Section 3.8</span></a></p></li>
<li><p><span class="quarto-unresolved-ref">?sec-otherevidence</span></p></li>
</ul>
</div>
</div>
</section>
<section id="sec-microevol" class="level2" data-number="3.3">
<h2 data-number="3.3" class="anchored" data-anchor-id="sec-microevol"><span class="header-section-number">3.3</span> Microevolution</h2>
<p>Microevolution is the change in inherited traits of a population from one generation to the next, ultimately leading to the accumulation of changes and the transformation of species through time. <em>Heritable trait</em> in this context can refer to any phenotypic trait (for example the average beak size in a population of a bird) or a molecular trait (<em>e.g.</em> the frequency of alternative alleles at a particular locus) or genomic variant (<em>e.g.</em> a copy number variant or an inversion polymorphism). While changes in most traits from one generation to the next are subtle at best, strong natural selection can lead to significant and detectable evolutionary changes in very short periods of time.</p>
<p>One way to think about microevolution is “observing change over time of a heritable trait”. This is effectively the definition of evolution, so any examples of observed change over time is by definition observing evolution.</p>
<p>A convenient way of discussing the various evidence out there (and there is a near infinite number of them) is to sort them into categories such as the following:</p>
<section id="natural-populations" class="level3 unnumbered">
<h3 class="unnumbered anchored" data-anchor-id="natural-populations"><strong>Natural Populations</strong></h3>
<p>Observing changes in natural populations directly or at least over historical time is powerful evidence for evolution: it is anagenesis in action. Over the decades there has been an incredible number of examples of populations changing over time published, and the number continues to increase every year. There are simply too many to cover in any detail here, so below is a very short list of some of the classic examples:</p>
<ul>
<li>industrial melanism in the peppered moth <em>Biston betularia</em> (<span class="citation" data-cites="cook2012">Cook and Saccheri (<a href="#ref-cook2012" role="doc-biblioref">2012</a>)</span>); recent studies has demonstrated the genetic basis of the melanic form (<span class="citation" data-cites="hof2016">Hof et al. (<a href="#ref-hof2016" role="doc-biblioref">2016</a>)</span>; more recently <span class="citation" data-cites="tian2024">Tian et al. (<a href="#ref-tian2024" role="doc-biblioref">2024</a>)</span>)</li>
<li>the soapberry bug in Florida, which has shown changes in beak size over approximately 100 years, in response to changes in its preferred fruit size (<span class="citation" data-cites="carroll2005">Carroll et al. (<a href="#ref-carroll2005" role="doc-biblioref">2005</a>)</span>).</li>
<li>the evolution of house sparrows in North America, which have exhibited changes in size and color over roughly 100 generations (<span class="citation" data-cites="johnston1964">Johnston and Selander (<a href="#ref-johnston1964" role="doc-biblioref">1964</a>)</span>).</li>
<li>natural populations of the three spine stickleback <em>Gasterosteus aculeatus.</em> (see below)</li>
</ul>
<p><strong><em>Human-Driven Change:</em></strong> Resistance to drugs, insecticides, and herbicides offers a clear example of evolution in action. HIV’s resistance to treatments like AZT, and the shrinking size of fish due to commercial fishing pressures, are notable examples, as are changes that occur in invasive species when they colonize a new environment (<span class="citation" data-cites="palumbi2001">Palumbi (<a href="#ref-palumbi2001" role="doc-biblioref">2001</a>)</span>).</p>
<p><strong><em>Change in the fossil record:</em></strong> Though much of the evidence for observed change over time comes from extant populations or populations that we can at least follow and observe over historical time, we also see these types of observed changes in the fossil record. Collecting this kind of data can be very work intensive, yet we still have a number of excellent examples. One of my favorites is an outstanding study on changes in a fossil assemblage of the sticklebacks from Bell and colleagues (<span class="citation" data-cites="bell2009">Bell (<a href="#ref-bell2009" role="doc-biblioref">2009</a>)</span>).</p>
<div>

</div>
<div class="quarto-layout-panel" data-layout="[[40,60], [100]]">
<div class="quarto-layout-row">
<div class="quarto-layout-cell" style="flex-basis: 40.0%;justify-content: center;">
<p><img src="images/SticklebackFossil.png" class="img-fluid"></p>
</div>
<div class="quarto-layout-cell" style="flex-basis: 60.0%;justify-content: center;">
<p><img src="images/SticklebackPelvicChange.png" class="img-fluid"></p>
</div>
</div>
<div class="quarto-layout-row">
<div class="quarto-layout-cell" style="flex-basis: 100.0%;justify-content: flex-start;">
<p>Fig A is a an example of a fossil stickleback. Fig B shows the change in the mean pelvic phenotype over time.</p>
</div>
</div>
</div>
</section>
<section id="artificial-selection" class="level3 unnumbered">
<h3 class="unnumbered anchored" data-anchor-id="artificial-selection"><strong>Artificial selection</strong></h3>
<p>Artificial selection is effectively the same as natural selection only we become the selecting agent. We decide who gets to survive and reproduce, with potentially profound impacts on the diversity of these organisms. What are talking about here is our impact on the evolution of our domestic crops, livestock, and pets, for which there are a vast array of examples in the literature.</p>
<div>

</div>
<div class="quarto-layout-panel" data-layout="[[40,60], [100]]">
<div class="quarto-layout-row">
<div class="quarto-layout-cell" style="flex-basis: 40.0%;justify-content: center;">
<p><img src="images/brassica_artificial_selection.jpg" class="img-fluid"></p>
</div>
<div class="quarto-layout-cell" style="flex-basis: 60.0%;justify-content: center;">
<p><img src="images/dog.jpg" class="img-fluid"></p>
</div>
</div>
<div class="quarto-layout-row">
<div class="quarto-layout-cell" style="flex-basis: 100.0%;justify-content: flex-start;">
<p>Images from <a href="https://evolution.berkeley.edu/lines-of-evidence/artificial-selection/?t">Understanding Evolution</a>.</p>
</div>
</div>
</div>
<p>In fact, artificial selection (in particular breeds of dogs and pigeons) profoundly impacted Darwin’s thinking, using artificial selection as an analogy to natural selection and the extensive variation that existed in populations. In some cases artificial selection acting over generations can result in reproductive isolation, which can be the precursor to the formation of new species.</p>
</section>
<section id="experimental-populations" class="level3 unnumbered">
<h3 class="unnumbered anchored" data-anchor-id="experimental-populations"><strong>Experimental Populations</strong></h3>
<p>Experiments like Richard Lenski’s Long-Term Evolution Experiment (LTEE) with <em>E. coli</em> (<a href="https://www.discovermagazine.com/planet-earth/how-a-30-year-experiment-has-fundamentally-changed-our-view-of-how?t">Discover website</a>) and laboratory studies with <em>Drosophila</em> (<em>e.g.</em> <span class="citation" data-cites="burke2009">Burke and Rose (<a href="#ref-burke2009" role="doc-biblioref">2009</a>)</span>), plus numerous others, have clearly demonstrated that we can demonstrate and study evolution in the laboratory with controlled studies.</p>
<p>For a particulary compelling visual example of experimental evolution in a labortory population of <em>E. coli</em>, check out the following video produced by the <a href="https://kishony.technion.ac.il/">Kishony Lab at Harvard Medical School</a>. They have designed a simple way to observe how bacteria evolve as they encounter increasingly higher doses of an antibiotic and adapt to survive—and thrive—despite of it.</p>
<div class="quarto-video ratio ratio-16x9"><iframe data-external="1" src="https://www.youtube.com/embed/plVk4NVIUh8" title="Why Evolution is True (2012 Jerry Coyne Lecture)" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe></div>
<p>You might say that bacteria are different. After all, assuming a generation time of 30 minutes, the two-week experiment described in the video represents over 670 generations of bacterial evolution. Translated to humans, that would represent about 17,000 years. Looking back that far in history, that was a time when humans exclusively lived as hunter-gatherers and just started to migrate into North America over the Bering Land Bridge…</p>
</section>
<section id="a-case-study-rapid-change-and-sticklebacks" class="level3 unnumbered">
<h3 class="unnumbered anchored" data-anchor-id="a-case-study-rapid-change-and-sticklebacks"><strong>A case study: rapid change and sticklebacks</strong></h3>
<p>One of the most persistent misconceptions about evolution is that it takes millions of years to occur. However, the reality is that microevolution—in principle—can happen in as little as one generation. Those short-term changes can be very hard to detect, because our measurement error of a trait of interest is often larger than the actual per-generation evolutionary change. Nonetheless, over the course of just a handful of generations, natural populations may exhibit significant evolutionary change that we can detect with high confidence using genetic markers (<em>i.e.</em>, measuring changes in allele frequencies) or phenotypic measurements.</p>
<p>The convergence of ecological and evolutionary timescales is a relatively recent insight. Darwin did not think that we would be able to directly observe evolutionary change over short periods of time:</p>
<blockquote class="blockquote">
<p>“We see nothing of these slow changes in progress, until the hand of time has marked the lapse of ages.”</p>
<p>— Darwin, 1859</p>
</blockquote>
<p>However, with technological breakthroughs that improved the precision of measurements we take in natural populations and with scientists’ ability to track populations continuously through time, we have accumulated data across dozens of study systems—from microbes to vertebrates—documenting microevolutionary change within a few to a few dozen generations (see <span class="citation" data-cites="hairston2005">Hairston et al. (<a href="#ref-hairston2005" role="doc-biblioref">2005</a>)</span>; <span class="citation" data-cites="carroll2007">CARROLL et al. (<a href="#ref-carroll2007" role="doc-biblioref">2007</a>)</span>). Here, Professor Tobler will briefly introduce you to evidence for rapid evolution gathered in one such study system (the threespine stickleback). In this chapter’s case study, you will explore another example based on a time series of beak size variation of a species of Darwin’s finch.</p>
<section id="the-case-of-threespine-stickleback" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="the-case-of-threespine-stickleback"><strong><em>The Case of Threespine Stickleback</em></strong></h4>
<p>Threespine stickleback (<em>Gasterosteus aculeatus</em>; <a href="#fig-stickleback" class="quarto-xref">Figure&nbsp;<span>3.1</span></a>)) are a widely used system to study evolution and have been shown to rapidly adapt to novel environmental conditions. Stickleback are primarily marine and inhabit coastal waters throughout much of the Northern hemisphere. They are small fish (usually less than 8 cm in length) that exhibit exquisite adaptations to avoid predation in their environment: the sides of their body are covered in bony plates, and they have spines associated with their dorsal and pelvic fins that, when spread out, can dissuade a predator from capturing or consuming them.</p>
<div id="fig-stickleback" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-stickleback-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="images/Three-spined_stickleback.jpg" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-stickleback-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.1: Threespine stickleback (<em>Gasterosteus aculeatus</em>). Photo by Gilles San Martin, <a href="https://creativecommons.org/licenses/by-sa/2.0">CC BY-SA 2.0</a>, via Wikimedia Commons.
</figcaption>
</figure>
</div>
<p>Since the last ice age, as glaciers retreated and left behind a plethora of new streams and lakes, stickleback have also colonized freshwater habitats, which differ in many ways from the original marine habitats. Freshwaters not only exhibit a different water chemistry, but they also tend to harbor fewer predators and different food resources. Over the past 10,000-20,000 years, stickleback in freshwater environments have evolved a number of phenotypic differences compared to their marine ancestors, including a drastic reduction of the armor plates along the body and—in some instances—a loss of the pelvic spines (Jones et al.&nbsp;2012). Moreover, stickleback have also adapted to different niches within freshwaters, and there are distinct morphs in streams and lakes, and in benthic and pelagic habitats within lakes (Hendry et al.&nbsp;2013). Different freshwater ecotypes exhibit distinct body shapes and colorations and are adapted to consuming different types of prey items.</p>
<p>So, how long might it take for the evolution of the traits that vary so drastically across different stickleback forms? Sure, 20,000 years is a blink of an eye in the history of life on the planet, but it is still an eternity for any researcher that might want to observe evolution in action.</p>
<p>One hint at how fast stickleback might evolve comes from a fascinating natural experiment. In 1964, the Great Alaska Earthquake brought widespread destruction to the region and literally reshaped the regional topology. For example, multiple islands in the Prince William Sound and the Gulf of Alaska were lifted up further out of the ocean, creating new freshwater ponds where previously were none. In the time since the earthquake, stickleback have colonized these new freshwater ponds, and within just 50 years, they have evolved similar phenotypic traits that we know from stickleback in continental freshwaters (Lescak et al.&nbsp;2015). Hence, adaptation to freshwaters upon colonization from the ocean may occur in a matter of a few decades rather than gradually over thousands of years of evolution.</p>
<p>To get a better understanding of just how fast evolution may proceed, researchers from the University of Basel in Switzerland decided to conduct a field experiment using lake and stream stickleback (Laurentino et al.&nbsp;2020). The researchers first sequenced the genomes of lake and stream stickleback to detect the genomic regions that are differentiated between ecotypes and likely contain the genes involved in shaping the phenotypic differences between them. After that, they generated F2 crosses between the ecotypes, which is like shuffling a deck of cards from a genomic perspective: individual F2 offspring essentially exhibit a random mixture of genomic segments from their lake and stream ancestors. If these F2 hybrids were introduced into a stream environment, it is predicted that individuals that exhibit stream alleles in regions of the genome important for the expression of stream-specific phenotypic traits perform better than individuals with lake alleles. At a population level, this should lead to an increase in the frequency of alleles characteristic for stream stickleback. And, this is exactly what happened when the researchers actually conducted the experiment. More importantly, the predicted genetic changes were detectable within just one generation of F2 individuals being released into a stream habitat. So, when selection is strong and researchers have the capability to measure changes in traits with adequate precision, we can actually detect the small, generation-to-generation changes that ultimately accumulate to give rise to more conspicuous evolutionary changes that are easier to detect.</p>
</section>
</section>
</section>
<section id="sec-spec" class="level2" data-number="3.4">
<h2 data-number="3.4" class="anchored" data-anchor-id="sec-spec"><span class="header-section-number">3.4</span> Speciation</h2>
<p>Speciation is the process by which new species arise. Before we dive into how speciation actually works, we should agree on what species actually are:</p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Note
</div>
</div>
<div class="callout-body-container callout-body">
<p><b>Definition: Species</b></p>
<p>A <em>biological species</em> is a group of organisms that can reproduce with one another in nature and produce fertile offspring. Species are characterized by the fact that they are reproductively isolated from other such groups, which means that the organisms in one species are incapable of reproducing with organisms in another species. (As you may know, there are alternative definitions of a species, which we will discuss in more detail in <a href="speciation-1.html">Chapter 11</a>.)</p>
</div>
</div>
<p>In phylogenetic trees, speciation is depicted as a singular point that represents the moment one lineage splits into two (red circle in <a href="#fig-speccont" class="quarto-xref">Figure&nbsp;<span>3.2</span></a>. Although speciation can occur instantaneously, for example when polyploidization is involved (see <a href="speciation-1.html">Chapter 11</a>), new species typically evolve gradually, from a single variable population, to populations within a species that are differentiated but still connected through gene flow, to distinct species that are completely isolated from each other (<a href="#fig-speccont" class="quarto-xref">Figure&nbsp;<span>3.2</span></a>). Movement along this “speciation continuum” is driven by the accumulation of reproductive barriers that prevent individuals from mating or successfully producing offspring with each other. Importantly, movement along the speciation continuum can be bidirectional, and reproductive barriers can disappear such that two species merge back together into one. If speciation is a gradual process, we should be able to observe all stages along the speciation continuum in nature, not just the endpoints of the speciation process with reproductively isolated species.</p>
<div id="fig-speccont" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-speccont-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="images/speciation_continuum.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-speccont-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.2: Speciation is not typically an instantaneous process. Rather species evolve gradually along a speciation continuum.
</figcaption>
</figure>
</div>
<section id="ring-species" class="level3 unnumbered">
<h3 class="unnumbered anchored" data-anchor-id="ring-species"><strong>Ring Species</strong></h3>
<p>One phenomenon that perfectly illustrates the speciation continuum and provides evidence for ongoing speciation are so-called ring species, in which two reproductively isolated populations living sympatrically (red and brown in <a href="#fig-ringspecies" class="quarto-xref">Figure&nbsp;<span>3.3</span></a> are connected by a geographic ring of populations that can interbreed. Such ring species arise when an original population disperses around a geographic barrier, and populations diverge gradually, for example as a consequence of adaptation to local environmental conditions. Once populations come into sympatry again behind the geographic barrier, sufficient differences have accumulated such that populations cannot interbreed with each other anymore.</p>
<div id="fig-ringspecies" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-ringspecies-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="images/rings-species.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-ringspecies-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.3: A schematic representing a ring species. Individuals are able to successfully reproduce with members of adjacent populations, as indicated by the black arrows. However, as populations disperse around a geographic barrier and diverge gradually, they are unable to reproduce when they come into contact again. This process represents a form of speciation occurring with gene flow.
</figcaption>
</figure>
</div>
<p>Several well-studied examples of ring species exist. For example, plethodontid salamanders of the <em>Ensatina eschscholtzii</em> complex have colonized different parts of California from the north and expanded southward around the Central Valley, which represents unsuitable habitat for salamanders. At the southern tip of the Central valley, salamander populations from the eastern and western mountain ridges that surround the valley came into secondary contact and are unable to interbreed due to the genetic changes that have accumulated during evolution in isolation (Pereira et al.&nbsp;2011). Other examples of ring species include the herring and lesser black-backed gulls (genus <em>Larus</em>) that have a circumpolar distribution and cannot interbreed in northern Europe. In addition, greenish warblers (<em>Phylloscopus trochiloides</em>) form a ring species around the Himalayas (Irwin et al.&nbsp;2005).</p>
</section>
<section id="catching-speciation-in-action" class="level3 unnumbered">
<h3 class="unnumbered anchored" data-anchor-id="catching-speciation-in-action"><strong>Catching Speciation in Action</strong></h3>
<p>Evidence for ongoing speciation also comes from a wide variety of study systems that do not occur in a ring. Especially when populations are subject to strong natural selection (for example if they inhabit different habitat types), we do not only observe adaptive differentiation across populations but also the inadvertent emergence of reproductive isolation. As populations acquire traits that make them better suited to the environments they inhabit, individuals also tend to stop interbreeding with individuals from different populations that have different traits. This is called ecological speciation, and we explore this concept in more detail in <a href="speciation-1.html">Chapter 11</a>. As a consequence, we can often find populations along various stages of the speciation continuum.</p>
<p><em>Timema</em> stick insects are a great example. These insects are distributed in the western United States, and different species of <em>Timema</em> have adapted to live and feed on different host plants. While most species are uniformly green, gray, or brown and live on broad-leaved host plants, several species have independently evolved a dorsal stripe that provides camouflage on needle-like leaves, providing protection from predators <a href="#fig-timema" class="quarto-xref">Figure&nbsp;<span>3.4</span></a>. In at least one species, <em>T. cristinae</em>, both uniformly colored and striped populations exist depending on whether they live on host plants with broad or needle-like leaves. While these phenotypic differences within <em>T. cristinae</em> parallel differences between other <em>Timema</em> species, reproductive isolation between different <em>T. cristinae</em> populations utilizing alternative host plants is still incomplete (Nosil 2007). Hence, the different ecotypes represent intermediate stages of speciation. Similar variation along the speciation continuum has been uncovered in a wide variety of other natural study systems, and we will get to know some of them in more detail later in the book.</p>
<div id="fig-timema" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-timema-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="images/timema.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-timema-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.4: <em>Timema</em> stick insects have adapted to different host plants. Left: <em>T. cristinae</em> on one of its hosts, Greenbark (<em>Ceanothus spinosus</em>). Photo by <a href="https://www.flickr.com/photos/90276319@N07/14165790793">Aaron C</a>, <a href="https://creativecommons.org/publicdomain/mark/1.0/">public domain</a>. Right: <em>T. poppensis</em> on its host, Redwood (<em>Sequoia sempervirens</em>). Photo by <a href="https://www.flickr.com/photos/bmc_ecology/8592831488">Moritz Muschick</a>, <a href="https://creativecommons.org/licenses/by/2.0">CC BY 2.0</a>.
</figcaption>
</figure>
</div>
<p>A more recent and very impressive study on the order Phasmatodea, which includes stick and leaf insects (<span class="citation" data-cites="boisseau2024">Boisseau, Bradler, and Emlen (<a href="#ref-boisseau2024" role="doc-biblioref">2024</a>)</span>), has demonstrated that similar environmental conditions experienced by various species plus the time since divergence can predict the level of morphological convergence, demonstrating that evolution can be surprisingly predictable even when lineages have been evolving independently for tens of millions of years.</p>
</section>
</section>
<section id="sec-macroevol" class="level2" data-number="3.5">
<h2 data-number="3.5" class="anchored" data-anchor-id="sec-macroevol"><span class="header-section-number">3.5</span> Macroevolution</h2>
<p>When discussing evidence for microevolution and speciation, we primarily turned to evidence from observations of living forms and from experiments. Here, we will consider a different type of evidence, fossils, which are simply the remains of prehistoric organisms. Fossils are usually petrified or preserved as a mold in rock, although the rapid thawing of permafrost accelerated by climate change has also revealed frozen fossils with amazing preservation of soft tissues (for example, check out <a href="https://www.smithsonianmag.com/history/well-preserved-30000-year-old-baby-woolly-mammoth-emerges-from-yukon-permafrost-180980388/">this story</a> about the recent discovery of nun cho ga, a baby woolly mammoth). The mere fact that fossils exist and often represent forms that are distinct from any organisms alive today (<a href="#fig-mammoth" class="quarto-xref">Figure&nbsp;<span>3.5</span></a>) is testament to the ever-changing faunas and floras that have inhabited Earth through time. The fossil record also teaches us that extinction is an important aspect of evolutionary change, just as the generation of novel forms.</p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Note
</div>
</div>
<div class="callout-body-container callout-body">
<p><b>Explore More</b></p>
<p>If you are interested in learning more about what fossils teach us about evolution, I highly recommend Donald R. Prothero’s book “<a href="https://cup.columbia.edu/book/evolution/9780231139625">Evolution: What the Fossils Say and Why It Matters</a>”.</p>
</div>
</div>
<div id="fig-mammoth" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mammoth-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="images/MammothVsMastodon.jpg" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-mammoth-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.5: A woolly mammoth (left) and an American mastodon (right) facing each other, showing the physical differences between the two extinct animals. Illustration by <a href="https://en.wikipedia.org/wiki/User:Dantheman9758">Dantheman9758</a>, <a href="https://creativecommons.org/licenses/by-sa/3.0/deed.en">CC BY-SA 3.0</a>.
</figcaption>
</figure>
</div>
<section id="geographic-and-temporal-patterns-of-succession" class="level3 unnumbered">
<h3 class="unnumbered anchored" data-anchor-id="geographic-and-temporal-patterns-of-succession"><strong>Geographic and Temporal Patterns of Succession</strong></h3>
<p>Perhaps the strongest evidence fossils provide for Darwin’s notion of evolution is that they are not randomly distributed, neither in terms of geography nor time. The constant cycle of descent with modification and extinction has created predictable patterns in fossil deposits across the planet, which is evident as a succession of novel forms that are derived from earlier ones.</p>
<p>Patterns of succession are evident geographically, because there tends to be a regional correspondence between fossils and living forms. That is, we tend to find fossil relatives of extant species in the same areas where extant species live today. For example, marsupial fossils are particularly common in Australia just like living marsupials today. This is also reflected in the biogeographic distribution of extant forms, which frequently tracks the changing landmasses driven by continental drift. Marsupials are not only found in Australia but also South America, which were once connected through Antarctica in a large landmass called Gondwana until about 140 million years ago. The non-random distribution of fossils means that we can develop hypotheses and test predictions about the fossil record. For example, if the biogeographic distribution of marsupials was the consequence of a once contiguous distribution on Gondwana, we would predict the presence of marsupial fossils on Antarctica, which have indeed been found (Woodburne &amp; Zinsmeister 1982).</p>
<p>Patterns of succession are also evident temporally, with fossils exhibiting more ancestral traits being found in older layers of rock compared to derived forms. Subsequently deposited rock layers can therefore shed light into the temporal dynamics of trait evolution and reveal entire time series that connect forms with disparate phenotypes. For example, we know that horses with their single hoof descended from multi-toed ancestors, because we have a time series of horse fossils that illustrates the toe reductions through time (<a href="#fig-horse" class="quarto-xref">Figure&nbsp;<span>3.6</span></a>).</p>
<div id="fig-horse" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-horse-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="images/Equine_evolution.jpg" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-horse-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.6: Equine evolution, composed from skeletons of the State Museum for Natural History Karlsruhe, Germany. From left to right: Size development, biometrical changes in the cranium, reduction of toes on the left forefoot. Image by H. Zell, <a href="https://creativecommons.org/licenses/by-sa/3.0">CC BY-SA 3.0</a>, via Wikimedia Commons.
</figcaption>
</figure>
</div>
</section>
<section id="transitional-fossils" class="level3 unnumbered">
<h3 class="unnumbered anchored" data-anchor-id="transitional-fossils"><strong>Transitional Fossils</strong></h3>
<p>If novel forms are indeed descendants from earlier forms, the fossil record should capture evidence of what Darwin called the “transmutation of species”. Hence, we should be able to find transitional fossils that exhibit traits common to both an ancestral group and its derived descendants. The first, and perhaps still one of the most spectacular, transitional fossil ever found was <em>Archaeopterix</em> (<a href="#fig-archaeopteryx" class="quarto-xref">Figure&nbsp;<span>3.7</span></a>), which shared traits with Mesozoic dinosaurs and modern birds. Its jaws contained sharp teeth, and it has wings with fingers and claws, a long bony tail, hyperextensible second toes (like <em>Velociraptor</em>’s), and feathers. Hence, <em>Archaeopterix</em> represents a transitional fossil between modern birds and non-avian dinosaurs (Ostrom 1976). Similar transitional fossils have been discovered to link the transition from aquatic to terrestrial vertebrates (<em>Tiktaalik</em> <a href="#sec-tiktaalik" class="quarto-xref"><span>Section 3.9</span></a>), from marine mammals to their terrestrial ancestors (<em>Pakicetus</em>, <em>Ambulocetus</em>, and <em>Remingtonocetus</em>), and from quadrupedal to bipedal hominids (<a href="human-origins-and-human-mediated-evolution.html"><em>Australopithecus afarensis</em></a>), among many others.</p>
<div id="fig-archaeopteryx" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-archaeopteryx-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="images/Archaeopteryx_lithographica.jpg" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig quarto-uncaptioned" id="fig-archaeopteryx-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.7
</figcaption>
</figure>
</div>
<p>An extremely important aspect of transitional forms is where they are found in the fossil record. If evolution is correct, then transitional forms should always be found in geologic strata that dates between the ancestral and derived fossil species. If we were to find a transitional form that either predates the ancestral fossils or post-dates the derived fossil species, that would raise some serious questions about the nature of the transitional form.</p>
<p>It is not just the presence of transitional forms, but where in the fossil record they are found. To date, there has yet to be a single reported example in the literature where a transitional form is found inconsistent with our expectations. This is exceptionally solid evidence for evolution.</p>
</section>
</section>
<section id="sec-comanc" class="level2" data-number="3.6">
<h2 data-number="3.6" class="anchored" data-anchor-id="sec-comanc"><span class="header-section-number">3.6</span> Common Ancestry and Homology</h2>
<p>A key prediction of Darwin’s notion of evolution is that species are not independent but connected by descent from a common ancestor. Phylogenetic trees (Figure @ref(fig:ithink)) are representations of that connectedness, and if we were to travel back in time toward the root of the phylogenetic tree, we would expect lineages to merge into the origin of life, the original being from which all living forms descended (also known as the last universal common ancestor, or LUCA). In the absence of time travel, the critical evidence for common ancestry of all life is homology.</p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Note
</div>
</div>
<div class="callout-body-container callout-body">
<p><b>Definition: Homology</b></p>
<p>Homology is the similarity of the structure, physiology, or development of different species based upon their descent from a common evolutionary ancestor.</p>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/I_think.png" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption>The original phylogenetic trees Darwin used to illustrate common ancestry. Left: The Tree of Life image that appeared in Darwin’s <em>On the Origin of Species by Natural Selection</em> (1859). It was the book’s only illustration. Right: Charles Darwin’s original 1837 sketch, his first diagram of an evolutionary tree from his <em>First Notebook on Transmutation of Species</em> (1837). Illustrations by Charles Darwin, <a href="https://creativecommons.org/publicdomain/zero/1.0/">Public Domain</a>.</figcaption>
</figure>
</div>
</div>
</div>
<p>Homology explains why all forms of life share certain characteristics. All forms of life, from microbes to plants and animals, share the same molecular building blocks: lipids that form the boundaries of cells and organelles, nucleic acids that encode information, proteins that play both structural and catalytic roles, and glycans that serve structure, energy storage, and regulatory purposes (Marth 2008). Different life forms, even those separated by billions of years of evolution, share these building blocks because they inherited them from a common ancestor.</p>
<p>Homologies also occur at a more narrow scope. For example, Darwin was puzzled by the structural similarity in the forelimbs of terrestrial vertebrates even though they serve entirely different functions (<a href="#fig-homology" class="quarto-xref">Figure&nbsp;<span>3.8</span></a>):</p>
<blockquote class="blockquote">
<p>What could be more curious than that the hand of a man, formed for grasping, that of a mole for digging, the leg of a horse, the paddle of the porpoise, and the wing of a bat, should all be constructed on the same patterns, and should include the same bones, in the same relative position?</p>
<p>— Darwin, 1859</p>
</blockquote>
<p>How do these similarities in structure arise? Again, it is because all terrestrial vertebrates inherited this shared limb structure from their common ancestor. Rather than “inventing” different types of forelimbs for different purposes (grabbing, running, flying, swimming), evolution has gradually modified and repurposed existing forelimb structures for new functions.</p>
<div id="fig-homology" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-homology-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="images/homology.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig quarto-uncaptioned" id="fig-homology-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.8
</figcaption>
</figure>
</div>
<p>Homologies occur in nested sets. Closely related species exhibit a higher number of homologies, because they inherited those traits from a shared ancestor. More distantly related taxa exhibit differences in their traits, because they have been on independent evolutionary trajectories for prolonged periods of time. Accordingly, analyses of homologous structures are used to infer phylogenetic relationships among taxa. We group species that share a lot of homologous traits closely together on a phylogenetic tree, while those that share few are further apart (see <a href="molecular-evolution.html">Chapter 7</a> for more information).</p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Note
</div>
</div>
<div class="callout-body-container callout-body">
<p><img src="images/important.png" class="img-fluid" width="20"> <strong>Important Note</strong></p>
<p>Not all trait similarities are the consequence of homology. <strong>Analogy</strong> in biology describes similarity of function and superficial resemblance of structures that have different origins. For example, the wings of a fly, a moth, and a bird are analogous, because they evolved independently as adaptations to a common function (flying). Analogies are a consequence of convergent evolution, where unrelated lineages evolve similar traits, typically as adaptations to similar lifestyles or environmental conditions.</p>
</div>
</div>
<section id="shared-flaws" class="level3 unnumbered">
<h3 class="unnumbered anchored" data-anchor-id="shared-flaws"><strong>Shared Flaws</strong></h3>
<p>Perhaps the most compelling evidence for common ancestry comes from homologous structures that serve no purpose at all. The origin of these structures cannot be explained by adaptive evolution, and the only reason that some organisms exhibit functionless structures is because they have inherited them from an ancestor. I want to briefly introduce two such cases, vestigial organs and pseudogenes.</p>
</section>
<section id="vestigial-organs" class="level3 unnumbered">
<h3 class="unnumbered anchored" data-anchor-id="vestigial-organs"><strong>Vestigial Organs</strong></h3>
<p>Vestigial organs are rudimentary traits that lost some or all of the ancestral functions of the structure. Classic examples of vestigial organs in humans are the appendix (vestigial caecum), the coccyx (vestigial tail), and some muscles connected to the ear, which allow for ear mobility in other primates. Structural vestigial organs in other animals include remnants of limbs that are still expressed in some whales, snakes, and flightless birds (<a href="#fig-vestigial" class="quarto-xref">Figure&nbsp;<span>3.9</span></a>).</p>
<div id="fig-vestigial" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-vestigial-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="images/vestigial.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-vestigial-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.9: Vestigial limbs are common in tetrapods. A. Skeleton of a baleen whale showing the vestigial hindlegs (structure c). Illustration from Meyers Konversionlexikon, <a href="https://creativecommons.org/publicdomain/zero/1.0/">Public Domain</a>. B. Vestigial hindlegs (spurs) in a <em>Boa constrictor</em>. Photo by Stefan3345, <a href="https://creativecommons.org/licenses/by-sa/4.0">CC BY-SA 4.0</a>. C. Little spotted kiwi (<em>Apteryx owenii</em>) have vestigial wings that are completely invisible below the plumage. Photo by Judi Lapsley Miller, <a href="https://creativecommons.org/licenses/by/4.0">CC BY 4.0</a>.
</figcaption>
</figure>
</div>
<p>One of the most fascinating examples of vestigiality comes from Mexican cavefish (<em>Astyanax mexicanus</em>) that are eyeless and completely blind as adults (Figure @ref(fig:astyanax)). Cavefish actually start growing eyes during embryonic development, and the lack of eyes in adult fish is a consequence of the abortion of eye development within the first few days of the growing embryo. The eye in this case is a developmental (rather than a structural) vestigial organ that makes a transient appearance during certain embryonic stages. More importantly, cavefish actually possess all the genes required for the normal development of an eye. It turns out, eye abortion is initiated by signaling factors associated with the developing cavefish lens. Transplantation of a surface fish lens into a developing cavefish leads to the normal formation of an eye, just like transplantation of a cave fish lens into a surface fish embryo leads to eye abortion (Krishnan and Rohner 2017). If cavefish had originated independently, there would be no need for evolution to “create” the developmental machinery for eye development. Cavefish have that machinery because they lost eyes secondarily and inherited all the information for making an eye from their eyed surface ancestors.</p>
<div id="fig-astyanax" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-astyanax-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="images/astyanax.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-astyanax-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.10: Different forms of <em>Astyanax mexicanus</em>. Left: Cave form, which is completely blind and lacks body pigmentation as an adult. Right: An individual from a surface stream for comparison. Photos by <a href="https://www.flickr.com/photos/nihgov/27589386037">Daniel Castranova, NICHD/NIH</a>, <a href="https://creativecommons.org/share-your-work/public-domain/">Public Domain</a>.
</figcaption>
</figure>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
<strong>Atavisms</strong>
</div>
</div>
<div class="callout-body-container callout-body">
<p>Related to vestigial organs, atavisms are the reappearance of an ancestral trait or characteristic in an organism after several generations of absence. This phenomenon occurs when genes for previously existing phenotypic features are preserved in DNA and become expressed through mutations or changes in gene regulation. Some examples include:</p>
<ul>
<li>Whales and dolphins with hind limbs: Some cetaceans have been found with vestigial hind limbs, reminiscent of their land-dwelling ancestors (e.g. <span class="citation" data-cites="hall1984">HALL (<a href="#ref-hall1984" role="doc-biblioref">1984</a>)</span>).</li>
<li>Chickens with teeth (this one is a little creepy): A mutation in chickens can lead to the development of cone-shaped crocodile-like teeth, reflecting their common ancestry with reptiles (<span class="citation" data-cites="harris2006">Harris et al. (<a href="#ref-harris2006" role="doc-biblioref">2006</a>)</span>).</li>
<li>Humans with tails: Rarely, infants are born with vestigial tails, a trait present in our distant primate ancestors (<span class="citation" data-cites="tomic2011">Tomić and Meyer-Rochow (<a href="#ref-tomic2011" role="doc-biblioref">2011</a>)</span>).</li>
<li>Horses with extra toes: Occasionally, horses are born with additional toes, resembling their multi-toed ancestors (<span class="citation" data-cites="tyson2004">Tyson et al. (<a href="#ref-tyson2004" role="doc-biblioref">2004</a>)</span>).</li>
</ul>
<p>Atavisms provide strong support for the theory of evolution for several reasons. (1) They demonstrate that genetic information can be retained but not expressed for many generations, showing the continuity of genetic material through evolutionary time (Genetic preservation); (2) Atavisms often reveal traits shared with ancestral species, supporting the idea of common descent; and (3) They highlight the role of gene regulation in evolution, showing how changes in gene expression can lead to the reappearance of ancestral traits (Developmental plasticity).</p>
<p>Atavisms are powerful evidence for evolution, perhaps more a retrodiction than prediction. Nonetheless, by revealing these <em>evolutionary throwbacks</em>, atavisms offer tangible evidence of the genetic and developmental links between modern species and their ancestors, thereby strengthening the case for evolutionary theory.</p>
</div>
</div>
</section>
<section id="molecular-and-genomic-evidence-of-shared-ancestry" class="level3 unnumbered">
<h3 class="unnumbered anchored" data-anchor-id="molecular-and-genomic-evidence-of-shared-ancestry"><strong>Molecular and genomic evidence of shared ancestry</strong></h3>
<p>Some of the strongest evidence for evolution comes from molecular and genomics data, with much of it falling into the category of <em>shared ancestry</em>. In fact, if there were no other evidence for evolution, if there we no fossil record from which to draw, the evidence from the fields of molecular biology and genomics would alone prove evolution as the only explanation for the diversity of life. An entire field of research has emerged called <strong>molecular evolution</strong>, more recently rebranded as <strong>evolutionary genomics</strong> due to the staggering amount of DNA (and soon to be protein) data that we can now collect quickly, efficiently, and accurately. Molecular and genomic data is not only providing irrefutable evidence for evolution, it is also providing unique and exciting new insights into how evolution works.</p>
<p>In light of the above paragraph, what I am about to write should not come as a surprise; that the amount of evidence from molecular and genomic research is simply too vast to cover adequately, too overwhelming for this OER. I would point to advanced courses in the fields of molecular evolution and evolutionary genomics, and encourage you to explore the deep literature out there in the numerous scientific journals and books being published near daily, to really get a sense of the breath taking progress being made in these fast moving fields and the vast treasure trove of evidence coming from them. Nonetheless, it certainly is worth reviewing some of this evidence provided by molecular/genomic data, with the understanding that what we are about to discuss is but a small taste of what is in the literature. Later in this section we will discuss additional examples from molecular/genomics research that support other types of evidence. For now, we focus on those researches that fall under th category of <em>common ancestry</em>.</p>
<p>That being said, we will confine our discussion to the following:</p>
<section id="shared-molecular-mechanisms-across-all-life" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="shared-molecular-mechanisms-across-all-life"><strong><em>Shared molecular mechanisms across all life</em></strong></h4>
<p>Universally shared features, such as: - DNA as the basis of inheritance all life - chemical form of amino acids: L-amino acids, never mirror image D-stereoisomers - a relatively highly conserved genetic code - basic machinery of life and their molecular components. For example the mechanisms making up the so called Central Dogma of Molecular Biology; transcription, translation, and replication and the molecules that drive these processes. - other less well known molecular mechanisms universally shared, for example the amino-acyl-tRNA synthases, the essential molecules that charge tRNAs with the appropriate amino acid, considered to be the third genetic code.</p>
</section>
<section id="other-fundamental-conserved-molecular-mechanisms-and-biochemicalphysiological-pathways" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="other-fundamental-conserved-molecular-mechanisms-and-biochemicalphysiological-pathways"><strong><em>Other fundamental conserved molecular mechanisms and biochemical/physiological pathways</em></strong></h4>
<p>Biochemical, physiological, and developmental pathways that are shared across broad swaths of life. For example, a recently the conserved role of microRNA mir-193 in controlling melanic coloration in insects, demonstrates evolutionary continuity. Or the role of HOX gene clusters in driving development and morphological structures in complex Eukaryotes.</p>
<p>On a more fundamental level, the sharing of mitochondria adn their role in the production of ATP to drive cellular processes in just about all Eukaryotic lineages, or the sharing of chloroplasts in plants.</p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Note
</div>
</div>
<div class="callout-body-container callout-body">
<p>Horizontal gene transfer: Recognition of horizontal gene transfer events has expanded our understanding of evolutionary processes beyond vertical inheritance[9]. Yet even then, we can follow patterns of inhertiance after HGT that are due to sharing a common ancestor.</p>
</div>
</div>
</section>
<section id="levels-of-divergence" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="levels-of-divergence"><strong><em>Levels of divergence</em></strong></h4>
<p>The ability to sequence and compare genomes across species has clearly demonstrated that levels of sequence divergence are exactly as expected from divergence from common ancestry. The more closely related two taxa are from each other (from an evolutionary perspective, i.e.&nbsp;sharing a more recent common ancestor), the more similar they are in terms of DNA sequence divergence. The further apart, the more divergent. We see this not just at the level of DNA and protein sequence divergence, but also at “higher” levels of genomic and chromosomal organization and structure.</p>
</section>
<section id="pseudogenes-molecular-vestigial-structures" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="pseudogenes-molecular-vestigial-structures"><strong><em>Pseudogenes: molecular vestigial structures</em></strong></h4>
<p>Pseudogenes represent some of the strongest evidence out there for common ancestry. Just their very presence in genomes demands an evolutionary explanation, for how else could we reasonably explain their distribution among taxa and their persistance in most Eukaryotic genomes.</p>
<p>Pseudogenes are inactive copies of functional genes in the genome and represent another kind of evolutionary “flaw”. Pseudogenes can arise from two processes: (1) via a complex process generally referred to as <em>gene duplication</em>, which leads to what is referred to as <strong>classical pseudogene</strong>; (2) when processed messenger RNA (mRNA) is reverse-transcribed and inserted back into the genome, refrred to as a <strong>processed pseudogene</strong>. Reverse transcription is typically associated with retrotransposons or the activity of retroviruses in cells. Because processed mRNAs lack introns and other genetic elements important for transcription and translation, the complimentary DNA (cDNA) that is built back into the genome ends up being functionless. In the case of a classical pseudogene, once there is more than one copy of a gene in the genome, if the organism only requires one copy of the gene without an impact on the fitness, one of the copies can begin to accumulate mutations over time that can effectively knock the gene out for some reason. Hence, pseudogenes essentially represent junk DNA (though I really disike that term) invisible not only to the cellular machinery responsible for protein synthesis, but for natural selection as well.</p>
<p>The reason that pseudogenes are invisible for natural selection is that generally speaking they make no contribution to the phenotype of an organism, neither good or bad. While copy mistakes (<em>i.e.</em>, mutations) that impair the function of normal genes are usually eliminated by selection, similar mutations in pseudogenes have no effect and just linger around. As generations pass, pseudogenes consequently tend to accumulate more and more mutations compared to the original functional gene they originated from. Since mutations in the genome occur at predictable rates, we can compare pseudogenes to their functional equivalents to estimate when pseudogenes first arose. Conducting such analyses for pseudogenes in the human genome revealed that some of them are really old—much older in fact than the human species. This suggests that those pseudogenes must have arisen in an ancient ancestor, which we share with other closely related species. If this is the case, we should be able to find the same pseudogenes in other primates, but only those that have diverged from a common ancestor after the origin of the pseudogene. (See the section on mapping characters on a phylogeny for further information on this logic.)</p>
<div class="callout callout-style-default callout-important callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Important
</div>
</div>
<div class="callout-body-container callout-body">
<p>Recently there is increasing evidence that pseudogenes are actually not non-functional in the sense that they “do nothing”. Many pseudogeens are in fact transcribed, but no longer make functional mRNA for the original gene products they are derived from. It is not uncommon at all to find pseudogenes that code for various kinds of other functional RNAs, like miRNA, snoRNA, and lncRNAs, that serve important roles in regulating gene expression and other cellular processes. There is also some evidence that pseudogenes may impact gene expression levels. Nonetheless, this does not change the fact that there are clear patterns of pseudogenes shared among lineages due to the sharing of a common ancestor.</p>
</div>
</div>
<p>Friedberg and Rhoads (2000) put this hypothesis to the test (Table 2.1). The oldest pseudogene they investigated (CALM II 𝛙<sub>3</sub>), which has an estimated age of about 36 million years (Myr) based on the mutational difference from the functional equivalent, is found in all five species of primates that they investigated (including divergence times between 8 and 36 Myr). In contrast, the youngest pseudogene (𝛂-Enolase 𝛙<sub>1</sub>; 11 Myr old) is only found in chimpanzees and gorillas, the only primates that have diverged from the human lineage less than 11 Myr ago. Overall, the pattern of the presence and absence of pseudogenes is consistent with common ancestry. There is no reason for species to evolve the same pseudogenes independently in a predictable pattern. Rather, species share these “flaws” simply because they were passed down from one generation to the next, even as lineages split and formed new species.</p>
<table class="table-striped table-hover caption-top table">
<caption>Table 2.1: Pseudogenes that Friedberg and Rhoads (2000) detected in different primates (with hamsters as an outgroup). Individual pseudogenes identified in the human genome, along with their age estimates, are listed in rows. Different species with their estimated divergence times from humans are in columns. Symbols indicat the presence (+) or absence (-) of specific pseudogenes in a particular species.</caption>
<colgroup>
<col style="width: 26%">
<col style="width: 13%">
<col style="width: 10%">
<col style="width: 12%">
<col style="width: 9%">
<col style="width: 11%">
<col style="width: 11%">
</colgroup>
<thead>
<tr class="header">
<th></th>
<th style="text-align: center;">Chimpanzee</th>
<th style="text-align: center;">Gorilla</th>
<th style="text-align: center;">Orangutan</th>
<th style="text-align: center;">Rhesus</th>
<th style="text-align: center;">Capuchin</th>
<th style="text-align: center;">Hamster</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td></td>
<td style="text-align: center;">8 Myr</td>
<td style="text-align: center;">9 Myr</td>
<td style="text-align: center;">16 Myr</td>
<td style="text-align: center;">25 Myr</td>
<td style="text-align: center;">36 Myr</td>
<td style="text-align: center;">&gt;85 Myr</td>
</tr>
<tr class="even">
<td><p><strong>𝛂-Enolase</strong> 𝛙<sub><strong>1</strong></sub></p>
<p>11 Myr</p></td>
<td style="text-align: center;">+</td>
<td style="text-align: center;">+</td>
<td style="text-align: center;">-</td>
<td style="text-align: center;">-</td>
<td style="text-align: center;">-</td>
<td style="text-align: center;">-</td>
</tr>
<tr class="odd">
<td><p><strong>AS</strong> 𝛙<strong>7</strong></p>
<p>16 Myr</p></td>
<td style="text-align: center;">+</td>
<td style="text-align: center;">-</td>
<td style="text-align: center;">+</td>
<td style="text-align: center;">-</td>
<td style="text-align: center;">-</td>
<td style="text-align: center;">-</td>
</tr>
<tr class="even">
<td><p><strong>CALM II</strong> 𝛙<sub><strong>2</strong></sub></p>
<p>19 Myr</p></td>
<td style="text-align: center;">+</td>
<td style="text-align: center;">+</td>
<td style="text-align: center;">+</td>
<td style="text-align: center;">-</td>
<td style="text-align: center;">-</td>
<td style="text-align: center;">-</td>
</tr>
<tr class="odd">
<td><p><strong>AS</strong> 𝛙<sub><strong>1</strong></sub></p>
<p>25 Myr</p></td>
<td style="text-align: center;">+</td>
<td style="text-align: center;">+</td>
<td style="text-align: center;">+</td>
<td style="text-align: center;">+</td>
<td style="text-align: center;">-</td>
<td style="text-align: center;">-</td>
</tr>
<tr class="even">
<td><p><strong>AS</strong> 𝛙<sub><strong>3</strong></sub></p>
<p>25 Myr</p></td>
<td style="text-align: center;">+</td>
<td style="text-align: center;">+</td>
<td style="text-align: center;">+</td>
<td style="text-align: center;">+</td>
<td style="text-align: center;">-</td>
<td style="text-align: center;">-</td>
</tr>
<tr class="odd">
<td><p><strong>CALM II</strong> 𝛙<sub><strong>3</strong></sub></p>
<p>36 Myr</p></td>
<td style="text-align: center;">+</td>
<td style="text-align: center;">+</td>
<td style="text-align: center;">+</td>
<td style="text-align: center;">+</td>
<td style="text-align: center;">+</td>
<td style="text-align: center;">-</td>
</tr>
</tbody>
</table>
<p>There are numerous other examples of pseudogenes in the literature, more than can be given justice here. However, a few of my favorites include:</p>
<ul>
<li><p>L-gulano-g-lactone oxidase gene (GLO), required for vitamin C synthesis (discussed in the Coyne video)</p></li>
<li><p>olfactory receptors (OR), where something along the line of thousands of functional ORs are found in mice and dog genomes, but only around 400 functional ORs are found in humans, plus an additional 400 OR pseudogenes homologous with those found in mice or dogs. It is even more interesting in the Cetacea, dolphins and whales, where some 80% of ORs are pseudogenes (referenece needed).</p></li>
</ul>
<div id="fig-ORgenes" class="quarto-float quarto-figure quarto-figure-center anchored" width="75%">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-ORgenes-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="images/ORgenes.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-ORgenes-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.11: The full OR repertoire of 58 mammalian genomes. The sequenced genomes of 58 ecologically and phylogenetically diverse mammals were mined for OR gene sequences. The number of functional ORs (yellow bars) and non-functional ORs (grey bars) in the full OR gene repertoire varies between different species. Species are ranked based on the size of their functional repertoire. The coverage of the genomes from which these data were mined, as noted in their assembly builds online, are also included with “C” representing “complete” for human and mouse. From <span class="citation" data-cites="hughes2018">Hughes et al. (<a href="#ref-hughes2018" role="doc-biblioref">2018</a>)</span>.
</figcaption>
</figure>
</div>
<ul>
<li>hemoglobin in Antarctic icefishes (e.g. <span class="citation" data-cites="obrien2016">O’Brien (<a href="#ref-obrien2016" role="doc-biblioref">2016</a>)</span> ).</li>
</ul>
</section>
<section id="structural-variants-and-other-genomic-regions" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="structural-variants-and-other-genomic-regions"><strong><em>Structural variants and other genomic regions</em></strong></h4>
<ul>
<li>sharing of classes of transposable elements and sharing insertion sites in the genome following the sharing of MRCA</li>
</ul>
<div id="fig-SINEdist" class="quarto-float quarto-figure quarto-figure-center anchored" width="75%">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-SINEdist-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="images/BIO303 Evidence S21.074.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-SINEdist-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.12: Distribution of various SINE retroelements among primate lineages. The image on the left shows the reults of PCR amplification for 5 different SINEs in human, chimp, bonobo, gorilla, orang-utan, siamang, green monkey, and owl monkey. In each gel image, the presence of a larger fragment signifies the presence of that particular SINE element in the genome. The image on the right shows the mapping of presence of the various SINEs along the primate phylogeny.
</figcaption>
</figure>
</div>
<ul>
<li>distribution and evolutionary pattern of members of gene families</li>
<li>distribution and evolutionary pattern of Copy Number Variants (CNV)</li>
</ul>
<p>These molecular and genomic findings collectively strengthen the evidence for common descent, natural selection, and the gradual accumulation of genetic changes over time, all key components of the Theory of Evolution.</p>
</section>
</section>
<section id="why-homologies-matter" class="level3 unnumbered">
<h3 class="unnumbered anchored" data-anchor-id="why-homologies-matter"><strong>Why Homologies Matter</strong></h3>
<p>The fact that evolutionary novelties occur in nested sets as predicted by descent with modification provides strong evidence for common ancestry. The finding also has far reaching implications, because it underlies all biomedical research and applications. The reason we can study DNA repair mechanisms in bacteria to learn about their role in cancer development is because DNA repair mechanisms in bacteria and humans are homologous. The reason we can study cell cycle regulation in yeast is because both yeast and humans inherited the same regulatory machinery from a common ancestor. The reason we can study drug responses in rodents is because the physiological processing of many substances is mediated by homologous pathways in rodents and humans. And we can gain insights about neurophysiology and psychiatry from other primates, again because we all inherited our neurosystem from a common ancestor. The reason evolution is the unifying theory of biology is because it provides the critical framework for comparative studies among species, helping us to make sure that we are actually comparing apples to apples (<em>i.e.</em>, homologous structures). If not, inferences from comparative analyses can be deeply flawed.</p>
</section>
</section>
<section id="sec-biogeo" class="level2" data-number="3.7">
<h2 data-number="3.7" class="anchored" data-anchor-id="sec-biogeo"><span class="header-section-number">3.7</span> Biogeography</h2>
<p>Patterns seen in the distribution of species both on a macro- (broad scale) and micro-level had been well recognized since before Darwin published OTOS. For example, the similarity and differences in species across continents such as marsupials in Australia while placental mammals dominated the rest of the world, or the deserts of Africa and of the Americas which harbor species that were superficially similar in form but showed other more fundamental diﬀerences, were well recognized. Even on smaller scales, for example the unique characteristic of island fauna and the relationship of island fauna to nearby continental or other island regions, were also increasingly being described. In both cases these patterns of species distributions were being descibed yet difficult to explain under the natural theistic explanations that dominated western science at the time.</p>
<p>Darwin knew of these distribution patterns, He had observed some directly and read extensively about these and similar patterns, and thought long on what might explain these types of observations. His explanation, which took up two entire chapters in the OTOS (evidence of how important he thought it was), provided an evolutionary explanation of the geography of life. His original evolutionary explanation was largely correct when first proposed, and has only been refined and supported by a multitude of later studies. Since then molecular studies demonstrating the relationship of species both within and among regions has only further solidified the evidence of biogeographic patterns as evidence for evolution</p>
<section id="continental-scale" class="level3" data-number="3.7.1">
<h3 data-number="3.7.1" class="anchored" data-anchor-id="continental-scale"><span class="header-section-number">3.7.1</span> Continental scale</h3>
<p>Some examples of macrogeographic evidence are:</p>
<ul>
<li>Similar ecosystems on different continents often harbor species that look the same. They occupy similar niches in the two continental systems, but clearly are very different in terms of sharing of a common ancestor. The most famous example of diﬀerent species filling similar roles involves is the marsupial mammals, now found mainly in Australia (the Virginia opossum <em>Didelphis virginiana</em> is a familiar exception), and placental mammals, which predominate elsewhere in the world (you probably discussed this extensively in your high school biology courses and in BIO114). The two groups show some important anatomical diﬀerences (most notably in their reproductive systems, where almost all marsupials have pouches and give birth to very undeveloped young, while placentals have placentas that enable young to be born at a more advanced stage) and represent two very different lineages that diverged approximately 160 million years ago (Ma) during the Jurassic period. Nevertheless, in other ways some marsupials and placentals are astonishingly similar looking. There are burrowing marsupial moles that look and act just like placental moles, marsupial mice that resemble placental mice, marsupial anteaters which do exactly what South American anteaters do, and a marsupial wolf <em>Thylacinus cynocephalus</em>, also known as the Tasmanian wolf or Tasmanian tiger, which is strikingly canine-like appearance and served the same ecological niche as mammalian wolves.</li>
</ul>
<div id="fig-marsup" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-marsup-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="images/Biogeog.jpg" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-marsup-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.13: Examples of convergent evolution in marsupial and placental mammals as biogeographic evidence for evolution (@ MacMillen Press.
</figcaption>
</figure>
</div>
<ul>
<li><p>Descent with modification predicts that organisms will be most similar in regions representing places of origin and subsequent dispersal. For example, descent with modification predicts that organisms in African tropics will be more similar to organisms in African deserts than to organisms in South American tropics because they are more likely to share a common ancestor. As discussed in Coyne (), we see this in many desert plants are succulents: they show an adaptive combination of traits that include large fleshy stems to store water, spines to deter predators, and small or missing leaves to reduce water loss. But diﬀerent deserts have diﬀerent types of succulents. In North and South America, the succulents are members of the cactus family. But in the deserts of Asia, Australia, and Africa, there are no native cacti, and the succulents belong to a completely diﬀerent family, the euphorbs. You can tell the diﬀerence between the two types of succulents by their flowers and their sap, which is clear and watery in cacti but milky and bitter in euphorbs. Yet despite these fundamental diﬀerences, cacti and euphorbs can look very much alike.</p></li>
<li><p>Another body of evidence can be found in the existence of similar, yet distinct, species inhabiting comparable environments across different continents. For instance, the red deer of Northern Europe and the North American elk share a striking resemblance. Similarly, the Pipidae family of tongueless aquatic frogs can be found in both South America and Africa. These distribution patterns would be perplexing if the continents had always been in their current positions. These animals would have no way to traverse the vast oceans.</p></li>
</ul>
<div>

</div>
<div class="quarto-layout-panel" data-layout-ncol="2">
<div class="quarto-layout-row">
<div class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/reddeer.jpg" class="img-fluid figure-img"></p>
<figcaption>Red deer; from https://en.wikipedia.org/wiki/Red_deer</figcaption>
</figure>
</div>
</div>
<div class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/elk.jpg" class="img-fluid figure-img"></p>
<figcaption>Elk; from https://en.wikipedia.org/wiki/Elk</figcaption>
</figure>
</div>
</div>
</div>
</div>
<p>However, the theory of continental drift clarifies this phenomenon. Ancient land bridges, such as the Bering Land Bridge connecting Asia and North America, facilitated the dispersal of these species. Once these species established themselves in new territories, they underwent evolution, leading to the diversity we observe today.</p>
<ul>
<li>Another prediction of evolution is that species in a particular region should be descendants of earlier species that inhabited the same area. This prediction is supported by fossil records. For example, fossil wambats that most closely resemble modern wambats are found exclusively in Australia. Similarly, the unique armadillos of the Americas, with their bony armor, have fossil relatives called glyptodonts, also exclusive to the Americas. These glyptodonts were massive, armored herbivores that resembled oversized armadillos.</li>
</ul>
<div id="fig-succ" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-succ-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="images/Succession.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-succ-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.14: Fossil and living organisms in the same region are more similar to each other than they are to organisms from other regions, referred to as “succession”
</figcaption>
</figure>
</div>
</section>
<section id="smaller-scale-microgeographic-patterns-of-similarity-in-island-chains" class="level3" data-number="3.7.2">
<h3 data-number="3.7.2" class="anchored" data-anchor-id="smaller-scale-microgeographic-patterns-of-similarity-in-island-chains"><span class="header-section-number">3.7.2</span> Smaller scale, microgeographic patterns of similarity in island chains</h3>
<p>On a smaller, island-like scale, descent with modification successfully predicts explains a number of observed patterns:</p>
<ul>
<li>island fauna and flora tend to be subsets of those found on nearby continents, though the distribution of species tend to be unbalanced. For example, we usually find island systems having plants, birds, insects (flying mostly), and bats that are quite similar to nearby continental ecosystems, while oftentimes completely lacking amphibians, mammals (except bats), reptiles, and fish. Of course this is not always the case as it depends on the distance of the islands from the mainland, the surrounding oceanic currents, and the dispersal ability of the species (in addition to pure luck).</li>
<li>island fauna are more similar to adjacent mainland forms, even when environments differ, than they are to island forms from other regions (even when environments the same).</li>
<li>patterns of similarity among island forms within a chain (<em>e.g.</em> archepelegoes like Hawai’i) often conform to patterns of origins and distributions of islands – <em>i.e.</em>, patterns of similarity among organisms follow those predicted based on when islands originate, when they could be colonized, etc. Spectacular examples of these include the Honey-creepers of the Hawai’in islands (and numerous other species as well), and of course the mockingbrids of Darwin fame in the Galapagos.</li>
</ul>
</section>
<section id="summarizing-biogeographic-evidence" class="level3" data-number="3.7.3">
<h3 data-number="3.7.3" class="anchored" data-anchor-id="summarizing-biogeographic-evidence"><span class="header-section-number">3.7.3</span> Summarizing biogeographic evidence</h3>
<p>The main lesson from biogeographic patterns is that evolution provides the only explanation for the distribution and diversity of life on continents and islands. It is a mixture of both prediction and retrodiction, consistent with and explained by how evoution works.</p>
<p>Today these patterns of distribution in species within and among continents and island systems are only reinforced by the deluge of molecular data in the form of levels of sequence divergence and phylogenetic trees. It is an absolutely overwhelming body of evidence, seeming to increase in number near exponentially on a yearly basis. In these types of studies, the levels of sequence divergence and the evolutionary relationships shown by phylogenetic analyses are usually consistent with the patterns observed from morphological and species distribution data across various geographic scales.</p>
<p>The theory of <a href="https://en.wikipedia.org/wiki/Plate_tectonics">Plate Tectonics</a> (only recognized and accepted by the scientific community in the mid-to-late 1960s), which unified earlier concepts like Alfred Wegener’s continental drift (proposed in 1912) with new geological evidence, provided a comprehensive explanation for Earth’s geological processes which proved to be consistent not only with observed species distribution patterns, but also the observed levels of sequence and phylogenetic divergence (phylogeographic patterns).</p>
<p>Taken as a whole, it is unarguable that biogeographic and phylogeographic patterns provide strong evidence for evolution.</p>
</section>
</section>
<section id="sec-deeptime" class="level2" data-number="3.8">
<h2 data-number="3.8" class="anchored" data-anchor-id="sec-deeptime"><span class="header-section-number">3.8</span> Deep Time</h2>
<p>The last prediction of Darwin’s idea of descent with modification is that Earth and life on it are old. The study of the age of Earth and the universe is not really a subject of biology (hence, I will only touch on this briefly). The age of Earth is chiefly studied by geologists who combine isotopic analyses with an understanding of radioactive decay (radiometric dating), and they have established that Earth is about 4.54 billion years old (see <a href="https://www.scientificamerican.com/article/how-science-figured-out-the-age-of-the-earth/">Paul Braterman’s article in <em>Scientific American</em></a> if you want more information). Similarly, astronomers have estimated the age of the universe at 13.8 billion years by measuring the rate of expansion of the universe and extrapolating back to the Big Bang (see <a href="https://www.forbes.com/sites/startswithabang/2019/12/10/this-is-how-astronomers-know-the-age-of-the-universe-and-you-can-too/?sh=31f9137d16d5">Ethan Siegel’s article in <em>Forbes</em></a>). Evidence for the age of life on Earth comes unsurprisingly from the fossil record. The oldest known fossils are cyanobacteria found in Australian rock formations. Radiometric dating has revealed that they are 3.5 billion years old. Hence, all evidence indicates that life on Earth has had incredibly long periods of time to evolve and create the diversity of organisms observable today.</p>
</section>
<section id="sec-tiktaalik" class="level2" data-number="3.9">
<h2 data-number="3.9" class="anchored" data-anchor-id="sec-tiktaalik"><span class="header-section-number">3.9</span> Correspondence of Different Lines of Evidence</h2>
<p>Inference in science is strongest when there is a clear correspondence between different lines of evidence that all support a central hypothesis (consilience). In fact, we can often use existing information to formulate testable hypotheses that then can be addressed with alternative approaches. You have already learned about one such example in the context of pseudogenes. Estimating the age of pseudogenes by tallying the number of mutations between a pseudogene and its functional equivalent led to clear predictions about the phylogenetic distribution of pseudogenes to test for common ancestry.</p>
<p>The discovery of <em>Tiktaalik</em>, a transitional fossil between aquatic and terrestrial vertebrates discovered by a research team around Neil Shubin (<a href="#fig-tiktaalik" class="quarto-xref">Figure&nbsp;<span>3.15</span></a>), is another example for the role of interdisciplinary research in making discoveries that transform our understanding. <em>Tiktaalik</em> was not discovered haphazardly by a bunch of rock-loving paleontologists that were just looking for fossils. Its discovery was deliberate and testimony to the power of the scientific method. Wanting to find a transitional fossil that exhibited characteristics of both fish and early tetrapods, Shubin and his team first turned to molecular phylogenetic analyses of vertebrates. Essentially, they used DNA sequences to not only infer the evolutionary relationships between different vertebrate groups but also to date when different lineages split from each other (we will learn exactly how this works in <a href="molecular-evolution.html">Chapter 7</a>). These analyses revealed that terrestrial vertebrates (Tetrapoda) are sister to the lungfishes (Dipnoi), a lineage from which they split between 350-425 million years ago (<a href="#fig-vertphylo" class="quarto-xref">Figure&nbsp;<span>3.16</span></a>). Any transitional fossils that exhibit traits intermediate between the two groups should consequently be found in rock layers of about that age. Accordingly, Shubin and colleagues took out a geological map of Earth in search of exposed rock formations of the correct age range, and they found some on Ellesmere Island in the Nunavut Territory of Canada. After a few disappointing field seasons, Shubin and his team indeed found a fossil with the predicted combination of traits in 2004. <em>Tiktaalik roseae</em>, as they named the newly discovered species, exhibited gills and scales like fish but also limb bones characteristic of today’s land animals (Daeschler et al.&nbsp;2006). It’s the combination of multiple approaches rooted in molecular biology, evolutionary analyses, and paleontology that ultimately led to the discovery of this missing link. Looking for corresponding evidence from different research approaches leads to the most robust inference in science, an approach frequently used in evolutionary biology.</p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Explore More
</div>
</div>
<div class="callout-body-container callout-body">
<p>If you want to learn more about the fascinating discovery of Tiktaalik and its implications for evolution and our own origins, I recommend you either read Neil Shubin’s book “<a href="https://www.amazon.com/Your-Inner-Fish-Journey-3-5-Billion-Year/dp/0307277453"><em>Your Inner Fish</em></a>” or watch the <a href="https://www.pbs.org/your-inner-fish/">PBS series based on the book</a>.</p>
</div>
</div>
<div id="fig-tiktaalik" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-tiktaalik-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="images/tiktaalik.jpg" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig quarto-uncaptioned" id="fig-tiktaalik-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.15
</figcaption>
</figure>
</div>
<div id="fig-vertphylo" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-vertphylo-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="images/titaakphyl.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-vertphylo-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.16: A simplified phylogenetic tree of vertebrates. Terrestrial vertebrates (Tetrapoda) are part of the lobed-finned fishes (Sacropterygii) and split from their sister group (the lungfishes, Dipnoi) between 350 and 425 million years ago. The estimated range for potential transitional fossils is highlighted in gray, the age of the <em>Tiktaalik</em> fossil in red.
</figcaption>
</figure>
</div>
</section>
<section id="absence-of-evidence" class="level2" data-number="3.10">
<h2 data-number="3.10" class="anchored" data-anchor-id="absence-of-evidence"><span class="header-section-number">3.10</span> Absence of Evidence…</h2>
<p>A frequent argument of critics of evolutionary theory (and science in general) is that we cannot explain everything, and indeed there are some major open questions that remain largely unaddressed: How did life on Earth originate? What were the characteristics of the last universal common ancestor of Archaea, Bacteria, and Eukaryotes? How and why did the eukaryotic cell arise? How did the transitional forms between some major taxonomic groups look like? Why is it that so many species reproduce by having sex?</p>
<p>The fact that we do not know the answers to these questions, and many others like them, does not undermine what we do know about evolution and science. It is precisely why evolutionary biology is an exciting field of research. More importantly, <strong>absence of evidence is not evidence for absence</strong>. For example, gaps in the fossil record and a lack of transitional forms between some taxonomic groups does not negate what we have learned about evolutionary patterns and processes. Perhaps some of these key fossils have just not been found yet, or they may be completely lost to time (because fossilization is actually a rare process). Ultimately, the probability that evolution is true based on other evidence is high enough that a lack of a specific fossil cannot call it into question. Only novel evidence—for example, new fossils that directly contradict our current understanding—has the potential to reshape evolutionary biology. In other words, the burden of proof about any inaccuracies in our current understanding of evolution lies with the critic and not the untouched gaps in our current understanding. And as scientists, it is our day-to-day business to detect and correct those inaccuracies, rather than defending the <em>status quo</em> blindly.</p>
</section>
<section id="in-closing" class="level2" data-number="3.11">
<h2 data-number="3.11" class="anchored" data-anchor-id="in-closing"><span class="header-section-number">3.11</span> In closing</h2>
<p>In closing this chapter, to summarize the evidence of evolution it is safe to say that <strong>there is overwhelming evidence from many different areas in the natural science that all support the Theory of Evolution.</strong> In all the decades since Darwin first proposed this theory of evolution, there has not been a single instance of evidence inconsistent with the interpretation that evolution is the only scientific explanation for the diversity of life. There certainly have been a lot of opinions and attempts to come up with alternative explanations (for example <a href="https://en.wikipedia.org/wiki/Creation_science">creation science</a> and <a href="https://en.wikipedia.org/wiki/Intelligent_design">intelligent design</a>), yet not a single one of these alternatives have withstood the cold, harsh filter of empirical science. Strong beliefs or opinions are not scientific evidence.</p>
<p>In other words, there is no <a href="https://en.wikipedia.org/wiki/Teach_the_Controversy?t">controversy to teach regarding the validity of evolutionary theory</a>. The perceived controversy exists solely in the realms of politics and religion, not science. Evolution is a decided fact.</p>
</section>
<section id="reflection-questions" class="level2" data-number="3.12">
<h2 data-number="3.12" class="anchored" data-anchor-id="reflection-questions"><span class="header-section-number">3.12</span> Reflection Questions</h2>
<ol type="1">
<li>There is phenomenal variation in human height. The graph below shows variation among over 80 human populations (countries; in different colors) and across time (from 1897 to 1996). As you can see, there is a spread of about 20 cm in mean height among populations that has persisted through time. In addition, mean height has increased by an average of ~8 cm over 100 years (black line). Do you think the variation in height among the populations and the change through time are the product of evolution? Why? If you want to explore these data further, you can download them <a href="data/human_height.csv">here</a>. Data was originally retrieved from <a href="https://ourworldindata.org/human-height"><em>Our World in Data</em></a> (<a href="https://creativecommons.org/licenses/by/4.0/">CC BY 4.0</a>).</li>
</ol>
<!-- -->
<ol start="2" type="1">
<li>The evidence for speciation discussed in this chapter hinged on a specific definition of speciation. What do you think are some of the disadvantages of that specific definition?</li>
<li>Transitional fossils are a hallmark of evolution. However, we lack transitional fossils between many groups of organisms, especially between phyla that arose during the Cambrian explosion. Why do you think this is? How does this undermine evolutionary theory?</li>
</ol>
</section>
<section id="references" class="level2" data-number="3.13">
<h2 data-number="3.13" class="anchored" data-anchor-id="references"><span class="header-section-number">3.13</span> References</h2>
<ul>
<li><p>Carroll, SP, AP Hendry, DN Reznick, CW Fox (2007). <a href="https://besjournals.onlinelibrary.wiley.com/doi/full/10.1111/j.1365-2435.2007.01289.x">Evolution on ecological time-scales</a>. <em>Functional Ecology</em> 21, 387–393.</p></li>
<li><p>Daeschler, EB, NH Shubin, FA Jenkins Jr.&nbsp;(2006). <a href="https://www.nature.com/articles/nature04639">A Devonian tetrapod-like fish and the evolution of the tetrapod body plan</a>. <em>Nature</em> 440, 757–763.</p></li>
<li><p>Darwin, C. (1859). <a href="https://www.biodiversitylibrary.org/item/122307#page/7/mode/1up"><em>On the origin of species based on natural selection, or the preservation of favoured races in the struggle of life</em></a>. John Murray.</p></li>
<li><p>Friedberg, F, AR Rhoads (2000). <a href="https://pubmed.ncbi.nlm.nih.gov/10877945/">Calculation and verification of the ages of retroprocessed pseudogenes</a>. <em>Molecular Phylogenetics and Evolution</em> 16, 127–130.</p></li>
<li><p>Hairston Jr, NG, SP Ellner, MA Geber, T Yoshida, JA Fox (2005). <a href="https://onlinelibrary.wiley.com/doi/full/10.1111/j.1461-0248.2005.00812.x">Rapid evolution and the convergence of ecological and evolutionary time</a>. <em>Ecology Letters</em> 8, 1114–1127.</p></li>
<li><p>Hendry, AP, CL Peichel, B Matthews, JW Boughman, P Nosil (2013). <a href="http://www.evolutionary-ecology.com/open/ccar2833.pdf">Stickleback research: the now and the next</a>. <em>Evolutionary Ecology Research</em>, 15, 111–141.</p></li>
<li><p>Irwin, DE, S Bensch, JH Irwin, TD Price (2005). <a href="https://science.sciencemag.org/content/307/5708/414">Speciation by distance in a ring species</a>. <em>Science</em> 307, 414–416.</p></li>
<li><p>Jones, FC, MG Grabherr, YF Chan, P Russell, E Mauceli, J Johnson, … DM Kingsley (2012). <a href="https://www.nature.com/articles/nature10944">The genomic basis of adaptive evolution in threespine sticklebacks</a>. <em>Nature</em> 484, 55–61.</p></li>
<li><p>Krishnan, J, N Rohner (2017). <a href="https://royalsocietypublishing.org/doi/10.1098/rstb.2015.0487">Cavefish and the basis for eye loss</a>. <em>Philosophical Transactions of the Royal Society of London B</em> 372, 20150487.</p></li>
<li><p>Laurentino, TG, D Moser, M Roesti, M Ammann, A Frey, F Ronco, B Kueng, D Berner (2020). <a href="https://www.nature.com/articles/s41467-020-15657-3">Genomic release-recapture experiment in the wild reveals within-generation polygenic selection in stickleback fish</a>. <em>Nature Communications</em> 11, 1928.</p></li>
<li><p>Lescak, EA, SL Bassham, J Catchen, O Gelmond, ML Sherbick, FA von Hippel, WA Cresko (2015). <a href="https://www.pnas.org/content/112/52/E7204">Evolution of stickleback in 50 years on earthquake-uplifted islands</a>. <em>Proceedings of the National Academy of Sciences USA</em> 112, E7204–E7212.</p></li>
<li><p>Marth, JD (2008). <a href="https://www.nature.com/articles/ncb0908-1015">A unified vision of the building blocks of life</a>. <em>Nature Cell Biology</em> 10, 1015–1016.</p></li>
<li><p>Nosil, P (2007). <a href="https://www.journals.uchicago.edu/doi/abs/10.1086/510634">Divergent host plant adaptation and reproductive isolation between ecotypes of <em>Timema</em> <em>cristinae</em> walking sticks</a>. <em>American Naturalist</em> 169, 151–162.</p></li>
<li><p>Ostrom, JH (1976). <a href="https://onlinelibrary.wiley.com/doi/abstract/10.1111/j.1095-8312.1976.tb00244.x"><em>Archaeopteryx</em> and the origin of birds</a>. <em>Biological Journal of the Linnean Society. Linnean Society of London</em> 8, 91–182.</p></li>
<li><p>Pereira, RJ, WB Monahan, DB Wake (2011). <a href="https://bmcecolevol.biomedcentral.com/articles/10.1186/1471-2148-11-194">Predictors for reproductive isolation in a ring species complex following genetic and ecological divergence</a>. <em>BMC Evolutionary Biology</em> 11, 194.</p></li>
<li><p>Woodburne, MO, WJ Zinsmeister (1982). <a href="https://science.sciencemag.org/content/218/4569/284">Fossil land mammal from Antarctica</a>. <em>Science</em> 218, 284–286.</p></li>
<li><p>Bell, M. A., Baumgartner, J. V., &amp; Olson, E. C. (1985). Patterns of Temporal Change in Single Morphological Characters of a Miocene Stickleback Fish. Paleobiology, 11(3), 258–271. :::::::::</p></li>
</ul>


<div id="refs" class="references csl-bib-body hanging-indent" data-entry-spacing="0" role="list">
<div id="ref-bell2009" class="csl-entry" role="listitem">
Bell, M. A. 2009. <span>“Implications of a Fossil Stickleback Assemblage for Darwinian Gradualism.”</span> <em>Journal of Fish Biology</em> 75 (8): 1977–99. <a href="https://doi.org/10.1111/j.1095-8649.2009.02416.x">https://doi.org/10.1111/j.1095-8649.2009.02416.x</a>.
</div>
<div id="ref-boisseau2024" class="csl-entry" role="listitem">
Boisseau, Romain P., Sven Bradler, and Douglas J. Emlen. 2024. <span>“Divergence Time and Environmental Similarity Predict the Strength of Morphological Convergence in Stick and Leaf Insects.”</span> <em>Proceedings of the National Academy of Sciences</em> 122 (1). <a href="https://doi.org/10.1073/pnas.2319485121">https://doi.org/10.1073/pnas.2319485121</a>.
</div>
<div id="ref-burke2009" class="csl-entry" role="listitem">
Burke, Molly K., and Michael R. Rose. 2009. <span>“Experimental Evolution with<span><em>Drosophila</em></span>.”</span> <em>American Journal of Physiology-Regulatory, Integrative and Comparative Physiology</em> 296 (6): R1847–54. <a href="https://doi.org/10.1152/ajpregu.90551.2008">https://doi.org/10.1152/ajpregu.90551.2008</a>.
</div>
<div id="ref-carroll2007" class="csl-entry" role="listitem">
CARROLL, S. P., A. P. HENDRY, D. N. REZNICK, and C. W. FOX. 2007. <span>“Evolution on Ecological Time<span>-</span>Scales.”</span> <em>Functional Ecology</em> 21 (3): 387–93. <a href="https://doi.org/10.1111/j.1365-2435.2007.01289.x">https://doi.org/10.1111/j.1365-2435.2007.01289.x</a>.
</div>
<div id="ref-carroll2005" class="csl-entry" role="listitem">
Carroll, Scott P., Jenella E. Loye, Hugh Dingle, Michael Mathieson, Thomas R. Famula, and Myron P. Zalucki. 2005. <span>“And the Beak Shall Inherit <span></span> Evolution in Response to Invasion.”</span> <em>Ecology Letters</em> 8 (9): 944–51. <a href="https://doi.org/10.1111/j.1461-0248.2005.00800.x">https://doi.org/10.1111/j.1461-0248.2005.00800.x</a>.
</div>
<div id="ref-cook2012" class="csl-entry" role="listitem">
Cook, L M, and I J Saccheri. 2012. <span>“The Peppered Moth and Industrial Melanism: Evolution of a Natural Selection Case Study.”</span> <em>Heredity</em> 110 (3): 207–12. <a href="https://doi.org/10.1038/hdy.2012.92">https://doi.org/10.1038/hdy.2012.92</a>.
</div>
<div id="ref-hairston2005" class="csl-entry" role="listitem">
Hairston, Nelson G., Stephen P. Ellner, Monica A. Geber, Takehito Yoshida, and Jennifer A. Fox. 2005. <span>“Rapid Evolution and the Convergence of Ecological and Evolutionary Time.”</span> <em>Ecology Letters</em> 8 (10): 1114–27. <a href="https://doi.org/10.1111/j.1461-0248.2005.00812.x">https://doi.org/10.1111/j.1461-0248.2005.00812.x</a>.
</div>
<div id="ref-hall1984" class="csl-entry" role="listitem">
HALL, BRIAN K. 1984. <span>“DEVELOPMENTAL MECHANISMS UNDERLYING THE FORMATION OF ATAVISMS.”</span> <em>Biological Reviews</em> 59 (1): 89–122. <a href="https://doi.org/10.1111/j.1469-185x.1984.tb00402.x">https://doi.org/10.1111/j.1469-185x.1984.tb00402.x</a>.
</div>
<div id="ref-harris2006" class="csl-entry" role="listitem">
Harris, Matthew P., Sean M. Hasso, Mark W. J. Ferguson, and John F. Fallon. 2006. <span>“The Development of Archosaurian First-Generation Teeth in a Chicken Mutant.”</span> <em>Current Biology</em> 16 (4): 371–77. <a href="https://doi.org/10.1016/j.cub.2005.12.047">https://doi.org/10.1016/j.cub.2005.12.047</a>.
</div>
<div id="ref-hof2016" class="csl-entry" role="listitem">
Hof, Arjen E. van’t, Pascal Campagne, Daniel J. Rigden, Carl J. Yung, Jessica Lingley, Michael A. Quail, Neil Hall, Alistair C. Darby, and Ilik J. Saccheri. 2016. <span>“The Industrial Melanism Mutation in British Peppered Moths Is a Transposable Element.”</span> <em>Nature</em> 534 (7605): 102–5. <a href="https://doi.org/10.1038/nature17951">https://doi.org/10.1038/nature17951</a>.
</div>
<div id="ref-hughes2018" class="csl-entry" role="listitem">
Hughes, Graham M, Emma S M Boston, John A Finarelli, William J Murphy, Desmond G Higgins, and Emma C Teeling. 2018. <span>“The Birth and Death of Olfactory Receptor Gene Families in Mammalian Niche Adaptation.”</span> Edited by Yoko Satta. <em>Molecular Biology and Evolution</em> 35 (6): 1390–1406. <a href="https://doi.org/10.1093/molbev/msy028">https://doi.org/10.1093/molbev/msy028</a>.
</div>
<div id="ref-johnston1964" class="csl-entry" role="listitem">
Johnston, Richard F., and Robert K. Selander. 1964. <span>“House Sparrows: Rapid Evolution of Races in North America.”</span> <em>Science</em> 144 (3618): 548–50. <a href="https://doi.org/10.1126/science.144.3618.548">https://doi.org/10.1126/science.144.3618.548</a>.
</div>
<div id="ref-obrien2016" class="csl-entry" role="listitem">
O’Brien, Kristin M. 2016. <span>“New Lessons from an Old Fish: What Antarctic Icefishes May Reveal about the Functions of Oxygen-Binding Proteins.”</span> <em>Integrative and Comparative Biology</em> 56 (4): 531–41. <a href="https://doi.org/10.1093/icb/icw062">https://doi.org/10.1093/icb/icw062</a>.
</div>
<div id="ref-palumbi2001" class="csl-entry" role="listitem">
Palumbi, Stephen R. 2001. <span>“Humans as the World’s Greatest Evolutionary Force.”</span> <em>Science</em> 293 (5536): 1786–90. <a href="https://doi.org/10.1126/science.293.5536.1786">https://doi.org/10.1126/science.293.5536.1786</a>.
</div>
<div id="ref-tian2024" class="csl-entry" role="listitem">
Tian, Shen, Yoshimasa Asano, Tirtha Das Banerjee, Shinya Komata, Jocelyn Liang Qi Wee, Abigail Lamb, Yehan Wang, et al. 2024. <span>“A microRNA Is the Effector Gene of a Classic Evolutionary Hotspot Locus.”</span> <em>Science</em> 386 (6726): 1135–41. <a href="https://doi.org/10.1126/science.adp7899">https://doi.org/10.1126/science.adp7899</a>.
</div>
<div id="ref-tomic2011" class="csl-entry" role="listitem">
Tomić, Nenad, and Victor Benno Meyer-Rochow. 2011. <span>“Atavisms: Medical, Genetic, and Evolutionary Implications.”</span> <em>Perspectives in Biology and Medicine</em> 54 (3): 332–53. <a href="https://doi.org/10.1353/pbm.2011.0034">https://doi.org/10.1353/pbm.2011.0034</a>.
</div>
<div id="ref-tyson2004" class="csl-entry" role="listitem">
Tyson, Reid, John P. Graham, Patrick T. Colahan, and Clifford R. Berry. 2004. <span>“Skeletal Atavism in a Miniature Horse.”</span> <em>Veterinary Radiology &amp; Ultrasound</em> 45 (4): 315–17. <a href="https://doi.org/10.1111/j.1740-8261.2004.04060.x">https://doi.org/10.1111/j.1740-8261.2004.04060.x</a>.
</div>
</div>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="./HistoricaPerspective.html" class="pagination-link" aria-label="A brief history of evolutionary thought">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">A brief history of evolutionary thought</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="./HowNaturalSelectionWorks.html" class="pagination-link" aria-label="How Natural Selection Works">
        <span class="nav-page-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">How Natural Selection Works</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->




</body></html>