<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.5.57">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>13&nbsp; The Evolution of Quantitative Traits – Evolution 303</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<link href="./KinAndSexualSelection.html" rel="next">
<link href="./MechEvolMigration.html" rel="prev">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>

  <script src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6"></script>
  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

<script type="text/javascript">
const typesetMath = (el) => {
  if (window.MathJax) {
    // MathJax Typeset
    window.MathJax.typeset([el]);
  } else if (window.katex) {
    // KaTeX Render
    var mathElements = el.getElementsByClassName("math");
    var macros = [];
    for (var i = 0; i < mathElements.length; i++) {
      var texText = mathElements[i].firstChild;
      if (mathElements[i].tagName == "SPAN") {
        window.katex.render(texText.data, mathElements[i], {
          displayMode: mathElements[i].classList.contains('display'),
          throwOnError: false,
          macros: macros,
          fleqn: false
        });
      }
    }
  }
}
window.Quarto = {
  typesetMath
};
</script>

</head>

<body class="nav-sidebar floating">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" role="button" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
        <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./GenesMutationGeneticVariation.html">Evolutionary Genetics</a></li><li class="breadcrumb-item"><a href="./EvolPhenotype.html"><span class="chapter-number">13</span>&nbsp; <span class="chapter-title">The Evolution of Quantitative Traits</span></a></li></ol></nav>
        <a class="flex-grow-1" role="navigation" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
        </a>
      <button type="button" class="btn quarto-search-button" aria-label="Search" onclick="window.quartoOpenSearch();">
        <i class="bi bi-search"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal quarto-sidebar-collapse-item sidebar-navigation floating overflow-auto">
    <div class="pt-lg-2 mt-2 text-left sidebar-header">
    <div class="sidebar-title mb-0 py-0">
      <a href="./">Evolution 303</a> 
    </div>
      </div>
        <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
        </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Evolution 303</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./Preface.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Preface - Evolution 303</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" role="navigation" aria-expanded="true">
 <span class="menu-text">Basic Concepts</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./WhatEvolutionIs.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">What evolution is</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./HistoricaPerspective.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">A brief history of evolutionary thought</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./EvidenceForEvolution.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Evidence for Evolution</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./HowNaturalSelectionWorks.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">How Natural Selection Works</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" role="navigation" aria-expanded="true">
 <span class="menu-text">Working With Evolutionary Trees</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-2" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./UnderstandingEvolTrees.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Understanding Evolutionary Trees</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./EstimatingEvoTrees.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Estimating evolutionary trees</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" role="navigation" aria-expanded="true">
 <span class="menu-text">Evolutionary Genetics</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-3" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./GenesMutationGeneticVariation.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Genes, mutation, and genetic variation</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./IntroEvolGenetics.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">Introduction: Evolutionary genetics</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./MechEvolDrift.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">Genetic Drift</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./MechEvolSelection.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">10</span>&nbsp; <span class="chapter-title">Mechanisms that drive evolution: natural selection</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./MechEvolMutation.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">11</span>&nbsp; <span class="chapter-title">Mutation</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./MechEvolMigration.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">12</span>&nbsp; <span class="chapter-title">Migration and forces combined</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./EvolPhenotype.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text"><span class="chapter-number">13</span>&nbsp; <span class="chapter-title">The Evolution of Quantitative Traits</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-4" role="navigation" aria-expanded="true">
 <span class="menu-text">In Light Of Evolution</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-4" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-4" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./KinAndSexualSelection.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">14</span>&nbsp; <span class="chapter-title">Social Behavior and Sexual Selection</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./Adaptations.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">15</span>&nbsp; <span class="chapter-title">Adaptation and Phenotypic Plasticity</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./SpeciesSpeciation.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">16</span>&nbsp; <span class="chapter-title">Species and Speciation</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./EvolutionGenomicLevel.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">17</span>&nbsp; <span class="chapter-title">Evolution at the genomic level</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./EvoDevo.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">18</span>&nbsp; <span class="chapter-title">EvoDevo</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-5" role="navigation" aria-expanded="true">
 <span class="menu-text">Evolution At The Macro Level</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-5" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-5" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./EvoBiogeogrphy.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">19</span>&nbsp; <span class="chapter-title">Evolutionary Biogeography</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./DivLife.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">20</span>&nbsp; <span class="chapter-title">Diversification and patterns of life through time</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-6" role="navigation" aria-expanded="true">
 <span class="menu-text">What About Us</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-6" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-6" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./HumanEvol.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">21</span>&nbsp; <span class="chapter-title">Human Evolution</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./EvolMed.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">22</span>&nbsp; <span class="chapter-title">Evolutionary Medicine</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./EvolMedPartI.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">23</span>&nbsp; <span class="chapter-title">Evolutionary Medicine I: Aging and Diseases of Civilization</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./EvolMedPartII.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">24</span>&nbsp; <span class="chapter-title">Evolutionary Medicine II: Evolving Pathogens</span></span></a>
  </div>
</li>
      </ul>
  </li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" class="quarto-sidebar-collapse-item" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#making-the-connection-between-single-gene-mendelain-characters-and-complex-continuous-quantitative-traits." id="toc-making-the-connection-between-single-gene-mendelain-characters-and-complex-continuous-quantitative-traits." class="nav-link active" data-scroll-target="#making-the-connection-between-single-gene-mendelain-characters-and-complex-continuous-quantitative-traits."><span class="header-section-number">13.1</span> Making the connection between single gene Mendelain characters and complex, continuous quantitative traits.</a></li>
  <li><a href="#quantitative-traits-a-product-of-genes-and-environment" id="toc-quantitative-traits-a-product-of-genes-and-environment" class="nav-link" data-scroll-target="#quantitative-traits-a-product-of-genes-and-environment"><span class="header-section-number">13.2</span> Quantitative Traits: a Product of Genes and Environment</a></li>
  <li><a href="#further-complications-qualitative-traits-epistasis-and-pleitropy" id="toc-further-complications-qualitative-traits-epistasis-and-pleitropy" class="nav-link" data-scroll-target="#further-complications-qualitative-traits-epistasis-and-pleitropy"><span class="header-section-number">13.3</span> Further Complications: Qualitative Traits, Epistasis, and Pleitropy</a></li>
  <li><a href="#quantitative-trait-evolution" id="toc-quantitative-trait-evolution" class="nav-link" data-scroll-target="#quantitative-trait-evolution"><span class="header-section-number">13.4</span> Quantitative Trait Evolution</a>
  <ul class="collapse">
  <li><a href="#quantifying-trait-heritability" id="toc-quantifying-trait-heritability" class="nav-link" data-scroll-target="#quantifying-trait-heritability"><span class="header-section-number">13.4.1</span> Quantifying Trait Heritability</a></li>
  <li><a href="#quantifying-the-strength-of-selection" id="toc-quantifying-the-strength-of-selection" class="nav-link" data-scroll-target="#quantifying-the-strength-of-selection"><span class="header-section-number">13.4.2</span> Quantifying the Strength of Selection</a></li>
  <li><a href="#calculating-the-response-to-selection" id="toc-calculating-the-response-to-selection" class="nav-link" data-scroll-target="#calculating-the-response-to-selection"><span class="header-section-number">13.4.3</span> Calculating the Response to Selection</a></li>
  </ul></li>
  <li><a href="#modes-of-selection-on-quantitative-traits" id="toc-modes-of-selection-on-quantitative-traits" class="nav-link" data-scroll-target="#modes-of-selection-on-quantitative-traits"><span class="header-section-number">13.5</span> Modes of Selection on Quantitative Traits</a>
  <ul class="collapse">
  <li><a href="#directional-selection" id="toc-directional-selection" class="nav-link" data-scroll-target="#directional-selection"><span class="header-section-number">13.5.1</span> Directional Selection</a></li>
  <li><a href="#stabilizing-selection" id="toc-stabilizing-selection" class="nav-link" data-scroll-target="#stabilizing-selection"><span class="header-section-number">13.5.2</span> Stabilizing Selection</a></li>
  <li><a href="#disruptive-selection" id="toc-disruptive-selection" class="nav-link" data-scroll-target="#disruptive-selection"><span class="header-section-number">13.5.3</span> Disruptive Selection</a></li>
  </ul></li>
  <li><a href="#identifying-genes-underlying-quantitative-traits" id="toc-identifying-genes-underlying-quantitative-traits" class="nav-link" data-scroll-target="#identifying-genes-underlying-quantitative-traits"><span class="header-section-number">13.6</span> Identifying Genes Underlying Quantitative Traits</a>
  <ul class="collapse">
  <li><a href="#qtl-mapping" id="toc-qtl-mapping" class="nav-link" data-scroll-target="#qtl-mapping"><span class="header-section-number">13.6.1</span> QTL Mapping</a></li>
  <li><a href="#gwas" id="toc-gwas" class="nav-link" data-scroll-target="#gwas"><span class="header-section-number">13.6.2</span> GWAS</a></li>
  </ul></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default"><nav class="quarto-page-breadcrumbs quarto-title-breadcrumbs d-none d-lg-block" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./GenesMutationGeneticVariation.html">Evolutionary Genetics</a></li><li class="breadcrumb-item"><a href="./EvolPhenotype.html"><span class="chapter-number">13</span>&nbsp; <span class="chapter-title">The Evolution of Quantitative Traits</span></a></li></ol></nav>
<div class="quarto-title">
<h1 class="title"><span id="sec-evolpheno" class="quarto-section-identifier"><span class="chapter-number">13</span>&nbsp; <span class="chapter-title">The Evolution of Quantitative Traits</span></span></h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<p><em>by Dr.&nbsp;Michael Tobler with modifications by SSMc</em></p>
<p>So far, our evolutionary analyses have assumed that variation at one locus translates to variation in one trait. This, in turn, causes environmentally-dependent variation in fitness. As discussed in <a href="GenesMutationGeneticVariation.html" class="quarto-xref"><span>Chapter 7</span></a>, however, the relationships between genotype, phenotype, and fitness are much more complex. Single genes can affect the expression of multiple, apparently unrelated phenotypic traits (pleiotropy), which can impose evolutionary trade-offs, because different traits cannot be optimized independently. Similarly, the expression of a single trait can be controlled by multiple loci. This chapter explores basic quantitative genetic approaches used to study the evolution of complex traits controlled by multiple genes.</p>
<section id="making-the-connection-between-single-gene-mendelain-characters-and-complex-continuous-quantitative-traits." class="level2" data-number="13.1">
<h2 data-number="13.1" class="anchored" data-anchor-id="making-the-connection-between-single-gene-mendelain-characters-and-complex-continuous-quantitative-traits."><span class="header-section-number">13.1</span> Making the connection between single gene Mendelain characters and complex, continuous quantitative traits.</h2>
<p>At one point in the history of genetics and evolutionary biology, there was a strong disagreement between those researchers that enthusiastically accepted a Mendelian interpretation of inheritance, led by William Bateson and his students, and those that argued most characters did not fall into the types of discrete groups (like round versus wrinkled pea) studied by Mendel. Rather, most traits were continuous in nature and therefore not inherited in a Mendelian sense, a position championed by the likes of Francis Galton and Karl Pearson (both leading figures in the field of statistics) referred to as the Biometricians. Biometricians argued that Mendelian inheritance was wholely insufficient to explain inheritance because discrete units of heredity, what we call genes today, could not explain the continuous distribution of characters found in most populations. The Mendelians on the other hand argued that the type of variation studied by Biometricians could not account for the evolution of new species.</p>
<p>This ongoing debate continued to dominate the field of genetics, delaying the acceptance of natural selection as a key mechanism in evoution, until R.A. Fisher resolved it by demonstrating that these seemingly contradictory views were actually compatible. His key insight was that:</p>
<ul>
<li><p>The continuous variation observed by biometricians and the discrete inheritance patterns observed by Mendelians were produced by the same underlying process</p></li>
<li><p>Continuous traits with “infinitesimal individual-to-individual gradations” resulted from multiple Mendelian factors (genes) combining to affect a trait</p></li>
<li><p>Environmental factors further “smeared” genotypic categories into one another, creating continuous distributions</p></li>
</ul>
<p>Fisher showed that Mendelian inheritance was not a special case but rather the fundamental mechanism of heredity. The continuous variation championed by biometricians could be explained by multiple Mendelian factors working together, along with environmental influences. This synthesis reconciled the two perspectives and laid the groundwork for the modern understanding of quantitative genetics.</p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-1-contents" aria-controls="callout-1" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Graphical view of Fisher’s insights
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-1" class="callout-1-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>The figure below, taken directly from Futuyma and Kirkpatrick, demonstrates beautifully the connection between simple Mendelian characters and the more complex phenotypic characters studied by the Biometricians, as shown by Fisher. The phenotypic distribution is determined by both genetic and environmental factors. The vertical blue bars show the phe- notypes that would be produced without any environmental influence. The green curves show the phenotypic distributions for each of the genotypes that result with the addition of environmental variance. The red curve shows the phenotypic distribution for the entire population. Each locus has two alleles with frequencies <span class="math inline">\(0.5\)</span>, and the alleles have equal and additive effects on the trait (there is no dominance or epistasis). Notice that the phenotypic distributions become smoother when larger numbers of loci contribute to the trait and as the environmental variance increases.</p>
<div id="fig-quantvar" class="quarto-layout-panel">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-quantvar-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="quarto-layout-row">
<div class="quarto-layout-cell" style="flex-basis: 100.0%;justify-content: flex-start;">
<img src="images/BasisQuantVariation.png" class="img-fluid figure-img">
</div>
</div>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-quantvar-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;13.1: Image: Figure 6.3 from Futuyma and Kirkpatric (2017). This image is reproduced from the original source under the Fair Use Doctrine for educational/scholarly purposes.
</figcaption>
</figure>
</div>
</div>
</div>
</div>
</section>
<section id="quantitative-traits-a-product-of-genes-and-environment" class="level2" data-number="13.2">
<h2 data-number="13.2" class="anchored" data-anchor-id="quantitative-traits-a-product-of-genes-and-environment"><span class="header-section-number">13.2</span> Quantitative Traits: a Product of Genes and Environment</h2>
<p>Quantitative traits vary continuously among individuals—for example, body size, thermal tolerance, and skin coloration in humans. The expression of quantitative traits depends on the cumulative action of many genes, which is why they are also referred to as polygenic traits. Quantitative traits are normally distributed within a population; most individuals exhibit an intermediate phenotype, and individuals with extreme trait values are more rare.</p>
<p>The normal distribution of quantitative traits is caused by the joint action of many genes with additive genetic effects. The continuous variation can be revealed using a simple F2-crossing experiment that is frequently used in quantitative genetics analyses (see QTL mapping below). If we cross two individuals from either extreme of a distribution (<em>i.e.</em>, the shortest and the tallest, the lightest and the darkest, or the slowest and the fastest), the resulting F1 offspring should be phenotypically homogenous and intermediate to their parents, because the F1 individuals will be heterozygous at all loci involved in the expression of the quantitative trait. If we then cross two F1 individuals to produce an F2 generation, we not only recover the extremes of the distribution (<em>i.e.</em>, the parental phenotypes), but the entire spectrum of phenotypic variants in between. The gradation of intermediate phenotypes is dependent on the number of loci controlling a quantitative trait. If a trait is controlled by a single locus with additive alleles, there are three phenotypic categories (<em>e.g.</em>, short, medium, and tall; <a href="#fig-qualquan" class="quarto-xref">Figure&nbsp;<span>13.2</span></a>), left panel). If two loci are involved, trait expression is the sum of the effects of all individual alleles across both loci. In other words, how tall an individual grows is simply a function of the number of “tall” alleles it inherits across all loci. An intermediate size can be achieved by having two “tall” alleles at one locus (<span class="math inline">\(AAbb\)</span> or <span class="math inline">\(aaBB\)</span>) or one “tall” allele at each of the loci (<span class="math inline">\(AaBb\)</span>). As a consequence, there are more offspring of intermediate size than small or large ones (<a href="#fig-qualquan" class="quarto-xref">Figure&nbsp;<span>13.2</span></a>, central panel). The more genes are involved in the expression of a particular phenotype, the narrower phenotypic categories become, and the more continuously distributed a trait is (<a href="#fig-qualquan" class="quarto-xref">Figure&nbsp;<span>13.2</span></a>), right panel).</p>
<div class="cell">
<div class="cell-output-display">
<div id="fig-qualquan" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-qualquan-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="EvolPhenotype_files/figure-html/fig-qualquan-1.png" class="img-fluid figure-img" width="672">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-qualquan-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;13.2: Phenotypes in F2 crosses are normally distributed if all alleles have additive effects. These three graphs show how the number of phenotypic categories increases as a function of the number genes involved in the expression of a particular trait.
</figcaption>
</figure>
</div>
</div>
</div>
<p>The expression of quantitative traits can also be affected by the environment an individual finds itself in. The ability of a genotype to produce multiple phenotypes depending on the environment is called <strong>phenotypic plasticity</strong>.</p>
<div class="callout callout-style-default callout-important callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Definition: Phenotypic Plasticity
</div>
</div>
<div class="callout-body-container callout-body">
<p>Phenotypic plasticity is the ability of a genotype to express different phenotypes. It can result in organisms changing their traits in response to stimuli or input from the environment.</p>
</div>
</div>
<p>In some species, phenotypic traits show phenomenal malleability in response to specific environmental cues that shape organismal development. For example, some anuran tadpoles develop alternative morphological traits depending on their exposure to predator cues early in development (<a href="#fig-rana" class="quarto-xref">Figure&nbsp;<span>13.3</span></a>).</p>
<div id="fig-rana" class="quarto-layout-panel">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-rana-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="quarto-layout-row">
<div class="quarto-layout-cell" style="flex-basis: 100.0%;justify-content: flex-start;">
<img src="images/rana.jpg" class="img-fluid figure-img">
</div>
</div>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-rana-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;13.3: Phenotypic plasticity in <em>Rana pirica</em> tadpoles when under predation from salamander larvae: normal (left) and defended (right) morphs. Mori et al.&nbsp;(2009), <a href="https://creativecommons.org/licenses/by/3.0">CC BY 3.0</a>, via Wikimedia Commons.
</figcaption>
</figure>
</div>
<p>Population differences in phenotypic traits then are not always a consequence of evolutionary divergence and adaptation. Rather, individuals in different populations may express different traits simply due to different environmental exposure. For example, the Alpine plant <em>Arabis alpina</em> is distributed across a broad elevational gradient, ranging from 800 to 3,000+ meters above sea level. Individuals from high elevation populations tend to be smaller and produce less fruit. However, common garden experiments—where families were split and raised at different elevations—revealed that population differences in most traits were largely due to phenotypic plasticity (<a href="#fig-plastarabis" class="quarto-xref">Figure&nbsp;<span>13.4</span></a>; de Villemereuil et al.&nbsp;2018). Hence, the expression of quantitative traits can be more dependent on the environmental context than the genetic information inherited from the parents.</p>
<div class="cell">
<div class="cell-output-display">
<div id="fig-plastarabis" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-plastarabis-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="EvolPhenotype_files/figure-html/fig-plastarabis-1.png" class="img-fluid figure-img" width="672">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-plastarabis-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;13.4: Phenotypic plasticity in <em>Arabis alpina</em>. The graph shows mean trait values for different populations (in different colors) that were either grown at low or high elevation. Each population was composed of multiple families that were represented at both locations. Differences between locations are more pronounced than differences among populations, indicating the influence of environmentally-dependent phenotypic plasticity. <a href="data/8_plasticity.csv">Data</a> from de Villemereuil et al.&nbsp;(2018).
</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="further-complications-qualitative-traits-epistasis-and-pleitropy" class="level2" data-number="13.3">
<h2 data-number="13.3" class="anchored" data-anchor-id="further-complications-qualitative-traits-epistasis-and-pleitropy"><span class="header-section-number">13.3</span> Further Complications: Qualitative Traits, Epistasis, and Pleitropy</h2>
<p>Qualitative traits have alternative phenotypes that fit into discrete categories. The spotted vs.&nbsp;melanistic forms of leopards, the round vs.&nbsp;wrinkled skin in peas, and the ABO blood groups in humans are all examples of qualitative traits. Many qualitative traits are controlled by variation at a single locus, but multiple loci can be involved when there are epistatic interactions. Epistasis describes the phenomenon where the effect of a gene is dependent on the effect of one or more modifier genes. For example, coat color variation in many mammals (including yellow, chocolate, and black Labrador retrievers) is dependent on both the ability of an individual to produce a particular color pigment (controlled by the <em>B</em> locus in <a href="#fig-epistasis" class="quarto-xref">Figure&nbsp;<span>13.5</span></a>) and the ability of an individual to deposit pigments in relevant tissues (controlled by the <em>A</em> locus). Variation at the <em>B</em> locus that determines brown vs.&nbsp;black phenotype has no effect on coat color unless an individual also inherits the ability to deposit pigment; otherwise the coat color is just white. Epistatic interactions can profoundly impact evolutionary dynamics at single loci and cause deviations from the predictions of the simple inhertance models that we have used so far.</p>
<div id="fig-epistasis" class="quarto-layout-panel">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-epistasis-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="quarto-layout-row">
<div class="quarto-layout-cell" style="flex-basis: 100.0%;justify-content: flex-start;">
<img src="images/epistasis.png" class="img-fluid figure-img">
</div>
</div>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-epistasis-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;13.5: Example of epistasis in coat color genetics: If no pigments can be deposited, the other coat color genes have no effect on color expression, no matter if they would be dominant or recessive, and no matter if the individual is homozygous.
</figcaption>
</figure>
</div>
<p>Likewise <strong>pleiotropy</strong>, where a gene can affect the expression of multiple, apparently unrelated phenotypic traits, can also complicate the genotype-pheontype connection. This can get a little complicated since there can be a complex interaction among alleles at different loci (eipstasis), the environment, and impact of selection acting on other phenotypic characters that a particular gene effects. Pleiotropic gene action can limit the rate of multivariate evolution when selection on one trait favors one allele, while selection on other traits favors a different allele, which can be dependent on the environment. This creates evolutionary trade-offs that can constrain adaptation. For the purpose of this text, even though pleitropy can be extremely important, and more than likely most if not all genes express some form of pleitropy, let’s just leave this as a “black box” without opening that box up too widely.</p>
<p>In conclusion on this section, the evolutionary analysis of quantitative traits has two complications: (1) Since many genes are involved in the expression of quantitative traits, it is not usually practical to follow changes in allele frequencies at relevant loci. Even worse, due to the large number of loci that often have small individual effects, even the identification of genes contributing to the expression of quantitative traits can be challenging. (2) We must be able to account for the potential effects of environmental variation on individual trait expression. Trait heritability is a key requirement for evolution by natural selection, but expressed traits may not be heritable if they were shaped by environmental effects. In the following section, we will explore two general approaches used to study the evolution of quantitative traits. One approach is agnostic about the genetic basis of quantitative traits but focuses on predicting the effects of selection on them. The other approach allows for the identification of the genetic basis of quantitative traits involved in adaptation.</p>
</section>
<section id="quantitative-trait-evolution" class="level2" data-number="13.4">
<h2 data-number="13.4" class="anchored" data-anchor-id="quantitative-trait-evolution"><span class="header-section-number">13.4</span> Quantitative Trait Evolution</h2>
<p>A key complication in the analysis of quantitative traits is the difficulty of identifying the genes involved in phenotypic expression. However, we can conduct meaningful analyses of quantitative trait evolution without diving into the details of the underlying genetics. We can predict the evolutionary response of a quantitative trait as a function of its heritability and the strength of selection. In this section, we will discuss how to quantify trait heritability and the strength of selection, and then how to apply the breeder’s equation to predict evolutionary change.</p>
<section id="quantifying-trait-heritability" class="level3" data-number="13.4.1">
<h3 data-number="13.4.1" class="anchored" data-anchor-id="quantifying-trait-heritability"><span class="header-section-number">13.4.1</span> Quantifying Trait Heritability</h3>
<p>Generally, heritability measures the degree to which variation in a phenotypic trait in a population is due to genetic variation among individuals, rather than non-heritable factors like phenotypic plasticity. Specifically, <em>broad sense heritability</em> (<span class="math inline">\(H^2\)</span>) is defined as:</p>
<p><span id="eq-38"><span class="math display">\[
\begin{align}
\frac{V_G}{V_P}
\end{align}
\tag{13.1}\]</span></span></p>
<p>where <span class="math inline">\(V_G\)</span> is genetic variation and <span class="math inline">\(V_P\)</span> is phenotypic variation. Since phenotypic variation among individuals is a consequence of genetic and environmental variation (<span class="math inline">\(V_E\)</span>), broad sense heritability can also be defined as:</p>
<p><span id="eq-39"><span class="math display">\[
\begin{align}
H^2=\frac{V_G}{V_G+V_E}
\end{align}
\tag{13.2}\]</span></span></p>
<p>Broad-sense heritability describes the degree of genetic determination in the variation of a trait, and it can range from 0 (no genetic effects) to 1 (no environmental effects). It is important to note that measures of heritability are a population-level metric. We cannot disentangle the degree to which genes and environmental influences have shaped trait expression in a particular individual. Rather, we have to consider populations of individuals to determine what proportion of variation is shaped by genetic variation and what proportion is shaped by environmental variation.</p>
<p>As you learned in <a href="HowNaturalSelectionWorks.html" class="quarto-xref"><span>Chapter 4</span></a>, heritability can be estimated by the slope of a regression between parental and offspring traits. If the slope of that regression is close to zero, parental trait expression is a poor predictor of offspring trait expression; hence, variation in the trait is primarily due to environmental and not genetic variation (Figure @ref(fig:herito)A). If, on the other hand, the slope of the parent-offspring regression is close to 1, the parental trait strongly predicts offspring trait expression, and variation in the traits is primarily driven by genetic variation (Figure @ref(fig:herito)C). Slopes between 0 and 1 describe the relative contribution of genetic and environmental variation to phenotypic variation in a population (Figure @ref(fig:herito)B).</p>
<div class="cell">
<div class="cell-output-display">
<div id="fig-herito" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-herito-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="EvolPhenotype_files/figure-html/fig-herito-1.png" class="img-fluid figure-img" width="672">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-herito-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;13.6: Parent-offspring regressions reveal the degree of heritability in quantitative traits. A slope of zero indicates no heritability; a slope of 1 indicates perfect heritability. Slopes in between those values describe the relative contributions of genetic and environmental variation on phenotypic variation.
</figcaption>
</figure>
</div>
</div>
</div>
<p>It is important to note that estimating heritability through parent-offspring regression represents <em>narrow-sense heritability</em> (<span class="math inline">\(h^2\)</span>), not broad-sense heritability . Broad- and narrow-sense heritability consider different aspects of genetic variation. Genetic variation (<span class="math inline">\(V_G\)</span>) is a composite of additive genetic variation (<span class="math inline">\(V_A\)</span>) and genetic variation caused by gene interactions, such as dominance (<span class="math inline">\(V_D\)</span>) and epistasis (<span class="math inline">\(V_I\)</span>). The slope of a parent-offspring correlation specifically quantifies the contribution of additive genetic variation to the observed phenotypic variation, such that narrow sense heritability is defined as:</p>
<p><span id="eq-40"><span class="math display">\[
\begin{align}
h^2 = \frac{V_A}{V_P}=\frac{V_A}{V_A+V_D+V_I+V_E}
\end{align}
\tag{13.3}\]</span></span></p>
<p>Hence, narrow-sense heritability only considers the additive portion of adaptive variation; genetic variation associated with dominance and epistasis are ignored. This is useful because the phenotypic response to selection on a quantitative trait depends on <span class="math inline">\(V_A\)</span> and not <span class="math inline">\(V_D\)</span>. In addition, non-additive genetic effects are comparatively rare, and <span class="math inline">\(V_A\)</span> is much larger than <span class="math inline">\(V_D\)</span> and <span class="math inline">\(V_I\)</span> for most quantitative traits (Hill et al.&nbsp;2008).</p>
<div id="imp-heredlimits" class="callout callout-style-default callout-important callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-3-contents" aria-controls="callout-3" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Important&nbsp;13.1: Limitations to estimates of heretiability
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-3" class="callout-3-contents callout-collapse collapse">
<section id="limitations-and-common-misconceptions-about-interpreting-heritability" class="level4 callout-body-container callout-body" data-number="13.4.1.1">
<h4 data-number="13.4.1.1" class="anchored" data-anchor-id="limitations-and-common-misconceptions-about-interpreting-heritability"><span class="header-section-number">13.4.1.1</span> Limitations and common misconceptions about interpreting heritability</h4>
<p><strong>Limitations based on experimental considerations</strong> From an experimental perspective, estimates of <span class="math inline">\(h^2\)</span> can be notoriously difficult in natural populations, let alone human populations. Some of the problems: - <strong>Covariation between genotype and environment</strong>, <em>e.g.</em> offspring raised in the same environment as parents: An optimal of accounting for this covariation would be randomly distribute all offspring among partent groups such that they are potential raised in an environment different from their parents. Obviously this is impossible or exceedingly difficult for some species (let alone immoral, like say humans). - <strong>Different environments for the different mating pairs:</strong> Differences in environmental quality, nutrition, stress, or temperature to just name a few environmental variances may impact the offspring genotypes, inflating the average <span class="math inline">\(V_E\)</span> - <strong>Other complicating issues</strong> like maternal effects, non-independence between number of offspring and phenotype, interactions among offspring, plus numreous other confounding variables. - <strong>Difficulty in determining parentage:</strong> Even when identifying parents seems obvious, extra-pair matings can occur and be difficult to determine (though molecular and genomic data can help). - Currently methods for identifying kinship in natural populations using genomic data has a lot of variance around the estimates, so identifying parent-offspring relationships from pooled population data, though incredibly useful, has it’s limitations today</p>
<p><strong>Limitations based on interpretations of</strong> <span class="math inline">\(h^2\)</span> Understanding what <span class="math inline">\(h^2\)</span> actually means can prevent misinterpretations. <span class="math inline">\(h^2\)</span> estimates can be useful in predicting the response to selection, but they can be (and have been) abused an misunderstood - <span class="math inline">\(h^2\)</span> estimates say little about the causes of differences <em>among individuals</em> within a population; just because a character has a relatively high <span class="math inline">\(h^2\)</span> does not imply that individuals with low values of the character have “poor genes”. - <span class="math inline">\(h^2\)</span> says nothing about the genetic basis of the character, for example how many genes contribute to a character or how theose genes interact to determine the phenotype. - <span class="math inline">\(h^2\)</span> estimates say little about the causes of differences among populations if the populations are from different environments. The only way to determine the cause of differences among populations is through proper experimentation</p>
<p><strong>Mischaracterizations of levels of</strong> <span class="math inline">\(h^2\)</span> - A <span class="math inline">\(h^2\approx0\)</span> does not mean there is no genetic basis to a character. All it means is that there may be very little variation in the genes that controbute to the character. Some traits may have a large genetic basis, but very little variation in a population (<em>e.g.</em> presence of two eyes in humans). These traits are still determined by genes, however since there is no variation among individuals, the heritability would be zero. - <span class="math inline">\(h^2\)</span> applies only to a particular population in a particular environment at a particular time. Using estimates of <span class="math inline">\(h^2\)</span> from a single population to infer the genetic basis of a character in other populations or other times should be used with caution. - Two populations with the same genetic variance may have very different heritabilities if <em>e.g.</em> one population has a higher environmental variance than the other</p>
<p>Nonetheless, despite these limitations, estimates of <span class="math inline">\(h^2\)</span> are invaluable in determining whether a character has a genetic basis, and if so what is the extent of the parent-offspring similarity, which is essential to determine if changes within and among populations are due to selection or some other mechanism.</p>
</section>
</div>
</div>
</section>
<section id="quantifying-the-strength-of-selection" class="level3" data-number="13.4.2">
<h3 data-number="13.4.2" class="anchored" data-anchor-id="quantifying-the-strength-of-selection"><span class="header-section-number">13.4.2</span> Quantifying the Strength of Selection</h3>
<p>The second step in quantitative genetic analyses is to infer the strength of selection on a focal trait, which can be measured as either the selection differential (<span class="math inline">\(S\)</span>) or the selection gradient (<span class="math inline">\(m\)</span>). The selection differential is the difference between the mean trait value of the selected (successful) individuals (<span class="math inline">\(T^*\)</span>) and the mean trait value of all individuals in a population (<span class="math inline">\(T^{'}\)</span>). Imagine a population of individuals that vary in a particular trait, as described in <a href="#fig-seldiffgrad" class="quarto-xref">Figure&nbsp;<span>13.7</span></a> Figure A. Some individuals survive to adulthood and reproduce (depicted in green), some perish early and have no reproductive success (depicted in red). The difference between the average trait value of the successful individuals and the average of all individuals is the selection differential. Note that if more individuals with smaller trait values were successful, the selection differential would shrink. If only individuals with smaller trait values were successful, the selection differential would become negative. Also note that the unit of the selection differential for a particular trait is the same as the unit used to quantify variation in that trait.</p>
<p>The second measure of the strength of selection on quantitative traits—the selection gradient—is more complicated, but has broader applicability, especially when fitness is measured quantitatively rather than qualitatively. The first step is to convert measurements of absolute fitness to measures of relative fitness. This can be accomplished by dividing every individual’s absolute fitness by the average fitness of the population. Using the same example as in <a href="#fig-seldiffgrad" class="quarto-xref">Figure&nbsp;<span>13.7</span></a>, we had 27 successful individuals (fitness of 1) and 73 unsuccessful individuals (fitness of zero). Hence, the average fitness in the population is <span class="math inline">\(\frac{27}{(27+73)}= 0.27\)</span>, which means that the relative fitness of successful individuals is <span class="math inline">\(\frac{1}{0.27}\approx3.7\)</span>. The relative fitness of unsuccessful individuals is of course still <span class="math inline">\(\frac{0}{0.27} = 0\)</span>. We can now plot relative fitness as a function of individual trait values. The selection gradient is the slope of the regression line between trait value and relative fitness <a href="#fig-seldiffgrad" class="quarto-xref">Figure&nbsp;<span>13.7</span></a> Figure B).</p>
<p>While the selection differential and the selection gradient may seem unrelated to each other, the two measures can actually be converted:</p>
<p><span id="eq-41"><span class="math display">\[
\begin{align}
m=\frac{S}{var(T)} \iff S=m*var(T)
\end{align}
\tag{13.4}\]</span></span></p>
<p>whereas <span class="math inline">\(var(T)\)</span> is the variance in the trait under consideration. In practice, if the strength of selection in quantitative genetic analyses is quantified using the selection gradient, it has to be converted to the selection differential, because the response to selection is proportional to <span class="math inline">\(S\)</span>.</p>
<div class="cell">
<div class="cell-output-display">
<div id="fig-seldiffgrad" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-seldiffgrad-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="EvolPhenotype_files/figure-html/fig-seldiffgrad-1.png" class="img-fluid figure-img" width="672">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-seldiffgrad-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;13.7: A. Graphical representation of the selection differential (<em>S</em>), which is the difference between the selected individuals (*T**) and all individuals in the population (<em>T’</em>). The gray triangle represents the average trait value of all individuals, and the green triangle the average of all the successful individuals. The distance between the two is the selection differential. B. Graphical representation of the selection gradient (<em>m</em>). The selection gradient is the slope of the best-fit line between individual trait values and relative fitness.
</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="calculating-the-response-to-selection" class="level3" data-number="13.4.3">
<h3 data-number="13.4.3" class="anchored" data-anchor-id="calculating-the-response-to-selection"><span class="header-section-number">13.4.3</span> Calculating the Response to Selection</h3>
<p>The final step in quantitative genetic analyses is to calculate the response to selection (<span class="math inline">\(R\)</span>) as a function of narrow-sense heritability (<span class="math inline">\(h^2\)</span>) and the selection differential (<span class="math inline">\(S\)</span>) using the breeder’s equation:</p>
<p><span id="eq-42"><span class="math display">\[
\begin{align}
R=h^2*S
\end{align}
\tag{13.5}\]</span></span></p>
<p>As you can see, the response to selection is dependent equally on the strength of selection and the degree to which a particular trait is heritable. If selection is weak, the evolutionary response will be small even when trait heritability is very high. Similarly, when heritability is low, the evolutionary response will be small even when selection is very strong. The principle of the breeder’s equation is also depicted in Figure <span class="citation" data-cites="figbreeder">(<a href="#ref-figbreeder" role="doc-biblioref"><strong>figbreeder?</strong></a>)</span>, a modified version of a parent-offspring regression. In this graph, successful parents are represented in green, and unsuccessful ones in red. As in <a href="#fig-seldiffgrad" class="quarto-xref">Figure&nbsp;<span>13.7</span></a> Figure A, the selection differential (<span class="math inline">\(S\)</span>) is the difference between the average trait of the successful parents and the average trait of all parents in the population. The response to selection (<span class="math inline">\(R\)</span>) is the difference between the average trait of the successful offspring and the average trait of all offspring in the population. Or, expressed mathematically:</p>
<p><span id="eq-43"><span class="math display">\[
\begin{align}
h^2=\frac{R}{S}=\frac{O^*-\overline{O}}{P^*-\overline{P}}
\end{align}
\tag{13.6}\]</span></span></p>
<p>You can imagine how <span class="math inline">\(R\)</span> will increase in this example as the slope of the best-fit line increases, reaching <span class="math inline">\(S\)</span> when <span class="math inline">\(h^2 = 1\)</span>. Similarly, <span class="math inline">\(R\)</span> will decrease as you imagine the slope of the regression line decreasing, eventually reaching 0 when <span class="math inline">\(h^2 = 0\)</span>.</p>
<p>Ultimately, the response to selection represents the per generation change in the average value of a population’s quantitative trait. Like the selection differential, it is measured in the same unit used to quantify trait variation. Together, the quantification of narrow-sense heritability, the selection differential, and the application of the breeder’s equation provide a critical tool for studying the evolution of polygenic traits, without the need to identify genes underlying phenotypic trait expression.</p>
<div class="cell">
<div class="cell-output-display">
<div id="fig-breeder" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-breeder-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="EvolPhenotype_files/figure-html/fig-breeder-1.png" class="img-fluid figure-img" width="672">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-breeder-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;13.8: The response to selection (<em>R</em>) is proportional to the product of th selection differential (<em>S</em>) and narrow-sense heritability (as indicated by the slope of the best-fit line).
</figcaption>
</figure>
</div>
</div>
</div>
</section>
</section>
<section id="modes-of-selection-on-quantitative-traits" class="level2" data-number="13.5">
<h2 data-number="13.5" class="anchored" data-anchor-id="modes-of-selection-on-quantitative-traits"><span class="header-section-number">13.5</span> Modes of Selection on Quantitative Traits</h2>
<p>In our consideration of quantitative trait evolution, we have assumed that individuals at one end of the phenotypic spectrum exhibit a fitness advantage over individuals at the other end of the phenotypic spectrum. However, fitness functions can have any shape. In this section, we discuss three modes of selection on quantitative traits and contrast their impact on phenotypic variation across generations.</p>
<section id="directional-selection" class="level3" data-number="13.5.1">
<h3 data-number="13.5.1" class="anchored" data-anchor-id="directional-selection"><span class="header-section-number">13.5.1</span> Directional Selection</h3>
<p>Under directional selection, fitness continuously increases (or decreases) with the value of a trait; hence, extreme phenotypes at one end of the spectrum have a fitness advantage over individuals at the other end of the spectrum. Fitness functions describing directional selection are often assumed to be linear (<a href="#fig-dirsel" class="quarto-xref">Figure&nbsp;<span>13.9</span></a>), but in practice, nonlinear relationships (<em>i.e.</em>, accelerating or decelerating curves) can also describe fitness variation under directional selection. The consequence of directional selection is an increase (or decrease) in the average trait value of the population (<a href="#fig-dirsel" class="quarto-xref">Figure&nbsp;<span>13.9</span></a>). In addition, directional selection leads to a slight decrease in trait variation across generations. We have already learned about many examples of directional selection throughout the semester. The change in beak size of Darwin’s finches during the drought was a consequence of directional selection, just as decrease in eye size in some cave fish populations.</p>
<div class="cell">
<div class="cell-output-display">
<div id="fig-dirsel" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-dirsel-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="EvolPhenotype_files/figure-html/fig-dirsel-1.png" class="img-fluid figure-img" width="672">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-dirsel-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;13.9: Under directional selection, individuals at one end of the phenotypic spectrum have a fitness advantage over individuals at the other end of the phenotypic spectrum. As a consequence, trait distributions after selection exhibit a higher mean (red triange) and slightly reduced variation (red bar).
</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="stabilizing-selection" class="level3" data-number="13.5.2">
<h3 data-number="13.5.2" class="anchored" data-anchor-id="stabilizing-selection"><span class="header-section-number">13.5.2</span> Stabilizing Selection</h3>
<p>Under stabilizing selection, individuals with intermediate traits have the highest fitness, while individuals with extreme trait values have a selective disadvantage. Consequently, fitness functions under stabilizing selection have some sort of a hump-shaped curve (<a href="#fig-stabsel" class="quarto-xref">Figure&nbsp;<span>13.10</span></a>). Unlike directional selection, stabilizing selection does not lead to changes in the average trait value in a population. However, it reduces trait variation across generations, because extreme phenotypes counter-selected. In natural populations, most traits are probably under stabilizing selection. Once a trait has reached an optimum in a population, selection will keep that trait at said optimum, removing variants that deviate from it.</p>
<div class="cell">
<div class="cell-output-display">
<div id="fig-stabsel" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-stabsel-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="EvolPhenotype_files/figure-html/fig-stabsel-1.png" class="img-fluid figure-img" width="672">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-stabsel-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;13.10: Under stabilizing selection, intermediate individuals exhibit a fitness advantage, which does not change the population trait average (red triangle), but it causes a decrease in trait variation across generations (represented by the red bar).
</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="disruptive-selection" class="level3" data-number="13.5.3">
<h3 data-number="13.5.3" class="anchored" data-anchor-id="disruptive-selection"><span class="header-section-number">13.5.3</span> Disruptive Selection</h3>
<p>Under disruptive selection, intermediate individuals have a fitness disadvantage over extreme phenotypes at either end of the spectrum. Hence, fitness functions under disruptive selection have a u-shaped curve (<a href="#fig-dissel" class="quarto-xref">Figure&nbsp;<span>13.11</span></a>). Disruptive selection does not change the average trait value in a population, but it increases trait variation across generations and is an important mechanism for the maintenance of genetic variation. If disruptive selection persists for multiple generations, populations can exhibit bimodal trait distributions.</p>
<p>Disruptive selection underlies many intraspecific polymorphisms where individuals of the same species are adapted to different habitats or resources. For example, the light and dark forms of the peppered moth that are well-camouflaged on either light or dark backgrounds, or the different beak morphs in seedcrackers that exploit different food resources, are a consequence of disruptive selection. In addition, disruptive selection can cause speciation; the stark beak size differences among species of Darwin’s finches are caused by disruptive selection.</p>
<div class="cell">
<div class="cell-output-display">
<div id="fig-dissel" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-dissel-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="EvolPhenotype_files/figure-html/fig-dissel-1.png" class="img-fluid figure-img" width="672">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-dissel-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;13.11: Under disruptive selection, intermediate individuals have a fitness disadvantage. As a consequence, trait variation is increased across generations (red bars) but the mean trait value (red triangle) does not change.
</figcaption>
</figure>
</div>
</div>
</div>
</section>
</section>
<section id="identifying-genes-underlying-quantitative-traits" class="level2" data-number="13.6">
<h2 data-number="13.6" class="anchored" data-anchor-id="identifying-genes-underlying-quantitative-traits"><span class="header-section-number">13.6</span> Identifying Genes Underlying Quantitative Traits</h2>
<p>While basic quantitative genetic analyses can ignore the genes underlying adaptation, a key goal in evolutionary biology is to bridge the genotype-phenotype gap. We aim to identify adaptive genetic variation and link it to the expression of phenotypic traits. Hence, identifying the genetic basis of quantitative traits involved in adaptation is a key objective of many evolutionary studies. Detecting genes that contribute to the expression of quantitative traits can be challenging—especially when there are many genes that impact a particular trait, when the average effect of individual loci is small, and when sample sizes for quantitative analyses are low. Here, I will briefly introduce two general approaches that can be used to identify the genes associated with quantitative traits: quantitative trait locus (QTL) mapping and genome-wide association studies (GWAS).</p>
<section id="qtl-mapping" class="level3" data-number="13.6.1">
<h3 data-number="13.6.1" class="anchored" data-anchor-id="qtl-mapping"><span class="header-section-number">13.6.1</span> QTL Mapping</h3>
<p>Quantitative trait loci (QTLs) are stretches of DNA that are correlated with variation in a phenotypic trait. QTLs are linked to specific genes that contribute to variation in a particular phenotypic trait. They can be identified through QTL mapping, which hinges on genotyping and phenotyping large numbers of individuals derived from experimental crosses (also called a mapping population). To generate a mapping population for a trait of interest, two individuals from the opposite extremes of a phenotypic spectrum are chosen. Let’s assume we are studying a species of fish with populations living under different environmental conditions. The fish are blue in some habitats and orange in others, and we want to identify the genetic basis of this color variation. In this case, we would cross a blue and an orange individual to produce an F1 generation. Individuals in the F1 generation are phenotypically intermediate between the two parents, because they are heterozygous at loci relevant for the expression of the quantitative trait (<a href="#fig-qtl1" class="quarto-xref">Figure&nbsp;<span>13.12</span></a>). To create a mapping population for QTL analysis, we need to produce at least an F2 cross by mating two F1 individuals with each other. Thanks to recombination, F2 individuals inherit a varying mixture of genetic material from either parental individual of the initial cross, and accordingly, some individuals will be entirely blue, some entirely orange, and others may exhibit any color mixture in between. With each generation of crossing (F3, F4, F5, etc.), genomic segments are reshuffled and the size of linkage blocks (contiguous stretches of DNA that come from one parental individual) gets smaller and smaller (<a href="#fig-qtl1" class="quarto-xref">Figure&nbsp;<span>13.12</span></a>). We can then use this mapping population to ask which genomic regions are correlated with the phenotype of interest: at what loci do the genotypes match the observed phenotypic expression? To address this question, we need to quantify the phenotype of individuals in our mapping population, and we need to identify the genotypes of the same individuals at as many loci in the genome as feasible.</p>
<div id="fig-qtl1" class="quarto-layout-panel">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-qtl1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="quarto-layout-row">
<div class="quarto-layout-cell" style="flex-basis: 100.0%;justify-content: flex-start;">
<img src="images/qtl.png" class="img-fluid figure-img">
</div>
</div>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-qtl1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;13.12: General approach for the creation of a mapping population for QTL analysis. Individuals from opposite ends of the phenotypic spectrum are crossed to obtain an F1 generation, which is then crossed with itself to produce F2 and later (Fx) crosses. Repeated crossing reshuffles genomic segments, which allows for the identification of correlations between genotype and phenotype. For example, the genomic region highlighted with the arrow shows a clear match between genotype and phenotype for each individual; i.e., this locus would be a QTL.
</figcaption>
</figure>
</div>
<p>Genotypic and phenotypic analyses create a data matrix where information for different individuals is organized in rows, and information about phenotypic traits and the genotype at particular loci is organized in columns (<a href="#fig-qtl2" class="quarto-xref">Figure&nbsp;<span>13.13</span></a>). This allows us to directly compare the phenotypic traits of individuals with their genotypes across loci. As you can see in the example in <a href="#fig-qtl2" class="quarto-xref">Figure&nbsp;<span>13.13</span></a>, there is no congruence between phenotype and genotype at loci <span class="math inline">\(A\)</span>, <span class="math inline">\(B\)</span>, and <span class="math inline">\(C\)</span>. However, there is a clear correlation between phenotype and the genotype at locus <span class="math inline">\(D\)</span>: all blue individuals have the <span class="math inline">\(DD\)</span> genotype, all orange individuals have the <span class="math inline">\(dd\)</span> genotype, and all brown individuals are heterozygous. Hence, locus D would be a QTL for the color variation observed in our mapping population.</p>
<p>In practice, correlations between phenotypes and genotypes are evaluated for all loci in the dataset, and the degree of association between a phenotype and the genotype at a particular locus is quantified with an LOD score (LOD stands for logarithm of odds). The statistical details are not that important at this stage, but a higher LOD score indicates a stronger association between a particular locus and the phenotype of interest. An LOD score of 3 is equivalent to a <span class="math inline">\(p\)</span>-value of 0.05; so we consider LOD scores greater than 3 as significant associations between a locus and a phenotype. Graphically, LOD scores are plotted as function of the location of the loci on a chromosome. You can imagine the chromosomes with their loci lined up along the x-axis, and LOD-score peaks above 3 are indicative of genomic regions containing a QTL (<a href="#fig-qtl2" class="quarto-xref">Figure&nbsp;<span>13.13</span></a>).</p>
<div id="fig-qtl2" class="quarto-layout-panel">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-qtl2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="quarto-layout-row">
<div class="quarto-layout-cell" style="flex-basis: 100.0%;justify-content: flex-start;">
<img src="images/qtl2.png" class="img-fluid figure-img">
</div>
</div>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-qtl2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;13.13: Conceptual illustration of the results from a QTL experiment. The gray box shows the data structure, with information from different individuals organized in rows and phenotypic measurements and genotypes at different loci in columns. The graph above shows a qualitative representation of the correlations between genotype and phenotype in the matrix below.
</figcaption>
</figure>
</div>
</section>
<section id="gwas" class="level3" data-number="13.6.2">
<h3 data-number="13.6.2" class="anchored" data-anchor-id="gwas"><span class="header-section-number">13.6.2</span> GWAS</h3>
<p>QTL mapping is a powerful approach for identifying genomic regions that are associated with quantitative traits. However, the approach also comes with obvious limitations: large-scale crossing experiments are not possible in many species for which we may want to investigate the genetic basis of adaptive trait variation. This is especially true if we want to better understand genotype-phenotype relationships in humans. Understanding the association between genetic variation and phenotypic traits in humans not only teaches us about our nature, but has profound implication for the prediction and treatment of infectious and noncommunicable diseases.</p>
<p>GWAS (genome-wide association study) is an approach to identify loci underlying complex traits that does not hinge on the availability of crosses between contrasting phenotypes. Rather than looking for correlations between trait expression and the genotypes at specific loci, GWAS contrasts allele frequencies between different groups of individuals. Imagine, for example, that we want to identify genes associated with the susceptibility to a particular disease. In this case, we would compare allele frequencies in individuals that have contracted that disease (cases) with allele frequencies of a random sample of individuals from the general population (controls). For loci that do not impact disease susceptibility, we do not expect significant differences in allele frequencies between cases and controls; the likelihood that a case or a control has a particular allele is equal to the allele frequency in the population. However, if there is a significant difference in the allele frequencies at a particular locus, it indicates that genetic variation at that locus is potentially associated with becoming a case (<em>i.e.</em>, contracting the disease). Like QTL mapping, GWAS requires extensive genotyping, both in terms of the number of loci in the genome and the number of individuals in the case and control groups. GWAS results are plotted similarly to QTL results, with the location of the genetic marker on the x-axis. Instead of LOD scores, GWAS plot the -log<sub>10</sub> of the <span class="math inline">\(p\)</span>-value associated with allele frequency differences at each locus, leading to peaks at loci showing significant associations with the trait of interest.</p>


</section>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="./MechEvolMigration.html" class="pagination-link" aria-label="Migration and forces combined">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">12</span>&nbsp; <span class="chapter-title">Migration and forces combined</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="./KinAndSexualSelection.html" class="pagination-link" aria-label="Social Behavior and Sexual Selection">
        <span class="nav-page-text"><span class="chapter-number">14</span>&nbsp; <span class="chapter-title">Social Behavior and Sexual Selection</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->




</body></html>