<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.5.57">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>21&nbsp; Human Evolution – Evolution 303</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<link href="./EvolMed.html" rel="next">
<link href="./DivLife.html" rel="prev">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


</head>

<body class="nav-sidebar floating">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" role="button" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
        <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./HumanEvol.html">What About Us</a></li><li class="breadcrumb-item"><a href="./HumanEvol.html"><span class="chapter-number">21</span>&nbsp; <span class="chapter-title">Human Evolution</span></a></li></ol></nav>
        <a class="flex-grow-1" role="navigation" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
        </a>
      <button type="button" class="btn quarto-search-button" aria-label="Search" onclick="window.quartoOpenSearch();">
        <i class="bi bi-search"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal quarto-sidebar-collapse-item sidebar-navigation floating overflow-auto">
    <div class="pt-lg-2 mt-2 text-left sidebar-header">
    <div class="sidebar-title mb-0 py-0">
      <a href="./">Evolution 303</a> 
    </div>
      </div>
        <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
        </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Evolution 303</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./Preface.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Preface - Evolution 303</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="./IntroBasicConcepts.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Introduction: Basic Concepts</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./WhatEvolutionIs.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">What evolution is</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./HistoricaPerspective.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">A brief history of evolutionary thought</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./EvidenceForEvolution.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Evidence for Evolution</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./HowNaturalSelectionWorks.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">How Natural Selection Works</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="./EvolTreesAndPhylogenetics.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Evolutionary Trees and Phylogenetics</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-2" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./UnderstandingEvolTrees.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Understanding Evolutionary Trees</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./EstimatingEvoTrees.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Estimating evolutionary trees</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="./EvolGenetics.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Evolutionary Genetics</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-3" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./GenesMutationGeneticVariation.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Genes, mutation, and genetic variation</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./IntroEvolGenetics.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">Introduction: Evolutionary genetics</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./MechEvolDrift.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">Genetic Drift</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./MechEvolSelection.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">10</span>&nbsp; <span class="chapter-title">Mechanisms that drive evolution: natural selection</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./MechEvolMutation.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">11</span>&nbsp; <span class="chapter-title">Mutation</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./MechEvolMigration.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">12</span>&nbsp; <span class="chapter-title">Migration and other factors</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./EvolPhenotype.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">13</span>&nbsp; <span class="chapter-title">The Evolution of Quantitative Traits</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-4" role="navigation" aria-expanded="true">
 <span class="menu-text">In Light Of Evolution</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-4" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-4" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./KinAndSexualSelection.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">14</span>&nbsp; <span class="chapter-title">Social Behavior and Sexual Selection</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./Adaptations.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">15</span>&nbsp; <span class="chapter-title">Adaptation and Phenotypic Plasticity</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./SpeciesSpeciation.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">16</span>&nbsp; <span class="chapter-title">Species and Speciation</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./EvolutionGenomicLevel.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">17</span>&nbsp; <span class="chapter-title">Evolution at the genomic level</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./EvoDevo.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">18</span>&nbsp; <span class="chapter-title">EvoDevo</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-5" role="navigation" aria-expanded="true">
 <span class="menu-text">Evolution At The Macro Level</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-5" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-5" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./EvoBiogeogrphy.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">19</span>&nbsp; <span class="chapter-title">Evolutionary Biogeography</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./DivLife.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">20</span>&nbsp; <span class="chapter-title">Diversification and patterns of life through time</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-6" role="navigation" aria-expanded="true">
 <span class="menu-text">What About Us</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-6" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-6" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./HumanEvol.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text"><span class="chapter-number">21</span>&nbsp; <span class="chapter-title">Human Evolution</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./EvolMed.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">22</span>&nbsp; <span class="chapter-title">Evolutionary Medicine</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./EvolMedPartI.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">23</span>&nbsp; <span class="chapter-title">Evolutionary Medicine I: Aging and Diseases of Civilization</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./EvolMedPartII.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">24</span>&nbsp; <span class="chapter-title">Evolutionary Medicine II: Evolving Pathogens</span></span></a>
  </div>
</li>
      </ul>
  </li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" class="quarto-sidebar-collapse-item" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#human-evolution" id="toc-human-evolution" class="nav-link active" data-scroll-target="#human-evolution"><span class="header-section-number">22</span> Human Evolution</a>
  <ul class="collapse">
  <li><a href="#phylogenetic-position" id="toc-phylogenetic-position" class="nav-link" data-scroll-target="#phylogenetic-position"><span class="header-section-number">22.1</span> Phylogenetic Position</a></li>
  <li><a href="#what-the-fossils-say" id="toc-what-the-fossils-say" class="nav-link" data-scroll-target="#what-the-fossils-say"><span class="header-section-number">22.2</span> What the Fossils Say</a>
  <ul class="collapse">
  <li><a href="#an-abridged-timeline" id="toc-an-abridged-timeline" class="nav-link" data-scroll-target="#an-abridged-timeline"><span class="header-section-number">22.2.1</span> An Abridged Timeline</a></li>
  <li><a href="#evolutionary-trends" id="toc-evolutionary-trends" class="nav-link" data-scroll-target="#evolutionary-trends"><span class="header-section-number">22.2.2</span> Evolutionary Trends</a></li>
  <li><a href="#a-lone-survivor-of-an-extinct-radiation" id="toc-a-lone-survivor-of-an-extinct-radiation" class="nav-link" data-scroll-target="#a-lone-survivor-of-an-extinct-radiation"><span class="header-section-number">22.2.3</span> A Lone Survivor of an Extinct Radiation</a></li>
  </ul></li>
  <li><a href="#modern-human-origins" id="toc-modern-human-origins" class="nav-link" data-scroll-target="#modern-human-origins"><span class="header-section-number">22.3</span> Modern Human Origins</a>
  <ul class="collapse">
  <li><a href="#hypotheses" id="toc-hypotheses" class="nav-link" data-scroll-target="#hypotheses"><span class="header-section-number">22.3.1</span> Hypotheses</a></li>
  <li><a href="#multiregional-evolution-vs.-out-of-africa" id="toc-multiregional-evolution-vs.-out-of-africa" class="nav-link" data-scroll-target="#multiregional-evolution-vs.-out-of-africa"><span class="header-section-number">22.3.2</span> Multiregional Evolution <em>vs</em>. Out-Of-Africa</a></li>
  <li><a href="#out-of-africa-vs.-hybridization-and-assimilation" id="toc-out-of-africa-vs.-hybridization-and-assimilation" class="nav-link" data-scroll-target="#out-of-africa-vs.-hybridization-and-assimilation"><span class="header-section-number">22.3.3</span> Out-of-Africa <em>vs</em>. Hybridization-and-Assimilation</a></li>
  </ul></li>
  <li><a href="#what-makes-us-human" id="toc-what-makes-us-human" class="nav-link" data-scroll-target="#what-makes-us-human"><span class="header-section-number">22.4</span> What Makes Us Human?</a></li>
  <li><a href="#case-study-modern-human-origins" id="toc-case-study-modern-human-origins" class="nav-link" data-scroll-target="#case-study-modern-human-origins"><span class="header-section-number">22.5</span> Case Study: Modern Human Origins</a></li>
  <li><a href="#reflection-questions" id="toc-reflection-questions" class="nav-link" data-scroll-target="#reflection-questions"><span class="header-section-number">22.6</span> Reflection Questions</a></li>
  <li><a href="#references" id="toc-references" class="nav-link" data-scroll-target="#references"><span class="header-section-number">22.7</span> References</a></li>
  </ul></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default"><nav class="quarto-page-breadcrumbs quarto-title-breadcrumbs d-none d-lg-block" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./HumanEvol.html">What About Us</a></li><li class="breadcrumb-item"><a href="./HumanEvol.html"><span class="chapter-number">21</span>&nbsp; <span class="chapter-title">Human Evolution</span></a></li></ol></nav>
<div class="quarto-title">
<h1 class="title"><span class="chapter-number">21</span>&nbsp; <span class="chapter-title">Human Evolution</span></h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<p><em>by Dr.&nbsp;Michael Tobler</em></p>
<p><em>DRAFT 20250113</em></p>
<section id="human-evolution" class="level1" data-number="22">
<h1 data-number="22"><span class="header-section-number">22</span> Human Evolution</h1>
<p>Viewing the world through the eyes of an evolutionary biologist allows us to appreciate the origins of biodiversity and develop tangible solutions to some of the world’s major problems. But it also raises a number of questions about ourselves: How did humans come about? Why did we start to walk on two feet, use tools, and make art? And are we still evolving today? Darwin recognized how his ideas challenged the prevailing dogma that humans are somehow separate from the rest of God’s creation. Like other naturalists before him, he aptly observed our apparent relationship to other primates, and his ideas implied that the resemblance of traits between humans and other apes was a consequence of descent from a common ancestor. Hence, he concluded in <em>The Origin of Species</em>:</p>
<blockquote class="blockquote">
<p>“Man could no longer be regarded as the Lord of Creation, a being apart from the rest of nature. He was merely the representative of one among many families of the order Primates in the class Mammalia.” —Darwin, 1859</p>
</blockquote>
<p>Considering how meticulously Darwin collected evidence in support of his hypotheses on descent with modification and natural selection, he offered surprisingly little insight about the implications of his ideas on the origin of modern humans. Instead, he left his audience with a cliffhanger:</p>
<blockquote class="blockquote">
<p>“In the distant future I see open fields for far more important researches, [and] light will be thrown on the origin of man and his history.” —Darwin, 1859</p>
</blockquote>
<p>Darwin did not omit a discussion of human evolution in <em>The Origin of Species</em> due to the lack of interest. Rather, he feared that the discussion of human evolution would be perceived as too radical, and he did not want distract from the debate about the scientific merits of his ideas. Twelve years after the publication of <em>The Origin of Species</em>, Darwin followed up with a book devoted to human evolution and sexual selection (The Descent of Man, and Selection in Relation to Sex, first published in 1871); he wrote:</p>
<blockquote class="blockquote">
<p>“During many years I collected notes on the origin or descent of man, without any intention of publishing in the subject, but rather with the determination not to publish, as I thought that I should thus only add to the prejudices against my views.” —Darwin, 1871</p>
</blockquote>
<p>Undoubtedly, Darwin was right. Over 160 years after the publication of <em>The Origin of Species</em>, evolutionary biology is still a controversial topic in some segments of our society, even though other scientific theories that clash with a literal interpretation of scripture—for example, the idea that Earth is round, heliocentrism, the Big Bang theory, and the germ theory of disease—have been more widely accepted. Be that as it may, Darwin predicted accurately; now, in the distant future from his time, researchers have thrown light on the origin of man. A treasure trove of fossil artifacts have not only provided clues about changes in our morphology and geographic distribution, but have also taught us how our ancestors manufactured tools, used fire, and hunted big game in prehistoric time. In addition, population genetic analyses of modern human populations—and more recently, genetic analyses of ancient DNA left by long-extinct ancestors—has shed light on human movement patterns, adaptation, and interaction with extinct lineages. In this chapter, we will first establish the evidence for the human phylogenetic position. Then, we will briefly explore what we have learned about human origins from the fossil record, and how modern molecular genetic approaches have transformed our ability to ask question about our nature and where we come from.</p>
<section id="phylogenetic-position" class="level2" data-number="22.1">
<h2 data-number="22.1" class="anchored" data-anchor-id="phylogenetic-position"><span class="header-section-number">22.1</span> Phylogenetic Position</h2>
<p>In 1735, Linnaeus was the first to classify humans alongside other animals. He assigned our species to the class called Quadrupeds (which later became Mammalia) and in the order Anthropomorpha (which later became Primates). Besides humans (genus <em>Homo</em>), Linnaeus’ Anthropomorpha also included the genera <em>Simia</em> (apes) and <em>Bradypus</em> (sloths). Thomas Henry Huxley—a fervent supporter of evolutionary thought that earned him the nickname “Darwin’s bulldog”—was the first to systematically investigate the phylogenetic position of humans relative to other primates, and since the publication of his work in 1863, countless studies have followed. Humans (<em>Homo sapiens</em>) are part of the taxon Catarrhini (Old World monkeys), which includes the superfamily Cercopithecoidea (24 genera with 138 species; including baboons, macaques, colobus, vervet, mangabey, mandrill, proboscis monkey, and many others) and the superfamily Hominoidea. Hominoidea is composed of two families (Figure @ref(fig:apes)): the Hylobatidae (4 genera and 18 species of gibbons) and the Homonidae (4 genera and 8 species of Great Apes). Hence, humans are closely related to the chimpanzees of Africa (genus <em>Pan</em> with 2 species), the gorillas of Africa (genus <em>Gorilla</em> with 2 species), and the orangutans of Indonesia and Malaysia (genus <em>Pongo</em> with 3 species).</p>
<p>While morphological traits commonly used for phylogenetic inference and taxonomic classification unequivocally group humans with the other great apes, they do not offer sufficient information to resolve relationships within the family Hominidae. Specifically, morphological data alone was not able to falsify hypotheses about the relative position of humans, chimpanzees, and gorillas. For a long time, it was unclear whether humans and chimpanzees together were sister to gorillas, whether humans and gorillas together were sister to chimpanzees, or whether chimpanzees and gorillas together were sister to humans. Only the advent of molecular genetic approaches in 1960s and 1970s provided conclusive evidence that humans and the two species of chimpanzees are closely related with each other (Figure @ref(fig:apes)). Both genera are derived from a shared ancestor that lived in sub-Saharan Africa between 6.5-9.3 million years ago, which is consistent with fossil evidence estimating the divergence time at 6-10 million years ago (Moorjani et al.&nbsp;2016).</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/primate_phylogeny.png" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption>The phylogenetic position of humans relative to other species within the taxon Catarrhini, which includes the family Hominidae (great apes) and Hylobatidae (gibbons). Adopted from Chatterjee et al.&nbsp;(2009).</figcaption>
</figure>
</div>
</div>
</div>
<p>Molecular genetic analyses—spearheaded by the Human Genome Project that produced draft genomes for both humans and chimpanzees—have since revealed just how similar we are to our forest-dwelling cousins. At first sight, the key genetic difference between humans and chimpanzees appears to be a difference in chromosome numbers (chimps have 24 pairs, humans only 23). However, the human chromosome 2 is a composite of two chimpanzee chromosomes that fused into one, sometime after the split of our lineages. The order of genes on our second chromosome precisely matches the order of genes on the analogous chromosomes in chimpanzees. In addition, genome-wide analyses of sequence similarity have indicated that humans and chimpanzees share over 98 % of the nucleotide sequences—even 99.2 % in protein-coding regions. There are however, substantial differences in patterns of gene expression especially in the brain, suggesting that regulatory changes have played an important role in shaping the phenotypic differences between species.</p>
<table class="caption-top table">
<caption>Table 14.1: Percent sequence difference between the genomes of humans and chimpanzees for different genetic regions.</caption>
<thead>
<tr class="header">
<th style="text-align: left;">Sequence type</th>
<th style="text-align: right;">Percent difference</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Pseudogenes</td>
<td style="text-align: right;">1.68</td>
</tr>
<tr class="even">
<td style="text-align: left;">Non-coding, intergenic</td>
<td style="text-align: right;">1.24</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Non-coding, intronic</td>
<td style="text-align: right;">0.93</td>
</tr>
<tr class="even">
<td style="text-align: left;">Coding, synonymous</td>
<td style="text-align: right;">1.11</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Coding, nonsynonymous</td>
<td style="text-align: right;">0.80</td>
</tr>
</tbody>
</table>
</section>
<section id="what-the-fossils-say" class="level2" data-number="22.2">
<h2 data-number="22.2" class="anchored" data-anchor-id="what-the-fossils-say"><span class="header-section-number">22.2</span> What the Fossils Say</h2>
<p>Humans and chimpanzees may look very similar from a genomic perspective, but nobody can deny just how different we are in phenotype, especially in our lifestyles and behaviors. So, how did these differences arise? How did modern humans evolve from the ancestors we shared with chimpanzees just 6-10 million years ago? To address these questions, we first turn to the fossil record. Over the past century, scientists have unearthed fossil remnants that represent thousands of individual specimens, as well as stone tools and other artifacts that provide us with insights into human evolution. And unlike the gradual ape-to-man illustration so popular in museums and cartoons, the fossils show that there was no straight line of evolution from a distant ancestor to our present form. Instead, fossils indicate that there was a diversity of hominid forms that lived in sub-Saharan Africa and other portions of the planet. As we will see, how many different species this diversity represents and the relationship between different lineages is less clear.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/human_fossils.png" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption>Overview of key fossil representatives since the human lineage split from the last common ancestor with chimpanzees. Additional details are provided in the text. Adopted from Wood and Grabowski (2015).</figcaption>
</figure>
</div>
</div>
</div>
<section id="an-abridged-timeline" class="level3" data-number="22.2.1">
<h3 data-number="22.2.1" class="anchored" data-anchor-id="an-abridged-timeline"><span class="header-section-number">22.2.1</span> An Abridged Timeline</h3>
<p>To provide a general overview of the different types of fossils that we currently have available, I will briefly introduce some key groups that span the past 7 million years. This overview is by no means comprehensive, especially because the taxonomy of human fossil remains is constantly changing.</p>
<section id="possible-early-hominids" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="possible-early-hominids">Possible Early Hominids</h4>
<p>Remnants assigned to three different genera, <em>Sahelanthropus</em>, <em>Orrorin</em>, and <em>Aridipithecus</em>, are typically considered early hominins that represent lineages on the human side of the branch since the last common ancestor with chimpanzees. These fossils vary in age from about 7 million years (<em>Sahelanthropus</em>) to about 4.5 million years (<em>Aridipithecus</em>). All of these forms lack the characteristic features of later hominids. However, their are assumed to be among our ancestors because of peculiarities in their dentition and morphological adaptations of the pelvis and the hind limbs that suggest habitual (obligate) bipedal locomotion.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/earlyhominins.png" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption>A. Skull cast of <em>Sahelenthropus tchadensis</em>. Photo by Rama, <a href="https://creativecommons.org/licenses/by-sa/3.0/fr/deed.en">CC BY-SA 3.0 FR</a>. B. Sketch of an <em>Aridipithecus</em> skeleton. Illustration by Ori~, <a href="https://creativecommons.org/licenses/by-sa/3.0">CC BY-SA 3.0</a>.</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="archaic-hominins" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="archaic-hominins">Archaic Hominins</h4>
<p>Fossils assigned to two genera—<em>Australipithecus</em> and <em>Kenyanthropus</em>—are considered archaic hominins (also known as slender australopithecines) and span a time frame from roughly 2 to 4 million years ago. These forms are unambiguously hominin with predominantly bipedal locomotion, although their upper limb morphology suggest that they were still effective climbers. Evidence for bipedalism has not only been inferred from morphological adaptations in the skeleton, but also from spectacular footprints attributed to <em>Australipithecus afarensis</em> found in Laetoli, Tanzania (Figure @ref(fig:archaic)). Archaic humans also exhibited apelike skulls with projecting faces and large chewing teeth, albeit with reduced canines. All archaic hominins were relatively small (1.1-1.5 m tall; 30-45 kg). The brains of archaic hominins were slightly larger than those of earlier forms, but still substantially smaller than those of species from the genus <em>Homo</em> that evolved later. Note that assignment of different forms to either <em>Australipithecus</em> or <em>Kenyanthropus,</em> and the differentiation of species within those genera, largely depends on variation in the size and shape of teeth and other cranial features.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/archaic.png" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption>A. Reconstruction of the skull of Lucy, an <em>Australopithecus afarensis</em>. Photo by Pbuergler, <a href="https://creativecommons.org/licenses/by-sa/3.0">CC BY-SA 3.0</a>. B. Footprints attributed to <em>Australopithecus afarensis</em> from Laetoli (Tanzania). Masao et al.&nbsp;(2016).</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="megadont-and-hyper-megadont-archaic-hominins" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="megadont-and-hyper-megadont-archaic-hominins">Megadont and Hyper-Megadont Archaic Hominins</h4>
<p>About 2.5 million years ago, two distinctly different lineages appear in the fossil record: one includes fossils with more and more similarities to modern humans and is assigned to the genus <em>Homo</em>; the other has quite distinct cranial features and includes the genus <em>Paranthropus</em> (also known as robust australopithecines). <em>Paranthropus</em> includes so-called megadont and hyper-megadont forms that are characterized by extremely large post-canine teeth with exceptionally thick enamel. In addition, most <em>Paranthropus</em> exhibited a cranial crest that we also observe in modern carnivores. The cranial crest serves as an attachment point for powerful jaw muscles. Together, the dental modifications and cranial crest suggests that these hominins were adapted to powerful chewing (hence their nickname, nutcracker man). Otherwise, <em>Paranthropus</em> exhibited similar traits to the slender australopithecines; <em>i.e.</em>, they were of small stature, had large faces, small brains, and predominantly bipedal locomotion. Interestingly, it has been suggested that <em>Paranthropus</em> exhibited a thumb morphology required for the manufacturing of simple tools. Some researchers have hypothesized that <em>Paranthropus</em> was responsible for the manufacturing of the oldest known stone tools (Oldowan tool) that are about 2.5 million years old.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/paranthropus.png" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption>A. The skull of <em>Paranthropus boisei</em>, known as KNM ER 406, photographed at the Nairobi National Museum. Photo by Bjørn Christian Tørrissen, <a href="https://creativecommons.org/licenses/by-sa/3.0">CC BY-SA 3.0</a>. B. Oldowan stone chopper. Photo by José-Manuel Benito Álvarez, <a href="https://creativecommons.org/licenses/by-sa/2.5">CC BY-SA 2.5</a>.</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="transitional-hominins" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="transitional-hominins">Transitional Hominins</h4>
<p>Concurrently with <em>Paranthropus</em>, forms with unmistakably human characteristics started to appear about 2.5 million years ago. The oldest forms assigned to the genus <em>Homo</em> are <em>H. habilis</em> and <em>H. rudolfensis</em>. Both forms show a mix of morphology, with some traits seen in pre-modern <em>Homo</em> and some in archaic hominins. Hence, the generic placement of the two species is somewhat controversial, and some researchers argue that they should be placed in either <em>Australipithecus</em> or <em>Kenyanthropus</em> instead.</p>
<p>Traits shared with older forms include long arms, a flat face, and large mandibles and postcanine teeth, which indicated that <em>H. habilis</em> and <em>H. rudolfensis</em> consumed mechanically demanding diets. Both forms, however, exhibited larger brain volumes and statures than archaic humans. Especially <em>H. habilis</em> fossils are frequently found with remains of butchered animals and simple stone tools, suggesting that they were hunters. They are widely considered to be the first humans to have manufactured stone tools at a large scales; hence, the epithet <em>habilis</em> (handy man).</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/transitional.png" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption>A. Skull of <em>Homo habilis</em>. B. Skull of <em>H. rudolfensis</em>. Photos by Hawks et al.&nbsp;(2017).</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="pre-modern-hominins-and-homo-sapiens" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="pre-modern-hominins-and-homo-sapiens">Pre-Modern Hominins and <em>Homo sapiens</em></h4>
<p>Starting about 1.8 million years ago, lineages appear that have modern humans’ characteristic postcranial morphology. Older forms still lack the distinctive size and shape of the modern human cranium, and they exhibit larger teeth and more robust jaws, but the similarities to modern humans get closer the younger the fossils are. Not surprisingly, there is tremendous variation in brain size in this group, ranging from about 600 to over 1,300 cm<sup>3</sup>.</p>
<p>The two earliest forms among pre-modern hominins are <em>H. ergaster</em> and <em>H. erectus</em>, which many scientists consider to the be same species (<em>H. erectus</em>). <em>H. erectus</em> is significant for many reasons, including its huge temporal range (1.8 million to 30,000 years ago) that overlapped with modern humans. In addition, <em>H. erectus</em> was the first hominin that left sub-Saharan Africa and colonized wide portions of Eurasia and and southeastern Asia. Compared to modern humans, <em>H. erectus</em> exhibited a smaller brain volume, larger teeth, and lacked a chin (Figure @ref(fig:erectheidel)A). However, <em>H. erectus</em> was of similar size and stature as us, and it was also the first human species to exhibit a flat face with a prominent nose and possibly a reduced coverage of body hair. Much controversy surrounds <em>H. erectus</em>’ ability to speak, but some researchers proposed that they communicated using some proto-language. African populations of <em>H. erectus</em> are likely the ancestors of later human forms (<em>H. heidelbergensis</em> and <em>H. antecessor</em>), which then gave rise to modern humans. Similarly, Asian populations of <em>H. erectus</em> have been hypothesized to be ancestral to <em>H. floresiensis</em> and other Asian hominins.</p>
<p>Two species that temporarily overlapped with <em>H. erectus,</em> and are likely derived from it, are <em>H. antecessor</em> (found in Spain) and <em>H. heidelbergensis</em> (found in primarily in Europe, but also northern Africa and parts of Asia; Figure @ref(fig:erectheidel)B). Fossils of these forms are about 100,000-600,000 years old. Both forms differ from modern humans in details of their cranial and dental morphology and bear a more robust postcranial skeleton, but they have increasingly large brains. <em>H. heidelbergensis</em> fossils are also accompanied by changing stone tools, suggesting technological advances compared to <em>H. erectus</em>. For example, there is evidence for the emergence of hafting (the attachment of rock tips to wooden shafts to create spears and axes) and for the regular use of fire in daily life about 400,000 years ago. Depending on the authority, <em>H. antecessor</em> or <em>H. heidelbergensis</em> is considered the direct ancestor of neanderthals and modern humans.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/erect_heidel.png" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption>A. <em>Homo ergaster</em> skull from Nariokotome, Lake Turkana area, Kenya. B. <em>Homo heidelbergensis</em> skull from Kabwe, central Zambia, southern Africa. Photos by <a href="https://www.flickr.com/photos/jsjgeology/">James St.&nbsp;John</a>, <a href="https://www.flickr.com/photos/jsjgeology/">CC BY 2.0</a>.</figcaption>
</figure>
</div>
</div>
</div>
<p>Finally, there is fossil evidence for three additional species that survived until relatively recently: <em>H. neanderthalensis</em> from Europe, the Middle East, and part of Asia (400,000-20,000 years ago); an undescribed form known as Denisovans from Siberia that we will talk about in more detail later in this chapter; and <em>H. florisiensis</em> from the Indonesian island of Flores (74,000-16,000 years ago). All three forms widely coexisted with modern humans that start to appear in the fossil record about 300,000 years ago.</p>
<p>Neanderthals were very similar to modern humans but exhibited a distinctly thick, double-arched brow ridge, a large nose, and other cranial peculiarities (Figure @ref(fig:sapnea)). On average, they had a larger brain volume than modern humans, and their hyoid bone—which in modern humans is associated with tongue mobility and speech—was identical to ours. Hence, it is widely assumed that <em>H. neanderthalensis</em> communicated through language. They used sophisticated technology; besides stone tools, they created fire, built cave hearths, made adhesives from plant sap, crafted clothes, wove, and traveled across stretches of water in rafts. Neanderthals are also suspected to have manufactured ornaments, simple instruments, and created the some of the oldest cave paintings that were previously attributed to modern humans.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/sapiens_neanderthals.png" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption>Comparison of modern human (left) and Neanderthal skulls from the Cleveland Museum of Natural History. Illustration by hairymuseummatt (original photo) and DrMikeBaxter (derivative work), <a href="https://creativecommons.org/licenses/by-sa/2.0">CC BY-SA 2.0</a>.</figcaption>
</figure>
</div>
</div>
</div>
<p>In comparison, <em>H. floriensis</em> was spatially much more restricted and only occurred on a single island (Flores) in Indonesia (Figure @ref(fig:flores)). Adults only reached about 1.1 m in height, which is why this species is nicknamed “the hobbit”. <em>H. floriensis</em> is not closely related to the lineages the gave rise to modern humans, but is instead thought to be derived from <em>H. erectus</em>. Its small size is likely an example of island dwarfism, as also evidenced by diminutive elephants and hippos that existed alongside them. The discovery of <em>H. florisensis</em> in 2004 caused extensive media attention because it suggested that other hominins roamed Earth just 12,000 years ago.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/liangbua.jpg" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption>Cave where the remainings of <em>Homo floresiensis</em> where discovered in 2003, Liang Bua, Flores, Indonesia. Photo by <a href="https://www.flickr.com/photos/rosino/">Rosino</a>, <a href="https://creativecommons.org/licenses/by-sa/2.0/">CC BY-SA 2.0</a>.</figcaption>
</figure>
</div>
</div>
</div>
</section>
</section>
<section id="evolutionary-trends" class="level3" data-number="22.2.2">
<h3 data-number="22.2.2" class="anchored" data-anchor-id="evolutionary-trends"><span class="header-section-number">22.2.2</span> Evolutionary Trends</h3>
<p>The fossil record of hominins spans almost 8 million years, revealing evolutionary trends. Many modifications pertain to attributes of the skull. For example, there is a gradual forward movement of the foramen magnum (the hole in the base of the skull through which the spinal chord enters) that is likely a consequence of changed posture and bipedalism. In addition, canines become smaller, molars larger, and there is a decrease in prognathism (<em>i.e.</em>, the extent to which the bottom part of the face juts forward). Finally, there is a clear increase in cranial capacity and a general rounding of the skull. All of these morphological changes are reflective or correlated with three major trends: the evolution of bipedal locomotion, a non-honing chewing complex and changed diets, and a stark encephalization of the brain.</p>
<section id="bipedalism" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="bipedalism">Bipedalism</h4>
<p>While other great apes are capable of facultative bipdalism, human habitually walk on two feet (we are obligate bipeds). The transition from quadrupedalism to bipedalism is actually one of the earliest evolutionary trends, and skeletal modifications associated with increased bipedalism have been inferred even in early hominids (<em>Sahelanthropus</em>, <em>Orrorin</em>, and especially <em>Aridipithecus</em>). The transition toward bipedal locomotion is associated with a wealth of skeletal changes, from the position of the foramen magnum, to the shape of the spinal chord and the structure of lumbar vertebrae, to the shape and position of the pelvis. In addition, there were changes in leg and foot anatomy, and the arms shorted as climbing and swinging became less important modes of locomotion. All of these changes occurred gradually across several million years of evolution, and fully bipedal locomotion likely did not occur until about 4 million years ago with <em>Australipithecus</em>. Why exactly hominins shifted from walking on all fours to walking on two feet remains a matter of controversy. Some of the most prominent hypotheses highlight the efficiency of locomotion and predator recognition in open savannas (as opposed to the ancestral forested habitats). In addition, a shift to bipedalism may have freed up the hands to perform tasks other than locomotion, such as the carrying of food or offspring or the use of tools.</p>
</section>
<section id="non-honing-chewing-complex" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="non-honing-chewing-complex">Non-Honing Chewing Complex</h4>
<p>Compared to other great apes, humans have highly reduced canines and enlarged molars (Figure @ref(fig:canine)). The large canines in the upper jaw of apes are equipped with a sharpened posterior edge. Every time the jaws close, that edge is sharpened by rubbing against the third premolar in the lower jaw, which makes up the so-called “honing (sharpening) complex”. Not surprisingly, the sharp canines play an important role in great ape foraging, as they cut and shred leaves, fruit, and other foods. In contrast, along with the reduction in canine size, humans have gradually lost the honing complex. The overall changes in dentition likely reflects changes in dietary habits, tool use, and the use of fire to cook food items prior to consumption.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="HumanEvol_files/figure-html/canine-1.png" class="img-fluid figure-img" width="768"></p>
<figcaption>The size of canines in the upper and lower jaw from different species of great apes, including humans and two species of <em>Australopithecus</em>. <a href="data/14_canine_size.csv">Data</a> from Ward et al.&nbsp;(2010).</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="brain-encephalization" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="brain-encephalization">Brain Encephalization</h4>
<p>Perhaps most importantly, human evolution is characterized by a continuous increase of brain size—both in terms of the size of the brain relative to body mass and absolute brain size. Gains in cranial volume increased only moderately at first; but starting with early forms of <em>Homo,</em> brains got substantially bigger very quickly (Figure @ref(fig:brain)A). In <em>H. erectus</em> alone, brain volume increased by about 50 % compared to transitional <em>Homo</em>. The largest brains were found in <em>H. neanderthalensis</em>, which had slightly bigger brains than anatomically modern humans. Note that the same patterns were found when researchers inferred the amount of blood flow through the internal carotid artery, which supplies the brain (Figure @ref(fig:brain)B).</p>
<p>Consistent with increases in brain size, there is also evidence for changes in brain structure and morphology. For example, the brains of non-human primates and early hominins were symmetrical, and there is evidence for increasing lateralization (tendency for some neural functions or cognitive processes to be specialized to one side of the brain or the other) through time. The subdivision into a left and right side of the brain is not only evident in modern humans, but also in endocasts—trace fossils of that arise when minerals replace soft tissues during fossilization—from earlier hominins. Endocasts also indicate significant changes in the cerebellum (associated with learned motor activities), the limbic system (involved in emotion and social communication), and the cerebral cortex (processing of sensory experiences). It is tempting to speculate that these changes in brain structure—and likely function—were directly related to changes in social organization, technological advances, and cultural practices like language.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="HumanEvol_files/figure-html/brain-1.png" class="img-fluid figure-img" width="768"></p>
<figcaption>A. Endocranial volume for different fossilized hominids, including presentatives from archaic homonids (<em>Australopithecus</em>), hyper-megadonts (<em>Paranthropus</em>), transitional form of <em>Homo</em> (<em>H. habilis</em> and <em>H. rudolfensis</em>), and premodern <em>Homo</em> (<em>H. erectus</em> and <em>H. heidelbergensis</em>). <a href="data/14_human_brain.csv">Data</a> from Du et al.&nbsp;(2018). B. Estimates of blood frow through the internal carotid artery into the brain for different hominids. <a href="data/14_human_brain.csv">Data</a> from Seymour et al.&nbsp;(2016).</figcaption>
</figure>
</div>
</div>
</div>
</section>
</section>
<section id="a-lone-survivor-of-an-extinct-radiation" class="level3" data-number="22.2.3">
<h3 data-number="22.2.3" class="anchored" data-anchor-id="a-lone-survivor-of-an-extinct-radiation"><span class="header-section-number">22.2.3</span> A Lone Survivor of an Extinct Radiation</h3>
<p>The fossil record clearly disproves the common notion of a straight evolutionary line between the common ancestor we shared with chimpanzees and modern humans, with gradual changes from small-brained quadrupeds to large-brained bipeds. Rather, there is clear evidence for a diversification of lineages around 4 million years ago. For most of the time since, multiple species of humans have coexisted in different parts of Africa and Eurasia, and in some cases there is also clear evidence that these species interacted ecologically. While the diversity of prehistoric humans reached a peak around 2 million years ago, there is robust evidence that even modern humans—which appear in the fossil record a mere 300,000 years ago—temporally overlapped with <em>H. neanderthalensis</em>, <em>H. heidelbergensis</em>, <em>H. floresiensis</em>, and likely even <em>H. erectus</em>. As we will see, there is also direct evidence that anatomically modern humans interacted with <em>H. neanderthalensis</em>, Denisovans, <em>H. heidelbergensis</em> and other distinct lineages that we do not know in detail. Hence, our lineage—the contemporary <em>H. sapiens</em>—represents a lone survivor from a considerable and likely underestimated diversity of related forms that existed for the better part of the past 4 million years.</p>
<p>While the fossil record has offered profound insights into our origins, it is important to also emphasize what it cannot tell us about human evolution. First and foremost, we do not know what forms actually represent different species incapable of interbreeding. The taxonomy of fossil remains hinges on the analysis of skeletal characteristics, assessed mostly from incomplete skeletons. Hence, we may be overestimating the past species diversity, if some of the more subtle morphological differences are reflective of intraspecific variation, perhaps in relation to geography. However, we may also be underestimating the past species diversity if the current fossil record is incomplete or if important species differences are not inferable from the partial remains that we have available.</p>
<p>The fossil record also leaves us largely ignorant about the phylogenetic relationship of different forms we know. Remember, Huxley was unable to resolve the phylogenetic relationship among gorillas, chimpanzees, and humans based on the analysis of skeletal traits. Needless to say that this task is not made any easier when more closely related and less complete skeletons are analyzed. So, while <em>Astralopithecus afarensis</em> is often assumed to be the ancestor of <em>Paranthropus</em> and <em>Homo</em> species, or <em>H. heidelbergensis</em> the ancestor of <em>H. neanderthalensis</em> and <em>H. sapiens</em>, we cannot test those hypotheses conclusively with the data available. Inferring evolutionary relationships and processes, especially in deeper time, is challenging because of the comparatively scant lines of evidence. However, it is important to remember just how young we are as a species. If we ask how exactly modern humans arose from the diversity of lineages that are evident in the fossil record, we can leverage a much broader toolset to test hypotheses framed on the basis of insights provided by fossils.</p>
</section>
</section>
<section id="modern-human-origins" class="level2" data-number="22.3">
<h2 data-number="22.3" class="anchored" data-anchor-id="modern-human-origins"><span class="header-section-number">22.3</span> Modern Human Origins</h2>
<p>A number of of hypotheses have been proposed to explain the origin of <em>Homo sapiens</em>. The different hypotheses cover a range of possibilities about how anatomically modern forms may have arisen from more archaic hominids. I will briefly introduce each hypothesis, and then we will examine the evidence for the alternative scenarios.</p>
<section id="hypotheses" class="level3" data-number="22.3.1">
<h3 data-number="22.3.1" class="anchored" data-anchor-id="hypotheses"><span class="header-section-number">22.3.1</span> Hypotheses</h3>
<p>On one extreme of the spectrum of modern human origins is the candelabra hypothesis, whose most prominent proponent was anthropologist Carleton Coon. The candelabra hypothesis posits that anatomically modern humans arose in Africa, Asia, and Europe from archaic forms that had colonized these regions earlier (Figure @ref(fig:hypotheses)A). Under this scenario, modern humans in different geographic regions evolved independently, and shared characteristics among lineages would be the product of convergent evolution. Coon mostly used this idea to justify and propagate his racist ideologies. He believed—without actual evidence, of course—that the transformation from archaic to modern humans started in European populations first, which in his eyes proved why Europeans were so advanced compared to he considered to be more primitive peoples of other regions. The candelabra hypothesis is firmly rooted in scientific racism and eugenics, both debunked pseudosciences that remind us of the painful historical track record of biology and notions of race. Even if we set the racist connotations aside, there is just no merit to the idea that modern humans represent multiple, deeply divergent evolutionary lineages, which is why candelabra is mentioned here more as a historical side note.</p>
<p>Similar to candelabra, the second hypothesis of modern human origins—the multiregional evolution hypothesis—proposes that archaic humans colonized Eurasia from Africa about 1 million years ago, and they then diverged in different geographic regions while remaining connected through gene flow (Figure @ref(fig:hypotheses)B). The time estimate of emigration out of Africa is based on the age of <em>H. erectus</em> fossils that appear in Eurasia around that time.</p>
<p>In contrast, the third hypothesis—the out-of-Africa hypothesis—proposes that there were two waves of emigration of African <em>Homo</em> that colonized Eurasia. A first wave involved <em>H. erectus</em>, which colonized Eurasia about 1 million years ago; again, consistent with the fossil record for that species. However, the out-of-Africa hypothesis also postulates that anatomically modern human originated in Africa. Modern humans then dispersed out of Africa much more recently (100,000-200,000 years ago) and displaced the other hominins in Eurasia that settled there before (Figure @ref(fig:hypotheses)C). The replacement of ancient lineages by modern <em>H. sapiens</em> derived from Africa is why this hypothesis is also known as the African replacement hypothesis.</p>
<p>Last but not least, the hybridization-and-assimilation hypothesis is very similar to the out-of-Africa hypothesis, in that it assumes an African origin of modern humans. However, instead of replacing more ancient lineages during the colonization of Eurasia, this hypothesis proposes that newly-arriving modern <em>H. sapiens</em> hybridized with other forms of <em>Homo</em> that colonized these areas (Figure @ref(fig:hypotheses)D). Hybridization essentially led to the assimilation—or fusion—of different lineages.</p>
<p>In the following sections, we will first address the multiregional evolution and the out-of-Africa hypotheses, which make contrasting predictions that can be address using phylogenetic and population genetic analyses of extant human populations. As you will see, there is overwhelming evidence for an African origin of modern humans and relatively recently migrations to colonize Eurasia and other portions of the planet. In a second step, we will then weigh the evidence for the out-of-Africa and the hybridization and assimilation hypotheses. Juxtaposing these alternative scenarios has become feasible in the past decade, as technological breakthroughs have enabled us to sequence the DNA of long-dead ancestors, including members of <em>H. neanderthalensis</em> and other extinct species.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/human_hypotheses.png" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption>Hypotheses explaining the origin of modern himans (see text for details): A. candelabra hypothesis; B. multiregional; C. out-of-Africa (or African replacement) hypothesis; D. hybridization-and-assimilation hypothesis.</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="multiregional-evolution-vs.-out-of-africa" class="level3" data-number="22.3.2">
<h3 data-number="22.3.2" class="anchored" data-anchor-id="multiregional-evolution-vs.-out-of-africa"><span class="header-section-number">22.3.2</span> Multiregional Evolution <em>vs</em>. Out-Of-Africa</h3>
<p>The multiregional evolution and out-of-Africa hypotheses make contrasting predictions that are based on evidence from the fossil record. We know that <em>H. erectus</em> was the first hominin to leave Africa and colonize Eurasia. Evidence of fossil bones and manufactured tools suggest that migrations of <em>H. erectus</em> started about 1.8 million years ago, and within 500,000 years the distribution of these ancient humans reached as far as China and Indonesia. It is important to note that these are conservative estimates, especially since new fossil discoveries from China suggest that the start of Eurasian colonization may in fact have occurred significantly earlier than what we had assumed. In contrast, fossil remains of anatomically modern humans are much younger. The oldest known fossils are between 200,000 and 300,000 years old and were all found on the African continent, including Morocco, Ethiopia, and South Africa. The oldest <em>H. sapiens</em> fossil outside of Africa was recently discovered in Israel and is about 185 million years old. In addition, a wealth of other modern human remains have been found in the Middle East and are dated at 90,000-120,000 years before present.</p>
<p>If the multiregional evolution hypothesis were true and modern humans in different geographic regions were derived from <em>H.-erectus</em>-like forms, then we would predict the divergence time between extant African and non-African human populations to be more than 1 million years, consistent with the fossil traces left behind by <em>H. erectus</em>. In contrast, if modern humans arose in Africa and then emigrated to and replaced more ancient humans in Eurasia, then the divergence time between African and non-African populations should be less than 200,000 years. The two hypotheses also make contrasting predictions about the location of ancestral lineages; they should be dispersed randomly across Africa and Eurasia if the multiregional evolution hypothesis was true, but restricted to Africa if the out-of-Africa hypothesis was true. Both of these predictions—the one related to divergence time and the one related to the location of ancestral lineages—can be addressed through phylogenetic analyses of extant human populations.</p>
<p>Two additional predictions arise from the population genetic consequences of dispersal and the effects of genetic drift associated with founder effects. As discussed in <a href="evolutionary-mechanisms-ii-mutation-genetic-drift-migration-and-non-random-mating.html#genetic-drift-the-random-force">Chapter 6</a>, founder effects occur when a random subset of individuals in a population disperses to a new geographic area and establishes new populations. In this case, only a subset of alleles present in the ancestral population is carried along. The out-of-Africa hypothesis—in which such a founder effect occurs—consequently predicts reduced genetic diversity and the presence of only a subset of neutral alleles in non-African populations compared to Africans. In contrast, the multiregional evolution hypothesis—in which such a founder effect does not occur—predicts that levels of genetic diversity are equal and the presence of sets of neutral alleles randomly distributed among human populations in different geographic regions. This second set of hypotheses are testable through population genetic analyses of extant human populations.</p>
<table class="caption-top table">
<caption>Table 14.1: Summary of contrasting predictions between the out-of-Africa and the multiregional evolution hypotheses.</caption>
<colgroup>
<col style="width: 33%">
<col style="width: 33%">
<col style="width: 33%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Prediction</th>
<th style="text-align: left;">Multiregional evolution</th>
<th style="text-align: left;">Out-of-Africa</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Divergence time between African and non-African populations</td>
<td style="text-align: left;">&gt;1,000,000 years</td>
<td style="text-align: left;">&lt;200,000 years</td>
</tr>
<tr class="even">
<td style="text-align: left;">Location of ancestral lineages</td>
<td style="text-align: left;">Random</td>
<td style="text-align: left;">Africa</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Levels of genetic diversity</td>
<td style="text-align: left;">Not significantly different among geographic regions</td>
<td style="text-align: left;">Highest in Africa, reduced everywhere else</td>
</tr>
<tr class="even">
<td style="text-align: left;">Sets of neutral alleles</td>
<td style="text-align: left;">Each geographic regions has unique alleles; no region’s alleles are a subset of another region’s</td>
<td style="text-align: left;">Only African populations have unique alleles; alleles present in non-African populations are a subset of those in Africa</td>
</tr>
</tbody>
</table>
<p>So, what is the evidence for and against the multiregional evolution and out-of-Africa hypotheses? Max Ingman and his colleagues (2000) were among the first to rigorously test these phylogenetic predictions by analyzing the mitochondrial DNA of 53 humans of diverse geographic provenance. The phylogeny they reconstructed (Figure @ref(fig:modhuphy)) provides unequivocal support for the out-of-Africa hypotheses. The age of the shared ancestor of all modern human population was estimated at about 170,000 years. More importantly, the age of the youngest clade containing both African and non-African populations (marked with an orange dot in Figure @ref(fig:modhuphy)) was estimated at around 52,000 years, falling squarely within the range of predictions for out-of-Africa, but not multiregional evolution. In addition, the phylogeny also indicated that ancestral lineages are restricted to the African continent, and they do not occur randomly across the continents as predicted by the multiregional evolution hypothesis. More recent phylogenetic analyses based on partial and whole-genome sequences of nuclear DNA have largely corroborated these results, even though humans have likely left Africa earlier. The most recent estimates put the exodus of modern humans from Africa at around 200,000 years ago.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/modernhumanphylo.png" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption>Phylogeny based on mitochondrial DNA sequences of extant human populations. African and non-African populations are indicated in color, and the estimated divergence times are indicated for the last common ancestor of all humans as well as the youngest clade containing both African and non-African populations. Adopted from Ingman et al.&nbsp;(2000).</figcaption>
</figure>
</div>
</div>
</div>
<p>Just like phylogenetic analyses, population genetic analyses that survey patterns of genetic variation have also uncovered evidence for the out-of-Africa hypothesis. Research on mitochondrial DNA, hypervariable microsatellites, transposable elements, and—most recently—whole-genome sequencing has found the highest levels of genetic variation in African populations, and reduced variation in non-African populations. For example, genetic variation estimated in 51 populations at 650,000 loci within the human genome revealed a clear negative correlation between heterozygosity and the distance of a population to Addis Ababa, Ethiopia, which is proximate to the presumed cradle of modern humans (Figure @ref(fig:gendiv)A). This is consistent with sequential founder effects that reduced genetic variation as bands of humans ventured ever farther away and established new populations, even in distant and isolated regions of Oceania and the Americas. Genetic analyses have also revealed that unique neutral alleles are not randomly distributed across different geographic regions. Rather, unique alleles are largely restricted to Africa, and alleles in other regions represent a mere subset of African alleles (Figure @ref(fig:gendiv)B).</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="HumanEvol_files/figure-html/gendiv-1.png" class="img-fluid figure-img" width="768"></p>
<figcaption>A. Relationship between the genetic diversity (heterozygosity) of 51 human populations and their distance from Addis Ababa, Ethiopia. <a href="data/14_humanheterozygosity.csv">Data</a> from Li et al.&nbsp;(2008). B. Frequency of different CD4 STRP alleles in African and non-African populations both for Alu(+) and Alu(-) chromosomes. Non-African alleles are clearly a subset of alleles present in African populations. <a href="data/14_subsets.csv">Data</a> from Tishkoff et al.&nbsp;(1996).</figcaption>
</figure>
</div>
</div>
</div>
<p>Consequently, both phylogenetic and population genetic analyses have resoundingly rejected the multiregional evolution hypothesis. Modern humans in different geographic regions are not derived from ancient <em>H. erectus</em> that left Africa in excess of 1,000,000 years ago. Instead, our species originated on the African continent relatively recently and then left to colonize the rest of the world around 200,000 years ago. The question then becomes: what happened as those modern human colonizers encountered other forms of <em>Homo</em> that had already settled all across Eurasia? The fossil evidence clearly indicates that those areas were not only occupied by <em>H. erectus</em>, but also <em>H. heidelbergensis</em>, <em>H. neanderthalensis</em>, and other lesser-known members of the genus <em>Homo.</em> Insights into the outcome of the interactions between modern humans and other homominids have only become available recently, as we have developed the capability to analyzing ancient DNA left behind by our ancestors.</p>
</section>
<section id="out-of-africa-vs.-hybridization-and-assimilation" class="level3" data-number="22.3.3">
<h3 data-number="22.3.3" class="anchored" data-anchor-id="out-of-africa-vs.-hybridization-and-assimilation"><span class="header-section-number">22.3.3</span> Out-of-Africa <em>vs</em>. Hybridization-and-Assimilation</h3>
<p>In 1997, a rumble went through the science world. For the first time in history, Svante Pääbo and his team were able to extract DNA from the skeletal remains of an extinct human, a specimen of <em>H. neanderthalensis</em> that lived about 40,000 years ago (Krings et al.&nbsp;1997). By 2017, scientists had sequenced partial or complete genomes from more than 1,100 ancient human and archaic hominin individuals, rewriting our understanding of our own evolutionary history. While analyses of these ancient DNA samples confirmed an African origin of anatomically modern humans, followed by dispersal to other portions of our planet, the new findings also revised the timing of events in our history and shed new light into how we interacted with other forms of <em>Homo</em> that had already settled all over Eurasia.</p>
<p>Sequencing of ancient DNA confirmed that Neanderthals indeed represented a distinct evolutionary lineage that diverged about 500,000 years ago from the ancestors of anatomically modern humans (Figure @ref(fig:ancientphylo)). Furthermore, clues left behind in the DNA of those ancient bones also revealed that we are likely underestimating the diversity of hominins that lived in the recent past and interacted with modern humans. For example, DNA extracted from finger bones found in the Denisova cave in Siberia turned out to be neither modern human nor Neanderthal in origin. Instead, these finger bones are the remains of a lineage that diverged from our shared ancestors with Neandertals roughly 900,000 years ago (Figure @ref(fig:ancientphylo)). Besides information encoded in the DNA, we have little knowledge about these so-called Denisovans, which is why they still await formal scientific description. The current lack of fossil traces, beyond the finger bones we currently have available, makes it impossible to know how the Denisovan’s features may have compared to ours and other extinct lineages.</p>
<p>DNA sequencing also shed light into the identity of fossils found in Sima de los Huesos, Spain. These fossils have long stirred controversy about the age and identity of Neanderthals, because they bear a striking resemblance to Neanderthals even though they are much older than all the other Neanderthal fossils we know. DNA analyses settled these debates to some degree, as the fossils from Sima de los Huesos are not actually closely related to Neanderthals. Rather, these ancient fossils are more closely related to the Denisovans, with which they shared a common ancestor around 700,000 years ago. Given the deep divergences, some researchers have speculated that that the Sima de los Huesos fossils may represent <em>H. heidelbergensis</em>. Overall, these exciting findings clearly show how a wealth of pre-modern humans existed not only in Africa but over wide stretches Eurasia as well. Given that DNA preserves best in the colder climates of northern latitudes but degrades fast in warmer and more humid subtropical and tropical climates, we are likely underestimating the past diversity simply because we have no ancient DNA specimens from those locations.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/ancientdnaphylo.png" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption>Phylogeny modern humann and pre-modern <em>Homo</em> for which we have ancient DNA available. This tree was derived from mitochondrial genome sequences and includes divergence time estimates for critical nodes. Adopted from Meyer et al.&nbsp;(2014). Inserts show crania collected at Lima de los Huesos. Photo of lateral view by UtaUtaNapishtim, <a href="https://creativecommons.org/licenses/by-sa/4.0">CC BY-SA 4.0</a>, photo of frontal view from Sala et al.&nbsp;(2015).</figcaption>
</figure>
</div>
</div>
</div>
<p>The availability of ancient DNA not only allowed us to confirm the evolutionary distinctness of different pre-modern humans and clarify the temporal patterns of diversification, but it also provided the template necessary to ask whether any modern humans exhibit genetic similarities to extinct forms. In other words, ancient DNA allows us to contrast the out-of -Africa and the hybridization-and-assimilation hypotheses (Figure @ref(fig:hypotheses)C-D). If modern humans indeed replaced pre-modern forms as they dispersed out of Africa, we would not expect to find any DNA sequences with close similarities to those from pre-modern humans in extant populations. Alternatively, if at least some modern humans carry DNA that bears similarities to the DNA of Neanderthals or other extinct forms, then interactions between the modern new arrivals from Africa and the more ancient lineages already settled in Eurasia may not have been purely antagonistic. Instead, it would indicate that modern humans hybridized with those ancient forms.</p>
<p>Perhaps not surprisingly, we now have unequivocal evidence that modern humans indeed interbred with Neanderthals as they emigrated out of Africa. Introgression of Neanderthal DNA is especially common in Eurasia, the Middle East, and North Africa, where we also have evidence for the sympatry of both species based on the fossil record (Figure @ref(fig:nea)). In contrast, evidence for introgression is comparatively rare in extant human populations from sub-Saharan Africa, which did not exhibit significant range overlap with Neanderthals (Figure @ref(fig:nea)). These findings substantially rewrite the history of human evolution, because they suggest that Neanderthals never really went extinct. Instead, they were simply assimilated into the gene pool of modern humans, and hence, traces of their existence continue to live on in many of us. Rather than being some distant relative that also inhabited this planet in the past, Neanderthals are actually among our ancestors.</p>
<p>The bits of Neaderthal DNA still present in non-African extant humans are not simply non-functional stretches of junk DNA, but many remain functional and impact the expression in phenotypic traits. For example, Neanderthal DNA has been linked to the morphology of our cranium and brain, the coloration of eyes, hair, and skin, as well as aspects of our immune system. If you have light skin, blond or red hair, or blue or green eyes, you most certainly have inherited those traits from your Neanderthal ancestors. Introgression of these specific genes may have been driven by natural selection and helped newly arriving human populations adapt to life at higher latitudes. By the time modern humans arrived from Africa, Neanderthals had already lived and adapted to the specific environmental conditions at high latitudes for about 300,000 years, carrying allelic variants that had proven beneficial. Interbreeding allowed these beneficial alleles to be introduced into the modern human population, and selection swept them to high frequency.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="HumanEvol_files/figure-html/nea-1.png" class="img-fluid figure-img" width="768"></p>
<figcaption>Estimated proportions of Neanderthal introgression in different human populations (based on a metric called f4 ratio). <a href="data/14_neanderthal_introgression.csv">Data</a> from Vyas and Mulligan (2019)</figcaption>
</figure>
</div>
</div>
</div>
<p>Interbreeding, it turns out, was not restricted to Neanderthals and modern humans. We also have evidence for interbreeding between Neanderthals and Denisovans, as well as Denisovans and modern humans (Figure @ref(fig:humanintro)). For example, about 5 % of the genome of people from Oceania is derived from Denisovans. Again, traits associated with pigmentation and immune function in those populations are likely derived from these extinct ancestors. Moreover, genes that have been associated with high-elevation adaptation in Tibetans also appear to have originated from admixture with Denisovans. Finally, analyses of genomic similarities across modern humans, Neanderthals, and Denisovans suggests that patterns of interbreeding also include a fourth member of the genus <em>Homo</em>. So far, we don’t know the identity of this fourth species, because we do not have a matching reference genome. However, scientists have speculated that it may be <em>H. heidelbergensis</em> or <em>H. erectus</em> based on the fossil record, which overlapped with the other species both spatially and temporally.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/humanintroduction.png" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption>Patterns of historical gene flow between different lineages of pre-modern and modern <em>Homo</em>. Adopted from Pfrüfer et al.&nbsp;(2014).</figcaption>
</figure>
</div>
</div>
</div>
<p>In conclusion, the history of recent human evolution is not really a bifurcating phylogenetic tree. Rather it may be better represented by a web of gene flow among divergent lineages. This, of course, raises questions about how many species are actually represented in the fossil record. Clearly, reproductive isolation among these lineages was not complete; otherwise, we would not be able to uncover evidence of widespread hybridization. However, depletion of Neanderthal DNA around functional genomic regions in extant humans suggests that there may have been deleterious epistatic interactions between Neanderthal and modern human alleles. One study estimated the fitness cost of hybridization with Neanderthals at about 0.5 % (Harris and Nielsen 2016)—not huge, but still sufficient for natural selection to cause substantial changes in allele frequencies through time.</p>
<p>The speed at which we have been—and continue to be—learning about our own evolutionary history is staggering. New studies employing revolutionary technologies and analyzing ever-more and ever-older samples are being published almost on a weekly basis. Just in February 2021, the record for the oldest DNA ever sequenced was shattered with the publication of the genome of a mammoth that lived between 1.1 and 1.6 million years ago (van der Valk 2021). No doubt, we can anticipate learning more exciting information about the nature of our ancestors and ourselves!</p>
</section>
</section>
<section id="what-makes-us-human" class="level2" data-number="22.4">
<h2 data-number="22.4" class="anchored" data-anchor-id="what-makes-us-human"><span class="header-section-number">22.4</span> What Makes Us Human?</h2>
<p>At the end of this chapter, you might be wondering what it really is that make us human, different from all the other creatures with which we share the planet. Scientists and philosophers have pondered this question for millennia. If you ask a biologist, they are likely to talk about how the anatomy of our hand, especially an additional muscle that facilitates additional mobility of the thumb, is very different from other apes and facilitated our extensive use of tools. Or they might highlight the peculiar shape of our hyoid bone, which allows us to articulate a much wider variety of sounds and likely facilitated the evolution of our complex languages. And they will surely highlight the magnificent complexity of our large brains, which make us exquisite learners. Because of our brain, information is not just passed across generations through our DNA, but also through cultural practices that play an important role in our survival and social interactions. It’s this magnificent brain that made us into the ultimate survival machine and the most invasive species on the planet. It made us into a species that appreciates art, ponders the meaning of life, and looks beyond Earth into the depths of the universe.</p>
<p>But who are we to say that no other animals share some of these same experiences? Certainly other creatures use tools and communicate in quite sophisticated ways. Elephants that carefully examine the remains of friends and family may experience feelings of grief, just like us. And who knows how many creatures look at the stars in the night sky, feeling humbled by how small and insignificant they are in the big scheme of things, and wondering why they are really there…</p>
<p>I will offer you my very own and very biased perspective of what makes us human: our ability to use fire and cook our food. No other species on the planet processes food quite the same way we do, and some researchers have speculated that food processing allowed us to extract more nutrients from our food items, thus facilitating the evolution of our costly brains… and everything that comes along with them.</p>
<p>So, I will end this book about evolution with a simple recipe—from my ancestors to you. Cook it for a loved one—or just for yourself, you deserve it—and I promise you will have a quintessential human experience in the process.</p>
<div class="alert alert-block alert-info">
<p><b>Pasta Insieme A Sugo Di Pomodoro</b></p>
<p>Yes… it’s just pasta with tomato sauce, but it is 🤤</p>
<p><strong><em>Sugo</em></strong></p>
<ul>
<li><p>4 tablespoons of olive oil</p></li>
<li><p>8 cloves of garlic, finely chopped</p></li>
<li><p>2 pounds of fresh tomatoes (get some of those sweet heirlooms if you can)</p></li>
<li><p>A dozen or two basil leaves, finely cut</p></li>
<li><p>Salt and pepper to taste</p></li>
</ul>
<p>Bring some water to a boil in a sauce pan, add the tomatoes, and simmer for 3 minutes; then remove the tomatoes, peel the skin, remove the seeds, and cut them into small pieces. Then heat up the olive oil and carefully brown the garlic (slow… don’t burn it), add the tomatoes, and let it simmer for about 20 minutes. Add salt and pepper to taste, and add the basil just before you are ready to serve.</p>
<p><strong><em>Pasta</em></strong></p>
<ul>
<li><p>1.5 cups of wheat flour</p></li>
<li><p>1.5 cups of semola flour</p></li>
<li><p>4 large eggs</p></li>
</ul>
<p>Sift the two flours together into a bowl, make a hole in the center with your hand, and add the eggs into the hole; then use your fingers mix the eggs with the flour (go slow, incorporating more and more flour as you go). Once the dough is uniform, knead it for 10 minutes (if its hard to knead because the dough is too dry, add a little water; if the dough is sticky, add a little flour); then make a ball, wrap it in plastic wrap, and let it rest for 30 minutes. After that, you can roll your dough and cut your noodles using a pasta maker, or you can use a rolling pin and a sharp knife instead. To cook the pasta, add it to a pot of well-salted boiling water for about 2 minutes.</p>
<p><strong><em>Serve</em></strong></p>
<p>Drain the pasta and add it to bowl, top it with your freshly-made sugo, and shave some Parmesan cheese on top. My ancestors typically enjoyed this with a glass of red wine:)</p>
</div>
</section>
<section id="case-study-modern-human-origins" class="level2" data-number="22.5">
<h2 data-number="22.5" class="anchored" data-anchor-id="case-study-modern-human-origins"><span class="header-section-number">22.5</span> Case Study: Modern Human Origins</h2>
<p>In <a href="exercises/BIOL520-ex13.zip">the case study associated with this chapter</a>, you will contrast different hypotheses of modern human origins. You have already learned all R skills required for the completion of this exercise.</p>
</section>
<section id="reflection-questions" class="level2" data-number="22.6">
<h2 data-number="22.6" class="anchored" data-anchor-id="reflection-questions"><span class="header-section-number">22.6</span> Reflection Questions</h2>
<ol type="1">
<li><p>Is it accurate to say that humans evolved from apes?</p></li>
<li><p>What are some of the limitations inherent to information derived from the fossil record? What questions about human evolution are not addressable by looking at fossils alone?</p></li>
<li><p>What are the key differences between the alternative hypotheses of modern human origins, how can they be tested, and what evidence do we have for and against each of them?</p></li>
<li><p>The out-of-Africa hypothesis postulates that non-African humans are derived from African populations. A common misconception about this idea is that African peoples are consequently more “primitive” than those from other geographic regions. Can you explain the flaw in this argument?</p></li>
<li><p>Why do you think we have such great coverage of ancient DNA samples from Eurasia but comparatively few from other geographic regions?</p></li>
<li><p>Do you think humans are still evolving today? If so, how?</p></li>
</ol>
</section>
<section id="references" class="level2" data-number="22.7">
<h2 data-number="22.7" class="anchored" data-anchor-id="references"><span class="header-section-number">22.7</span> References</h2>
<ul>
<li><p>Chatterjee HJ, Ho SYW, Barnes I, Groves C (2009). <a href="https://bmcecolevol.biomedcentral.com/articles/10.1186/1471-2148-9-259">Estimating the phylogeny and divergence times of primates using a supermatrix approach</a>. <em>BMC Evolutionary Biolology</em> 9: 259.</p></li>
<li><p>Du A, Zipkin AM, Hatala KG, et al (2018). <a href="https://doi.org/10.1098/rspb.2017.2738">Pattern and process in hominin brain size evolution are scale-dependent</a>. <em>Proceedings of the Royal Society B</em> 285. 20172738:</p></li>
<li><p>Harris K, Nielsen R (2016) <a href="https://www.genetics.org/content/203/2/881">The genetic cost of Neanderthal introgression</a>. <em>Genetics</em> 203: 881–891.</p></li>
<li><p>Hawks J, Elliott M, Schmid P, et al (2017). <a href="https://doi.org/10.7554/eLife.24232">New fossil remains of <em>Homo naledi</em> from the Lesedi Chamber, South Africa</a>. <em>eLife</em> 6: e24232.</p></li>
<li><p>Ingman M, Kaessmann H, Pääbo S, Gyllensten U (2000). <a href="https://www.nature.com/articles/35047064">Mitochondrial genome variation and the origin of modern humans</a>. <em>Nature</em> 408:708–713</p></li>
<li><p>Krings M, Stone A, Schmitz RW, et al (1997). <a href="https://www.cell.com/fulltext/S0092-8674(00)80310-4">Neandertal DNA sequences and the origin of modern humans</a>. <em>Cell</em> 90: 19–30.</p></li>
<li><p>Li JZ, Absher DM, Tang H, et al (2008) <a href="https://www.science.org/doi/10.1126/science.1153717">Worldwide human relationships inferred from genome-wide patterns of variation</a>. <em>Science</em> 319:1100–1104</p></li>
<li><p>Masao FT, Ichumbaki EB, Cherin M, Barili A, Boschain G, Iurino DA, Menconero S, Moggi-Cecchi J, Manzi G (2016). <a href="https://doi.org/10.7554/eLife.19568">New footprints from Laetoli (Tanzania) provide evidence for marked body size variation in early hominins</a>. <em>eLife</em> 5: e19568.</p></li>
<li><p>Meyer M, Fu Q, Aximu-Petri A, et al (2014). <a href="https://www.nature.com/articles/nature12788">A mitochondrial genome sequence of a hominin from Sima de los Huesos</a>. <em>Nature</em> 505: 403–406.</p></li>
<li><p>Prüfer K, Racimo F, Patterson N, et al (2014) <a href="https://www.nature.com/articles/nature12886">The complete genome sequence of a Neanderthal from the Altai Mountains</a>. <em>Nature</em> 505: 43–49.</p></li>
<li><p>Sala N, Arsuaga JL, Pantoja-Pérez A, et al (2015) <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0126589">Lethal interpersonal violence in the Middle Pleistocene</a>. <em>PLoS One</em> 10: e0126589.</p></li>
<li><p>Seymour RS, Bosiocic V, Snelling EP (2016). <a href="https://royalsocietypublishing-org.er.lib.k-state.edu/doi/10.1098/rsos.160305">Fossil skulls reveal that blood flow rate to the brain increased faster than brain volume during human evolution</a>. <em>Royal Society Open Scienc</em>e 3: 160305.</p></li>
<li><p>Tishkoff SA, Dietzsch E, Speed W, et al (1996). <a href="https://www.science.org/doi/abs/10.1126/science.271.5254.1380">Global patterns of linkage disequilibrium at the CD4 locus and modern human origins</a>. <em>Science</em> 271: 1380–1387.</p></li>
<li><p>van der Valk T, Pečnerová P, Díez-Del-Molino D, et al (2021). <a href="https://www.nature.com/articles/s41586-021-03224-9">Million-year-old DNA sheds light on the genomic history of mammoths</a>. <em>Nature</em> 591: 265–269.</p></li>
<li><p>Vyas DN, Mulligan CJ (2019). <a href="https://onlinelibrary.wiley.com/doi/10.1002/ajpa.23818">Analyses of Neanderthal introgression suggest that Levantine and southern Arabian populations have a shared population history</a>. <em>American Journal of Physical Anthropology</em> 169: 227–239.</p></li>
<li><p>Ward CV, Plavcan JM, Manthi FK (2010). <a href="https://royalsocietypublishing.org/doi/10.1098/rstb.2010.0039">Anterior dental evolution in the <em>Australopithecus anamensis-afarensis</em> lineage</a>. <em>Philosophical Transactions of the Royal Society London B</em> 365: 3333–3344.</p></li>
<li><p>Wood B, Grabowski M (2015). <a href="https://link.springer.com/chapter/10.1007/978-3-319-15045-1_11">Macroevolution in and around the hominin clade</a>. In: Serrelli E, Gontier N (eds) <em>Macroevolution: Explanation, Interpretation and Evidence</em>. Springer International Publishing, Cham, pp 345–376.</p></li>
</ul>


</section>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="./DivLife.html" class="pagination-link" aria-label="Diversification and patterns of life through time">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">20</span>&nbsp; <span class="chapter-title">Diversification and patterns of life through time</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="./EvolMed.html" class="pagination-link" aria-label="Evolutionary Medicine">
        <span class="nav-page-text"><span class="chapter-number">22</span>&nbsp; <span class="chapter-title">Evolutionary Medicine</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->




</body></html>