<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.5.57">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>10&nbsp; Mechanisms that drive evolution: natural selection – Evolution 303</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<link href="./MechEvolMutation.html" rel="next">
<link href="./MechEvolDrift.html" rel="prev">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>

  <script src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6"></script>
  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

<script type="text/javascript">
const typesetMath = (el) => {
  if (window.MathJax) {
    // MathJax Typeset
    window.MathJax.typeset([el]);
  } else if (window.katex) {
    // KaTeX Render
    var mathElements = el.getElementsByClassName("math");
    var macros = [];
    for (var i = 0; i < mathElements.length; i++) {
      var texText = mathElements[i].firstChild;
      if (mathElements[i].tagName == "SPAN") {
        window.katex.render(texText.data, mathElements[i], {
          displayMode: mathElements[i].classList.contains('display'),
          throwOnError: false,
          macros: macros,
          fleqn: false
        });
      }
    }
  }
}
window.Quarto = {
  typesetMath
};
</script>

</head>

<body class="nav-sidebar floating">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" role="button" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
        <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./GenesMutationGeneticVariation.html">Evolutionary Genetics</a></li><li class="breadcrumb-item"><a href="./MechEvolSelection.html"><span class="chapter-number">10</span>&nbsp; <span class="chapter-title">Mechanisms that drive evolution: natural selection</span></a></li></ol></nav>
        <a class="flex-grow-1" role="navigation" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
        </a>
      <button type="button" class="btn quarto-search-button" aria-label="Search" onclick="window.quartoOpenSearch();">
        <i class="bi bi-search"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal quarto-sidebar-collapse-item sidebar-navigation floating overflow-auto">
    <div class="pt-lg-2 mt-2 text-left sidebar-header">
    <div class="sidebar-title mb-0 py-0">
      <a href="./">Evolution 303</a> 
    </div>
      </div>
        <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
        </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Evolution 303</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./Preface.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Preface - Evolution 303</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" role="navigation" aria-expanded="true">
 <span class="menu-text">Basic Concepts</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./WhatEvolutionIs.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">What evolution is</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./HistoricaPerspective.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">A brief history of evolutionary thought</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./EvidenceForEvolution.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Evidence for Evolution</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./HowNaturalSelectionWorks.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">How Natural Selection Works</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" role="navigation" aria-expanded="true">
 <span class="menu-text">Working With Evolutionary Trees</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-2" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./UnderstandingEvolTrees.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Understanding Evolutionary Trees</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./EstimatingEvoTrees.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Estimating evolutionary trees</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" role="navigation" aria-expanded="true">
 <span class="menu-text">Evolutionary Genetics</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-3" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./GenesMutationGeneticVariation.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Genes, mutation, and genetic variation</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./IntroEvolGenetics.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">Introduction: Evolutionary genetics</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./MechEvolDrift.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">Genetic Drift</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./MechEvolSelection.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text"><span class="chapter-number">10</span>&nbsp; <span class="chapter-title">Mechanisms that drive evolution: natural selection</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./MechEvolMutation.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">11</span>&nbsp; <span class="chapter-title">Mutation</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./MechEvolMigration.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">12</span>&nbsp; <span class="chapter-title">Migration and other factors</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./EvolPhenotype.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">13</span>&nbsp; <span class="chapter-title">The Evolution of Quantitative Traits</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-4" role="navigation" aria-expanded="true">
 <span class="menu-text">In Light Of Evolution</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-4" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-4" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./KinAndSexualSelection.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">14</span>&nbsp; <span class="chapter-title">Social Behavior and Sexual Selection</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./Adaptations.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">15</span>&nbsp; <span class="chapter-title">Adaptation and Phenotypic Plasticity</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./SpeciesSpeciation.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">16</span>&nbsp; <span class="chapter-title">Species and Speciation</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./EvolutionGenomicLevel.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">17</span>&nbsp; <span class="chapter-title">Evolution at the genomic level</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./EvoDevo.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">18</span>&nbsp; <span class="chapter-title">EvoDevo</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-5" role="navigation" aria-expanded="true">
 <span class="menu-text">Evolution At The Macro Level</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-5" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-5" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./EvoBiogeogrphy.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">19</span>&nbsp; <span class="chapter-title">Evolutionary Biogeography</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./DivLife.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">20</span>&nbsp; <span class="chapter-title">Diversification and patterns of life through time</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-6" role="navigation" aria-expanded="true">
 <span class="menu-text">What About Us</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-6" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-6" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./HumanEvol.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">21</span>&nbsp; <span class="chapter-title">Human Evolution</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./EvolMed.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">22</span>&nbsp; <span class="chapter-title">Evolutionary Medicine</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./EvolMedPartI.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">23</span>&nbsp; <span class="chapter-title">Evolutionary Medicine I: Aging and Diseases of Civilization</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./EvolMedPartII.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">24</span>&nbsp; <span class="chapter-title">Evolutionary Medicine II: Evolving Pathogens</span></span></a>
  </div>
</li>
      </ul>
  </li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" class="quarto-sidebar-collapse-item" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#the-effects-of-selection" id="toc-the-effects-of-selection" class="nav-link active" data-scroll-target="#the-effects-of-selection"><span class="header-section-number">10.1</span> The Effects of Selection</a>
  <ul class="collapse">
  <li><a href="#frequency-dependent-selection" id="toc-frequency-dependent-selection" class="nav-link" data-scroll-target="#frequency-dependent-selection"><span class="header-section-number">10.1.1</span> Frequency-Dependent Selection</a></li>
  </ul></li>
  <li><a href="#modeling-the-effects-of-selection" id="toc-modeling-the-effects-of-selection" class="nav-link" data-scroll-target="#modeling-the-effects-of-selection"><span class="header-section-number">10.2</span> Modeling the Effects of Selection</a>
  <ul class="collapse">
  <li><a href="#relative-fitness" id="toc-relative-fitness" class="nav-link" data-scroll-target="#relative-fitness"><span class="header-section-number">10.2.1</span> Relative Fitness</a></li>
  <li><a href="#a-generalized-model-for-studying-selection" id="toc-a-generalized-model-for-studying-selection" class="nav-link" data-scroll-target="#a-generalized-model-for-studying-selection"><span class="header-section-number">10.2.2</span> A generalized model for studying selection</a></li>
  <li><a href="#deriving-a-generalized-equation-for-selection" id="toc-deriving-a-generalized-equation-for-selection" class="nav-link" data-scroll-target="#deriving-a-generalized-equation-for-selection"><span class="header-section-number">10.2.3</span> Deriving a generalized equation for selection</a></li>
  <li><a href="#a-diversity-of-selection-models" id="toc-a-diversity-of-selection-models" class="nav-link" data-scroll-target="#a-diversity-of-selection-models"><span class="header-section-number">10.2.4</span> A diversity of selection models</a></li>
  </ul></li>
  <li><a href="#next-step" id="toc-next-step" class="nav-link" data-scroll-target="#next-step"><span class="header-section-number">10.3</span> Next Step</a></li>
  <li><a href="#model-results" id="toc-model-results" class="nav-link" data-scroll-target="#model-results"><span class="header-section-number">10.4</span> Model Results</a></li>
  <li><a href="#drift-and-selection" id="toc-drift-and-selection" class="nav-link" data-scroll-target="#drift-and-selection"><span class="header-section-number">10.5</span> Drift and Selection</a>
  <ul class="collapse">
  <li><a href="#interactions-between-drift-and-selection" id="toc-interactions-between-drift-and-selection" class="nav-link" data-scroll-target="#interactions-between-drift-and-selection"><span class="header-section-number">10.5.1</span> Interactions Between Drift and Selection</a></li>
  </ul></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default"><nav class="quarto-page-breadcrumbs quarto-title-breadcrumbs d-none d-lg-block" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./GenesMutationGeneticVariation.html">Evolutionary Genetics</a></li><li class="breadcrumb-item"><a href="./MechEvolSelection.html"><span class="chapter-number">10</span>&nbsp; <span class="chapter-title">Mechanisms that drive evolution: natural selection</span></a></li></ol></nav>
<div class="quarto-title">
<h1 class="title"><span id="sec-mechselect" class="quarto-section-identifier"><span class="chapter-number">10</span>&nbsp; <span class="chapter-title">Mechanisms that drive evolution: natural selection</span></span></h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<p><em>DRAFT 20250404</em></p>
<p>The Hardy-Weinberg principle we learned about in the last chapter provides a simple framework to test whether evolutionary forces are acting on a locus within a population. If a population is in Hardy-Weinberg disequilibrium at a particular locus, we know that an evolutionary force is skewing relative genotype frequencies away from what is predicted based on the allele frequencies in a population, and that allele frequencies may change over generations. Evolutionary forces might include selection, mutation, genetic drift, migration, or non-random mating. In this and the following chapters, we will get to know the different evolutionary forces in more detail, and we will explore how each force—and interactions between them—shapes genetic variation and allele frequencies across generations. To do so, we will use simple mathematical models to simulate allele frequency changes under different scenarios. First, we will learn how the outcome of selection varies depending on the fitness distribution among genotypes and the starting allele frequency in a population. In the following chapters we will expand this simplified view and also include mutation, genetic drift, migration, and non-random mating in our thinking.</p>
<section id="the-effects-of-selection" class="level2" data-number="10.1">
<h2 data-number="10.1" class="anchored" data-anchor-id="the-effects-of-selection"><span class="header-section-number">10.1</span> The Effects of Selection</h2>
<p>Rosemary and Peter Grant’s work has shown us how natural selection can drive rapid evolution of phenotypic traits. But what are the genetic ramifications of selection? How does selection impact genetic variation and allele frequencies at a particular locus? Per definition, we know that selection favors the success of certain genotypes over others; this is one way Hardy-Weinberg disequilibrium can be generated. It is this biased success of certain genotypes that ultimately causes allele frequency changes across generations. For example, if the fitness of an individual is contingent on the number of <em>A</em> alleles at a particular locus, we would expect the <em>A</em> allele to become more common than the <em>a</em> allele through successive generations.</p>
<p>Such allele frequency changes across generations are well documented from evolution experiments. For example, Dawson (1970) conducted research on flour beetles (<em>Tribolium castaneum</em>). In his colonies, he observed a recessive deleterious mutation, where individuals with two copies of the recessive allele (<em>l</em>) could not survive, but heterozygotes (<em>wt</em>/<em>l</em>) and individuals that were homozygous for the the wildtype (<em>wt</em>) allele had normal fitness. This represents a classical dominant-recessive inheritance. To test the evolutionary consequences of the deleterious recessive allele, Dawson assembled experimental populations composed solely of heterozygous individuals, which means that the allele frequencies of <em>wt</em> and <em>l</em> were both 0.5. He then let the populations evolve for several generations and measured the allele frequencies of <em>wt</em> and <em>l</em> in each generation. As you can see in Figure @ref(fig:recdel), the allele frequency of <em>wt</em> increased in each generation. Since <em>p</em>+<em>q</em>=1 for any biallelic locus, the frequency of <em>l</em> decreased concurrently. Hence, selection against the recessive lethal allele caused it to decline in frequency over time. This makes intuitive sense, considering the offspring of the heterozygous individuals of the initial population were 25 % <em>wt/wt</em> (<em>p</em><sup>2</sup>), 50 % heterozygous <em>wt</em>/<em>l</em> (2<em>pq</em>), and 25 % <em>l</em>/<em>l</em> (<em>q</em><sup>2</sup>). While all the individuals with at least one <em>wt</em> allele survived and reproduced, individuals that were homozygous for the recessive allele perished, disproportionately removing <em>l</em> alleles from the population.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="MechEvolSelection_files/figure-html/recdel-1.png" class="img-fluid figure-img" width="672"></p>
<figcaption>Selection against a deleterious recessive mutation causes significant changes in allele frequencies over short periods of time. <a href="data/5_lethal_recessive.csv">Data</a> from Dawson (1970).</figcaption>
</figure>
</div>
</div>
</div>
<p>Similar changes in allele frequencies can be observed if populations that are polymorphic at a particular locus are exposed to different environmental conditions. For example, many fruit fly (<em>Drosophila melanogaster</em>) populations are polymorphic at the <em>Adh</em> locus, which encodes alcohol dehydrogenase, the same enzyme that detoxifies ethanol in your liver after a fine glass of wine. The two alleles in fruit flies differ in the rate at which they process ethanol, including a fast allele (<em>Adh</em><sup>f</sup>) and a slow allele (<em>Adh</em><sup>s</sup>). Cavener and Clegg (1981) used populations with this polymorphism, letting different replicate populations evolve on media that contained alcohol, or in non-alcoholic control populations. While the control populations only exhibited minor fluctuations in allele frequencies across generations, the frequency of the <em>Adh</em><sup>f</sup> allele significantly increased in populations exposed to ethanol (Figure @ref(fig:ethanol)). The difference in evolutionary trajectories can be explained by the fitness benefit the <em>Adh</em><sup>f</sup> allele confers in different environments: when ethanol is present, having faster detoxification is clearly advantageous. But if there is no ethanol in the environment, it does not matter whether you can detoxify it faster or slower, and genetic variation at the <em>Adh</em> locus becomes neutral.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="MechEvolSelection_files/figure-html/ethanol-1.png" class="img-fluid figure-img" width="672"></p>
<figcaption>Allele frequency changes at the Adh locus in ethanol-exposed and control populations. Allele frequencies change drastically in populations that are exposed to the critical source of selection, ethanol. <a href="data/5_ethanol.csv">Data</a> from Cavener and Clegg (1981).</figcaption>
</figure>
</div>
</div>
</div>
<p>Evidence for change in allele frequencies in response to selection does not just come from laboratory experiments, but also from natural populations where time series of the genotypic composition of populations are available. One of the most amazing data sets in recent years comes from over 540 whole or partial genome sequences that were obtained from fossilized human remains—some of which are almost 14,000 years old. This time series spans some major transitions in human evolution, including the advent of agriculture and correlated changes in human diets that occurred about 10,000 years ago. Around that time, humans also started to domesticate livestock for meat and milk production. Many of these changes represented novel sources of selection, especially the availability of milk as a dietary component past weaning and into adulthood. As much as mammals rely on milk for nutrition right after birth, this food source is not really available to any species after weaning. Accordingly, the production of lactase—the enzyme that mediates digestion of milk sugar (lactose)—stops after weaning, leaving adult mammals unable to digest lactose. Some of you may be familiar with that problem. Depending on your heritage, you may feel significant discomfort after eating dairy, because your body cannot handle the lactose (this is known as lactose intolerance). It turns out that lactose intolerance is prevalent in human populations that do not have a cultural history of using domesticated animals for milk production. This raises the question of how lactose tolerance evolved in populations that do have a history of dairy use. Analyses of ancient DNA indicated that alleles conferring lactose tolerance (also known as lactase persistence alleles) were virtually absent in human populations prior to the domestication of dairy animals (Figure @ref(fig:lactase)). In European populations with a history of dairy farming, lactase persistence alleles did not start to spread until about 5,000 years ago, matching the timeline of domestication. Overall, empirical evidence from experiments and natural systems indicates that selection can have profound impacts on allele frequencies even over relatively short time frames.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="MechEvolSelection_files/figure-html/lactase-1.png" class="img-fluid figure-img" width="672"></p>
<figcaption>Analyses of ancient DNA indicates an increase in the frequency of lactase persistence alleles starting about 5,000 years ago. Different geographic regions are color-coded and the size of markers is proportional to the number of genomes used to infer allele frequences. <a href="data/5_lactase.csv">Data</a> from Marciniak &amp; Perry (2017).</figcaption>
</figure>
</div>
</div>
</div>
<section id="frequency-dependent-selection" class="level3" data-number="10.1.1">
<h3 data-number="10.1.1" class="anchored" data-anchor-id="frequency-dependent-selection"><span class="header-section-number">10.1.1</span> Frequency-Dependent Selection</h3>
<p>While evolution experiments usually create conditions where the direction and strength of selection remains constant, selection in natural populations often varies spatially or temporally. For example, fluctuating selection can be associated with environmental change, which was exemplified in Darwin’s finches, whose beaks can evolve to be larger or smaller dependent on the available food resources. A special case of fluctuating selection is <strong>frequency-dependent selection</strong>, where the fitness of a genotype depends on the genotypic composition of the population. We distinguish two types of frequency-dependency: positive and negative.</p>
<p>In <strong>positive frequency-dependent selection</strong>, the fitness of a genotype increases as it becomes more common in a population, and consequently, selection rapidly drives common alleles to fixation. Positive frequency-dependent selection plays a major role in the evolution of signaling traits, where the efficiency of a signal is dependent on the frequency of its use. Notable examples include the evolution of social signals, flower coloration for pollinator attraction, and warning signals that indicate danger or unpalatability.</p>
<p>A great example of positive frequency-dependent selection includes selection on warning signals in toxic, South American butterflies of the genus <em>Heliconius</em>. Some <em>Heliconius</em> species exhibit extensive geographic variation in their warning colors, but different species in the same geographic region are often strikingly similar (Figure @ref(fig:heliconius)). This is because different species in the same region mimic each other, developing shared signals to deter predators. Similar signals mean that predators are more likely to recognize the pattern of unpalatability. This is similar to the independent evolution of yellow and black stripes in wasps, bees, and many other insects that signal danger.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/Heliconius_mimicry.png" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption>Mimicry in <em>Heliconius</em> butterflies. Specimens in the same row belong to the same species (top: <em>H. melpomene</em>; bottom: <em>H. erato</em>). Specimens in the same column were collected in the same geographic region. Image from Meyer (2006), <a href="https://creativecommons.org/licenses/by/2.5">CC BY 2.5</a>, via Wikimedia Commons</figcaption>
</figure>
</div>
</div>
</div>
<p>Chouteau et al.&nbsp;(2016) tested whether positive frequency-dependent selection could explain the evolution of mimicry in <em>Heliconius</em> butterflies. They used prey models and placed them in different <em>Heliconius</em> populations that varied in the relative frequencies of different warning colorations, predicting that the rate of predation would be low if models matched the common warning phenotype. Indeed, models that resembled rare morphs were much more likely to be attacked by predators than models that resembled the common morph (Figure @ref(fig:posfreqdep)). In other words, the more common a particular morph is, the less likely it is to succumb to a predator, and consequently, it will have a higher fitness compared to the rarer morphs. Over the course of multiple generations, such positive frequency dependent selection can generate the evolution of coordinated warning signals across species inhabiting the same region.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="MechEvolSelection_files/figure-html/posfreqdep-1.png" class="img-fluid figure-img" width="672"></p>
<figcaption>Positive frequency dependent selection. <a href="data/5_positivefreqdep.csv">Data</a> from Chouteau et al.&nbsp;(2016).</figcaption>
</figure>
</div>
</div>
</div>
<p>In <strong>negative frequency-dependent selection</strong>, the fitness of a genotype declines as it becomes more common. Negative frequency-dependency is a mechanism by which genetic variation is maintained in a population, and it can be an important evolutionary force acting on a wide variety of traits. Perhaps most well-known is how negative frequency-dependence shapes the evolution of many traits associated with host-pathogen interactions. As certain host defense strategies become more common, pathogens rapidly evolve to adapt to those defenses. As a consequence, individuals with more common defense strategies exhibit lower fitness than rare ones, leading to fluctuations in genotype frequencies over time.</p>
<p>Another great example of negative frequency-dependent selection comes from the orchid <em>Dactylorhiza sambucina</em>, which exhibits a striking polymorphism in flower coloration (Figure @ref(fig:orchid)). While many plant communities are subject to positive frequency-dependent selection to coordinate signals for pollinator attraction, <em>Dactylorhiza</em> cheats: unlike most flowers, those of <em>Dactylorhiza</em> do not provide any rewards to pollinators in the form of nectar. As a consequence, pollinators learn to avoid <em>Dactylorhiza</em> flowers.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/Dactylorhiza_sambucina.jpg" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption><em>Dactylorhiza sambucina</em> (yellow and purple/red forms growing together) in Andorra. Photo by Strobilomyces, <a href="https://creativecommons.org/licenses/by-sa/3.0">CC BY-SA 3.0</a>, via Wikimedia Commons.</figcaption>
</figure>
</div>
</div>
</div>
<p><em>Dactylorhiza</em>’s counter-adaptation to this learned avoidance is a flower color polymorphism, which is thought to be maintained by negative frequency-dependent selection. The idea is that if one color morph gets too abundant and pollinators start avoiding it, then they simply switch to the rarer color morph that they do not associate with a lack of reward yet. If this was correct, the fitness of a color morph should decline with its frequency. Gigord et al.&nbsp;(2001) tested this idea by conducting an experiment where they varied the relative frequency of different color morphs and quantified reproductive success. As predicted, male reproductive success was negatively correlated with relative frequency of the morph (Figure @ref(fig:negfreqdep)), indicating a rare morph advantage that is characteristic for negative frequency-dependent selection.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="MechEvolSelection_files/figure-html/negfreqdep-1.png" class="img-fluid figure-img" width="672"></p>
<figcaption>As predicted by negative frequency-dependent selection, reproductive success in <em>Dactyorhiza</em> color morphs is negatively correlated with the frequency of the morph in the population. <a href="data/5_negfreqdep.csv">Data</a> from Gigord et al.&nbsp;(2001).</figcaption>
</figure>
</div>
</div>
</div>
</section>
</section>
<section id="modeling-the-effects-of-selection" class="level2" data-number="10.2">
<h2 data-number="10.2" class="anchored" data-anchor-id="modeling-the-effects-of-selection"><span class="header-section-number">10.2</span> Modeling the Effects of Selection</h2>
<p>While the consequences of selection in terms of changing allele frequencies may seem trivial at first sight, the exact outcome of selection actually depends on a number of parameters, including the starting allele frequency and the fitness distribution across different genotypes. To develop a nuanced understanding of alternative evolutionary outcomes in response to selection, we will use simple mathematical models to explore how different starting conditions shape evolutionary trajectories. In the following sections we will explore the conceptual background on how these mathematical models work. Then you will learn how you can easily run different models in lab.</p>
<section id="relative-fitness" class="level3" data-number="10.2.1">
<h3 data-number="10.2.1" class="anchored" data-anchor-id="relative-fitness"><span class="header-section-number">10.2.1</span> Relative Fitness</h3>
<p>Modeling changes in allele frequencies in response to selection requires a modification of the formulas associated with the Hardy-Weinberg principle. When calculating allele frequencies across generations, we have to take into account fitness differences between the genotypes. While fitness is typically measured through a variety of proxies in natural populations (<em>e.g.</em>, survival, reproductive success, growth, etc.), having species- or even study-specific metrics for fitness is not conducive to mathematical modeling. So, rather than considering different fitness components, we can just subsume different metrics into a single, idealized measure of fitness, which is called <em>relative fitness.</em> Relative fitness expresses the fitness of different genotypes relative to each other, and it can be any positive number, including zero. In practice, we choose one genotype as a reference and set its relative fitness to 1 (representing 100 % fitness). The fitness of other genotypes can then be expressed in relation to the reference. Fitness values are higher than the reference (&gt;1) if alternate genotypes have higher fitness and lower than the reference (&lt;1) if they have lower fitness. The difference in relative fitness between genotypes is called the selection coefficient (<em>s</em>), indicating how fitness and selection are directly related to each other.</p>
</section>
<section id="a-generalized-model-for-studying-selection" class="level3" data-number="10.2.2">
<h3 data-number="10.2.2" class="anchored" data-anchor-id="a-generalized-model-for-studying-selection"><span class="header-section-number">10.2.2</span> A generalized model for studying selection</h3>
<p>To begin our exploration of selection, we will first develop a generalized model that will act the basis to conveniently explore the various ways selection can work. Assuming the following:</p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Assumptions of the population genetic models
</div>
</div>
<div class="callout-body-container callout-body">
<ul>
<li>diploid organism</li>
<li>sexually reproducing</li>
<li>single locus (gene) - call it the <strong>A</strong> gene</li>
<li>two alleles segregating in the population designated <span class="math inline">\(A_1\)</span> and <span class="math inline">\(A_2\)</span></li>
</ul>
<p>NOTE: Do not be confused by allele names. They are completely arbritrary for the most part. For example, if we are discussing two alleles at a gene, we can refer to them as <span class="math inline">\(A_1\)</span> and <span class="math inline">\(A_2\)</span>, or <span class="math inline">\(A\)</span> and <span class="math inline">\(a\)</span>, or <span class="math inline">\(George\)</span> ad <span class="math inline">\(Martha\)</span>. It really does not matter. I prefer <span class="math inline">\(A_1\)</span> and <span class="math inline">\(A_2\)</span> for this discussion since it tends to sidestep any <em>a priori</em> assumption about dominant/recessive relationship between the alleles. I have found the concept of dominant/recessive relationship between the alleles to be a major stumbling point for many students, probably owing to the way genetics is first introduced via Mendel. We will discuss this idea a little later as we explore various models of selection. For now, consider both alleles are equally expressed, and therfore there is no dominant/recessive relationship</p>
</div>
</div>
<p>Referring back to the chapter introducing evolutionary genetics and the HW equilibrium (<a href="IntroEvolGenetics.html" class="quarto-xref"><span>Chapter 8</span></a>), there are three possible genotypes in this model due to random mating within the populaton: <span class="math inline">\(A_1A_1\)</span>, <span class="math inline">\(A_1A_2\)</span>, and <span class="math inline">\(A_2A_2\)</span>, which under our null model (HW equil) will occur in the following frequences: <span class="math inline">\(p^2\)</span>, <span class="math inline">\(2pq\)</span>, and <span class="math inline">\(q^2\)</span>.</p>
<p>Since we are modelling selection, we also need to include some measure of fitness, specifically a genotype-specific measure of relative fitness, which we will designate <span class="math inline">\(W_{ii}\)</span> where the <span class="math inline">\(ii\)</span> indicates any possible genotype. Because they are relative fitnesses, for our purposes we will define the fitness coefficients <span class="math inline">\(W_{ii}\)</span> as ranging between <span class="math inline">\(0\longrightarrow1.0\)</span>. As mentioned above, we could have just as easily randomly assigned any genotype as our reference such that you could have a genotypic fitness <span class="math inline">\(W_{ii}&gt;1.0\)</span>, but for reasons of clarity I prefer setting our favored genotype as our reference such that the <span class="math inline">\(W_{ii}=0\longrightarrow1.0\)</span>.</p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Genotype specific fitness coefficients
</div>
</div>
<div class="callout-body-container callout-body">
<p>Recall our definition of fitness <a href="HowNaturalSelectionWorks.html#nte-fitness" class="quarto-xref">Note&nbsp;<span>4.1</span></a> from the chater on how selection works. A good way to think about what these genotype fitness coefficients mean is <em>what percentage of individual with a specific genotype survive and produce to the next generation</em>. For example, if all the individuals in our sample of a particular genotype (<em>e.g.</em> <span class="math inline">\(A_1A_1\)</span>) survive and reproduce, we would assign a genotype specific fitness to that genotype as <span class="math inline">\(W_{11} = 1.0\)</span>. If however, we were to specify that only 90% of the individuals with that genotype got to survive and reproduce, than the genotype specific fitness would be <span class="math inline">\(W_{11} = 0.9\)</span>.</p>
<p>These genotype specific fitness coefficients are obviously related to selection, so we define <span class="math inline">\(s\)</span> as the selection coefficient such that</p>
<p><span class="math inline">\(W = 1-s\)</span></p>
<p>Another way to specify fitness coefficients then is as selection coefficients <span class="math inline">\(s\)</span>. As above, if we stated that 10% of the individual in our population failed to survive or reproduce, then when <span class="math inline">\(s=0.10\)</span>, <span class="math inline">\(W_{ii} = 1-s = 0.9\)</span>.</p>
<p>This is a concenient way to talk about the various types of selection, as you will see, and we will try to remain as consistent as possible using this way to discuss genotype specific fitness and selection coefficients throughout the rest of the this text.</p>
</div>
</div>
<p>Summarizing as a table, under the above generalized conditions, we can now ask the question <em>what happens to the allele frequencies when selection is acting on the genotypes</em>? This leads to following table:</p>
<div id="tbl-selectmod" class="striped hover quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-selectmod-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;10.1: Generalized model for selection
</figcaption>
<div aria-describedby="tbl-selectmod-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<table class="table-striped table-hover caption-top table">
<colgroup>
<col style="width: 20%">
<col style="width: 20%">
<col style="width: 20%">
<col style="width: 20%">
<col style="width: 20%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">Genotype</th>
<th style="text-align: center;"><span class="math inline">\(A_1A_1\)</span></th>
<th style="text-align: center;"><span class="math inline">\(A_1A_2\)</span></th>
<th style="text-align: center;"><span class="math inline">\(A_2A_2\)</span></th>
<th style="text-align: center;">Sum</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">freq <span class="math inline">\(f_{ii}\)</span></td>
<td style="text-align: center;"><span class="math inline">\(p^2\)</span></td>
<td style="text-align: center;"><span class="math inline">\(2pq\)</span>.</td>
<td style="text-align: center;"><span class="math inline">\(q^2\)</span></td>
<td style="text-align: center;"><span class="math inline">\(1.0\)</span></td>
</tr>
<tr class="even">
<td style="text-align: center;">fitness <span class="math inline">\(W_{ii}\)</span></td>
<td style="text-align: center;"><span class="math inline">\(W_{11}\)</span></td>
<td style="text-align: center;"><span class="math inline">\(W_{12}\)</span></td>
<td style="text-align: center;"><span class="math inline">\(W_{22}\)</span></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td style="text-align: center;">freq after selection</td>
<td style="text-align: center;"><span class="math inline">\(p^2W_{11}\)</span></td>
<td style="text-align: center;"><span class="math inline">\(2pqW_{12}\)</span></td>
<td style="text-align: center;"><span class="math inline">\(q^2W_{22}\)</span></td>
<td style="text-align: center;"><span class="math inline">\(\overline{W}\)</span></td>
</tr>
<tr class="even">
<td style="text-align: center;">relative freq</td>
<td style="text-align: center;"><span class="math inline">\(\frac{p^2W_{11}}{\overline{W}}\)</span></td>
<td style="text-align: center;"><span class="math inline">\(\frac{2pqW_{12}}{\overline{W}}\)</span></td>
<td style="text-align: center;"><span class="math inline">\(\frac{q^2W_{22}}{\overline{W}}\)</span></td>
<td style="text-align: center;"><span class="math inline">\(1.0\)</span></td>
</tr>
</tbody>
</table>
</div>
</figure>
</div>
<p>Note that “freq <span class="math inline">\(f_{ii}\)</span>” are the HWequil frequencies for each genotype, the genotype specific fitness values are in the row labeled “fitness <span class="math inline">\(W_{ii}\)</span>”. To calculate the genotype frequencies after selection, we would simply multiply the frequency of the genotype by the genotype specific fitness coefficient, as indicated in row labeled “freq after selection”. If we sum up these genotype frequencies after selection, the sum does not equal 1.0 of course since we are selectively removing some indiduals of a specific genotype (there for sum would be <span class="math inline">\(&lt;1.0\)</span>). However, if you have had a statistics course, you might notice that sum of the genotype frequencies after selection <span class="math inline">\(p^2W_{11}+2pqW_{12}+q^2W_{22}\)</span> is how we can calculate an average, so we define:</p>
<p><span id="eq-meanpopfit"><span class="math display">\[
p^2W_{11}+2pqW_{12}+q^2W_{22}=\overline{W}
\tag{10.1}\]</span></span></p>
<p>Since these are relative fitness’, we want the sum of the frequencies after selection to equal <span class="math inline">\(1.0\)</span>. To do this, we simply divide each gentype frequency after selection by the mean population fitness <span class="math inline">\(\overline{W}\)</span>, as indicated on the last row of the table labeled “relative freq”:</p>
<p><span id="eq-geneqn1"><span class="math display">\[
\frac{p^2W_{11}}{\overline{W}} + \frac{2pqW_{12}}{\overline{W}} + \frac{q^2W_{22}}{\overline{W}} = \frac{p^2W_{11}+2pqW_{12}+q^2W_{22}}{\overline{W}}= 1.0
\tag{10.2}\]</span></span></p>
<div class="callout callout-style-default callout-tip callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Tip
</div>
</div>
<div class="callout-body-container callout-body">
<p>The table above serve as our entry point to explore various models of selection, so take a few minutes to look over <a href="#tbl-selectmod" class="quarto-xref">Table&nbsp;<span>10.1</span></a> and make sure you understand what the table is showing.</p>
</div>
</div>
</section>
<section id="deriving-a-generalized-equation-for-selection" class="level3" data-number="10.2.3">
<h3 data-number="10.2.3" class="anchored" data-anchor-id="deriving-a-generalized-equation-for-selection"><span class="header-section-number">10.2.3</span> Deriving a generalized equation for selection</h3>
<p>We now have estimates of the relative genotype frequencies after selection as shown in <a href="#tbl-selectmod" class="quarto-xref">Table&nbsp;<span>10.1</span></a>. However, based on our definition of evolution, what we really want to know are the allele frequencies and whether they have changed or not. Since we have the genotype frequencies, we can easily calculate the allele frequencies using <a href="IntroEvolGenetics.html#eq-2" class="quarto-xref">Equation&nbsp;<span>8.2</span></a> in the introduction to evolution genetics section. Substituting the values using the last row of <a href="#tbl-selectmod" class="quarto-xref">Table&nbsp;<span>10.1</span></a> into <a href="IntroEvolGenetics.html#eq-2" class="quarto-xref">Equation&nbsp;<span>8.2</span></a> and performing a little algebra:</p>
<p><span class="math display">\[
f_{(A)}=f_{AA}+\frac{1}{2}(f_{Aa})
\]</span></p>
<p><span id="eq-geneqn2"><span class="math display">\[
f_{(A)}=p'=\frac{p^2W_{11}}{\overline{W}}+\frac{1}{2}(\frac{2pqW_{12}}{\overline{W}})=\frac{(p^2W_{11}+pqW_{12})}{\overline{W}}
\tag{10.3}\]</span></span></p>
<p>The above equation tells us what the allele frequency for the <span class="math inline">\(A\)</span> allele is in the next generation after selection. This is useful, but again we are interested in the change in allele frequency <span class="math inline">\(\Delta p\)</span>, which is simply <span class="math inline">\(\Delta p = p'-p\)</span>, where <span class="math inline">\(p'\)</span> is from <a href="#eq-geneqn2" class="quarto-xref">Equation&nbsp;<span>10.3</span></a>.</p>
<p><span class="math display">\[
\Delta p = \frac{(p^2W_{11}+pqW_{12})}{\overline{W}}-p
\]</span></p>
<p>After some more algebra which we will not show, the above equation is simplified to:</p>
<p><span id="eq-geneqn3"><span class="math display">\[
\Delta p = \frac{p}{\overline{W}}(pW_{11}+qW_{12} - \overline{W})
\tag{10.4}\]</span></span></p>
<p>There are various other forms of this generalize equation, but I prefer this version (at this point anyway) because it allows us to make the following important observation. As with a lot of seemingly complicated equations, which can seem a little intimidating, the best way to make sense of them is to break them into smaller pieces and ask what happens when one part is larger than another. In this instance, ignore the <span class="math inline">\(\frac{p}{\overline{W}}\)</span> for now. Inside the curly brackets, the term <span class="math inline">\((pW_{11}+qW_{12})\)</span> is a weighted average, in this case the weighted average fitness for allele <span class="math inline">\(A\)</span>. The other term in the brackets is <span class="math inline">\(\overline{W}\)</span>, the mean population fitness. Just focus on these two terms for now.</p>
<p>What happens when the mean fitness of allele <span class="math inline">\(A\)</span> is greater than the mean population fitness <span class="math inline">\(\overline{W}\)</span>? When <span class="math inline">\((pW_{11}+qW_{12}) &gt; \overline{W}\)</span>, then <span class="math inline">\(\Delta p\)</span> will be positive: the frequency of allele <span class="math inline">\(A\)</span> will increase in the population.</p>
<p>When the mean fitness of allele <span class="math inline">\(A\)</span> is less than the mean population fitness <span class="math inline">\(\overline{W}\)</span>, then <span class="math inline">\((pW_{11}+qW_{12}) &lt; \overline{W}\)</span> and <span class="math inline">\(\Delta p\)</span> will be negative: the frequency of allele <span class="math inline">\(A\)</span> will decrease in the population.</p>
<p>This is an important insight. What this is telling us is that in order for an allele to be favored and increase in frequency in a population, it just needs to be better <em>on average</em> than the mean population fitness. In other words, rewording this to reflect beyond genes and alleles, <em>you do not need to be biggest, badest. or best out there, just just need to be better than average to be favored by selection</em>. This is a profoundy different way to think about selection and fitness. It is relative to those around you. We mentioned this earlier in the chapter on the nature of natural selection, and here we have mathematical proof.</p>
</section>
<section id="a-diversity-of-selection-models" class="level3" data-number="10.2.4">
<h3 data-number="10.2.4" class="anchored" data-anchor-id="a-diversity-of-selection-models"><span class="header-section-number">10.2.4</span> A diversity of selection models</h3>
<p>In terms of our discussion on selection, <a href="#eq-geneqn3" class="quarto-xref">Equation&nbsp;<span>10.4</span></a> is very important in helping to explore how selection works when taking into account the relationship between the two alleles and their impact on the phenotype (fitness in this case). What we mean by this is, recalling your basic genetics from intro biology or BIO211 genetics, in a diploid organism with two alleles, there are two homozygous genotypes (<span class="math inline">\(A_1A_1\)</span> and <span class="math inline">\(A_2A_2\)</span>) and one heterozygous genotype <span class="math inline">\(A_1A_2\)</span>. In the heterozygous genotypes, the two different alleles can interact in such a way as to impact the phenotype (fitness) of the heterozygotes. You saw this with classic Mendelian genetics where Mendel used traits (like pea color) where one allele dominated over the other in the heterozygotes such that the heterozygotes could not be differentiated from one of the homozygotes. This is called a dominance-recessive relationship among alleles, and though not all alleles at different genes are either dominant or recessive (a mistake many students unfortunately make), many are.</p>
<p>It is essential that we understand how dominance/recessive, or a lack thereof, impacts the heterozygotes since according to <a href="#eq-geneqn3" class="quarto-xref">Equation&nbsp;<span>10.4</span></a>, that will impact what happens to the frequency of alleles in the next generation. When we talk about a diversity of selection models, this is one aspect of what we are talking about. We will be exploring what happens in the case of complete dominance, incomplete dominance, co-dominance, heterozygous advantage, and heterozygous disadvantage since each of these models will have important consequences to how selection works and the impact it has on changing allele frequencies over time.</p>
<div id="imp-domrecexpl" class="callout callout-style-default callout-important callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-4-contents" aria-controls="callout-4" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Important&nbsp;10.1: What is dominance and recessive mean?
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-4" class="callout-4-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Before moving on to the different models, I have found that gaining a more nuanced understanding of dominance/recessive and the impacts on fitness of the heterozygotes really helps to appreciate the distinction among these models. So, before diving into the models, what is dominance; how do we define it?</p>
<p><strong>We formally define dominance/recessive based on the impact on the phenotype (fitness here) in the heterozygote genotype</strong>. On a scale of phenotypic effects, if the heterozygote is half way between the two homozygotes, then there is no dominance/recessive relationship between the alleles. This is classically referred to as codominance, but I really dislike this term. Since neither allele dominates in the heterozygote expression of the phenotype, it is incorrect to even use the term in this situation.</p>
<p>If however the heterozygote expresses the phenotype somewhere between the two homozygote genotypes other than half way, then we have a dominance/recessive relationship between the alleles. If one allele is dominant, than the other must be recessive. Allele impacts on the genotype can be either <strong>advantageous</strong>, meaning they tend to increase fitness, or <strong>deleterious</strong>, meaning they have a negative impact on fitess. Complete dominance is when the heterozygote phenotype is exactly the same as one of the homozygotes. This was case with many of the characters Mendel worked with, and is fairly common in genomes (as best as we can tell).</p>
<p>If we assume additive effects of the allele interactions, then we can easily model the different patterns of dominance/recessive interactions by introducing a term designated <span class="math inline">\(h\)</span> into our fitness coefficients. I am not sure what to call this term, but it is easy to remember: <span class="math inline">\(h\)</span> implies impact on heterozygotes. <span class="math inline">\(h\)</span> can range from <span class="math inline">\(0\longrightarrow1.0\)</span>. Recalling from above that our fitness coefficients are related to selection by <span class="math inline">\(W_{ii}=1-s\)</span>, we can then modify this relationship to:</p>
<p><span class="math inline">\(W_{12}=1-hs\)</span> with <span class="math inline">\(h\)</span> ranging from <span class="math inline">\(0\longrightarrow1.0\)</span></p>
<p>This simple addition allows us to model all the possible dominance/recessive - advantageous/deleterious relationships of alleles in the heterozygotes.</p>
<table class="caption-top table">
<thead>
<tr class="header">
<th style="text-align: center;"><span class="math inline">\(h\)</span></th>
<th style="text-align: center;">relationship of alleles</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;"><span class="math inline">\(0\)</span></td>
<td style="text-align: center;">complete dominance advantageous allele</td>
</tr>
<tr class="even">
<td style="text-align: center;"><span class="math inline">\(0&lt;h&lt;0.5\)</span></td>
<td style="text-align: center;">incomplete dominance advantageous allele</td>
</tr>
<tr class="odd">
<td style="text-align: center;"><span class="math inline">\(0.5\)</span></td>
<td style="text-align: center;">codominance (neither allele is dominant or recessive)</td>
</tr>
<tr class="even">
<td style="text-align: center;"><span class="math inline">\(0.5&lt;h&lt;1.0\)</span></td>
<td style="text-align: center;">incomplete dominance deleterious allele</td>
</tr>
<tr class="odd">
<td style="text-align: center;"><span class="math inline">\(1\)</span></td>
<td style="text-align: center;">complete dominance deleterious allele</td>
</tr>
</tbody>
</table>
<p>Graphically:</p>
<div id="fig-domrec" class="quarto-layout-panel">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-domrec-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="quarto-layout-row">
<div class="quarto-layout-cell" style="flex-basis: 100.0%;justify-content: flex-start;">
<img src="images/delrecdesc.png" class="img-fluid figure-img">
</div>
</div>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-domrec-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;10.1: Graphical view of dominance/recessive relationships among alleles. The x-axis is the phenotypic value of the character, which in our case is fitness. The two homozygous genotypes mark the extremes of the x-axis, with the value of the heterozygote varying depending on the value of <span class="math inline">\(W_{12} = 1-hs\)</span>. Note that for the cases of incomplete dominance, <span class="math inline">\(h\)</span> can have a rage of values.
</figcaption>
</figure>
</div>
<p>There are a couple of reasons for focusing so much on this concept of dominance/recessive. First, in my experience many students get stuck into the idea that all allele relationships are dominant/recessive. I think it is the way genetics is taught early on in their biology education, with a focus on Mendel. True, many of Mendel’s characters were dominate/recessive, but that was either a coincidence or purposeful on his part. And though the evidence suggests that many genes have alleles segregating in a population that have a dominance/recessive relationship (recall the ABO blood group in humans), many also do not (again, ABO). Having a clear understanding of domiance/recessive reltionships and using the language of dominance only when warrated, it will prevent a lot of misunderstanding later on.</p>
<p>The second important reason for focusing on this concept is that the allele interactions in the heterozygote will have a profound impact on how selection acts to change the frequency of alleles in a population over time. These are models we will be reviewing next, so again best to have a solid understanding before diving into the various models.</p>
</div>
</div>
</div>
<p>We are now in a good position to introduce the various models we will be exploring. We develop the equations for each of the various models by defining the genotype specific fitness coefficients for each genotype, referred to as the <strong>fitness array</strong>. The table below lists the various models with their associated fitness arrays for the models we will be exploring.</p>
<table class="stripe table-hover caption-top table">
<caption>Various selection models we will explore in lab. Note that the relative genotype specific fitness values are specified as selection coefficients (<span class="math inline">\(s\)</span>), recalling <span class="math inline">\(W_{ii} = 1-s\)</span>.</caption>
<colgroup>
<col style="width: 12%">
<col style="width: 12%">
<col style="width: 12%">
<col style="width: 12%">
<col style="width: 49%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">Genotype</th>
<th style="text-align: center;"><span class="math inline">\(A_1A_1\)</span></th>
<th style="text-align: center;"><span class="math inline">\(A_1A_2\)</span></th>
<th style="text-align: center;"><span class="math inline">\(A_2A_2\)</span></th>
<th style="text-align: center;">Model</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;"></td>
<td style="text-align: center;"><span class="math inline">\(1\)</span></td>
<td style="text-align: center;"><span class="math inline">\(1\)</span></td>
<td style="text-align: center;"><span class="math inline">\(1-s\)</span></td>
<td style="text-align: center;">Pure Dominance: advantageous dominant</td>
</tr>
<tr class="even">
<td style="text-align: center;"></td>
<td style="text-align: center;"><span class="math inline">\(1\)</span></td>
<td style="text-align: center;"><span class="math inline">\(1-s\)</span></td>
<td style="text-align: center;"><span class="math inline">\(1-s\)</span></td>
<td style="text-align: center;">Pure Dominance: deleterious dominant</td>
</tr>
<tr class="odd">
<td style="text-align: center;"></td>
<td style="text-align: center;"><span class="math inline">\(1\)</span></td>
<td style="text-align: center;"><span class="math inline">\(1-hs\)</span></td>
<td style="text-align: center;"><span class="math inline">\(1-s\)</span></td>
<td style="text-align: center;">Incomplete dominance (<span class="math inline">\(0 &lt; h &lt; 1\)</span>)</td>
</tr>
<tr class="even">
<td style="text-align: center;"></td>
<td style="text-align: center;"><span class="math inline">\(1\)</span></td>
<td style="text-align: center;"><span class="math inline">\(1-0.5s\)</span></td>
<td style="text-align: center;"><span class="math inline">\(1-s\)</span></td>
<td style="text-align: center;">Codominance (<span class="math inline">\(h=0.5\)</span>)</td>
</tr>
<tr class="odd">
<td style="text-align: center;"></td>
<td style="text-align: center;"><span class="math inline">\(1-s\)</span></td>
<td style="text-align: center;"><span class="math inline">\(1\)</span></td>
<td style="text-align: center;"><span class="math inline">\(1-t\)</span></td>
<td style="text-align: center;">Heterozygote advantage</td>
</tr>
<tr class="even">
<td style="text-align: center;"></td>
<td style="text-align: center;"><span class="math inline">\(1\)</span></td>
<td style="text-align: center;"><span class="math inline">\(1-s\)</span></td>
<td style="text-align: center;"><span class="math inline">\(1\)</span></td>
<td style="text-align: center;">Heterozygote disadvantage</td>
</tr>
</tbody>
</table>
<p>By replacing the fitness coefficients with the selection coefficients, we can then plug the selection values into <a href="#eq-geneqn3" class="quarto-xref">Equation&nbsp;<span>10.4</span></a>, do a lot of math, and eventually derive an equation that defines how the frequency of alleles will change over time given a model of selection. We will not go over any of the derivations, though we certainly can explore these if you are interested. Rather, our approach will be to focus on undertanding the models themselves, make casual mention to the final equations for each, but use a simulation program in lab to explore the consequences of each model.</p>
<p>The derived equations for the models above are shown in the following table.</p>
<table class="stripe caption-top table">
<caption>Table of deterministic equations for the various selection models considered.</caption>
<colgroup>
<col style="width: 37%">
<col style="width: 12%">
<col style="width: 12%">
<col style="width: 12%">
<col style="width: 23%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">Model</th>
<th style="text-align: center;"><span class="math inline">\(A_1A_1\)</span> <span class="math inline">\(p^2\)</span> <span class="math inline">\(W_{11}\)</span></th>
<th style="text-align: center;"><span class="math inline">\(A_1A_2\)</span> <span class="math inline">\(2pq\)</span> <span class="math inline">\(W_{12}\)</span></th>
<th style="text-align: center;"><span class="math inline">\(A_2A_2\)</span> <span class="math inline">\(q^2\)</span> <span class="math inline">\(W_{22}\)</span></th>
<th style="text-align: center;">Deterministic equations</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">Advantageous dominant deletereous recessive <span class="math inline">\((W_{11}=W_{12}&gt;W_{22})\)</span></td>
<td style="text-align: center;"><span class="math inline">\(1\)</span></td>
<td style="text-align: center;"><span class="math inline">\(1\)</span></td>
<td style="text-align: center;"><span class="math inline">\(1-s\)</span></td>
<td style="text-align: center;"><p><span class="math inline">\(\Delta p=\frac{spq^2}{1-sq^2}\)</span></p>
<p><span class="math inline">\(\Delta q=\frac{-spq^2}{1-sq^2}\)</span></p></td>
</tr>
<tr class="even">
<td style="text-align: center;">Additive fitness: advantageous partially dominant <span class="math inline">\((W_{11}&gt;W_{12}&gt;W_{22})\)</span></td>
<td style="text-align: center;"><span class="math inline">\(1\)</span></td>
<td style="text-align: center;"><p><span class="math inline">\(1-hs\)</span></p>
<p><span class="math inline">\(0&lt;h&lt;1\)</span></p></td>
<td style="text-align: center;"><span class="math inline">\(1-s\)</span></td>
<td style="text-align: center;"><span class="math inline">\(\Delta p=\frac{spq}{2(1-sq)}\)</span></td>
</tr>
<tr class="odd">
<td style="text-align: center;">Heterozygote advantage <span class="math inline">\((W_{11}&lt;W_{12}&gt;W_{22})\)</span></td>
<td style="text-align: center;"><span class="math inline">\(1-s\)</span></td>
<td style="text-align: center;"><span class="math inline">\(1\)</span></td>
<td style="text-align: center;"><span class="math inline">\(1-t\)</span></td>
<td style="text-align: center;"><p><span class="math inline">\(\Delta p=\frac{pq(-sp+tq)}{(1-sq^2-tq^2)}\)</span></p>
<p><span class="math inline">\(\hat{p}=\frac{t}{(s+t)}\)</span></p></td>
</tr>
<tr class="even">
<td style="text-align: center;">Heterozygote disadvantage <span class="math inline">\((W_{11}&gt;W_{12}&lt;W_{22})\)</span></td>
<td style="text-align: center;"><p><span class="math inline">\(1\)</span></p>
<p><span class="math inline">\(1+s\)</span></p></td>
<td style="text-align: center;"><p><span class="math inline">\(1-s\)</span></p>
<p><span class="math inline">\(1\)</span></p></td>
<td style="text-align: center;"><p><span class="math inline">\(1\)</span></p>
<p><span class="math inline">\(1+t\)</span></p></td>
<td style="text-align: center;"><span class="math inline">\(\Delta p=\frac{pq(sp-tq)}{(1+sq^2+tq^2)}\)</span> |</td>
</tr>
</tbody>
</table>
</section>
</section>
<section id="next-step" class="level2" data-number="10.3">
<h2 data-number="10.3" class="anchored" data-anchor-id="next-step"><span class="header-section-number">10.3</span> Next Step</h2>
<p>Using the information from the above section, in lab we will explore how the various selection models described above impact the change in allele frequencies over time. The easiest way to do this is using a simulation program that allows us to specify the various fitness arrays, the starting allele frequencies, and the number of generations to run the simulation. We will explore these models by breaking into small groups, running a series of simulations by altering the model parameters to reflect the model of interest, and at the end of lab collate our results and discuss what the important outcoe and consequences of the models say in terms of selection works.</p>
</section>
<section id="model-results" class="level2" data-number="10.4">
<h2 data-number="10.4" class="anchored" data-anchor-id="model-results"><span class="header-section-number">10.4</span> Model Results</h2>
<p>Below we will step through the results you should have seen in lab when running the simulations for the various models. In each instance we will define the model parameters, provide an example plot of the change in allele frequecies over time that hopefully is similar to what you probably saw, and end with a brief overview on the insights that model provides on how evolution works.</p>
<p>All models of selection required that you input a starting allele frequency (<span class="math inline">\(p_0\)</span>) for the initial generation, the number of generations (time) you want the model to run for, and—perhaps most importantly—the fitness array for the three different genotypes. For simplicity, all of our models assume a diploid organism and a single gene with two alleles, which will be designated either <span class="math inline">\(A_1\)</span> and <span class="math inline">\(A_2\)</span>, or <span class="math inline">\(A\)</span> and <span class="math inline">\(a\)</span>, depending on the simulation app we use.</p>
<p>Since selection is blind to the genotype and can only act on phenotypic traits, it is important to consider different modes of inheritance when studying the possible outcomes of selection. While it is often assumed that most traits exhibit a dominant-recessive inheritance, this is not actually true. Hence, we will consider a number of scenarios beyond dominant-recessive inheritance to explore how selection actually impacts evolutionary outcomes.</p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-5-contents" aria-controls="callout-5" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Advantageous dominant/deleterious recessive (pure dominance)
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-5" class="callout-5-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>In this model we are considering what happens in a situation where there is pure dominance, in this case for an advantageous allele <span class="math inline">\(A_1\)</span>.</p>
<blockquote class="blockquote">
<p><strong>Model settings</strong></p>
<ul>
<li><strong>Population Size:</strong> Infinite</li>
<li><strong>Generations:</strong> 500</li>
<li><strong>Starting Frequency of Allele</strong> <span class="math inline">\(A_1\)</span>: 0.01</li>
<li><strong>Fitness array:</strong> <span class="math inline">\(W_{11} = W_{12} &gt; W_{22}\)</span></li>
</ul>
<p><strong>Example fitness values:</strong></p>
<ul>
<li>Fitness Coefficient of <span class="math inline">\(A_1A_1\)</span>: 1.0</li>
<li>Fitness Coefficient of <span class="math inline">\(A_1A_2\)</span>: 1.0</li>
<li>Fitness Coefficient of <span class="math inline">\(A_2A_A2\)</span>: 0.95</li>
</ul>
</blockquote>
<section id="example-plot-of-change-in-allele-frequencies-over-time." class="level4" data-number="10.4.0.1">
<h4 data-number="10.4.0.1" class="anchored" data-anchor-id="example-plot-of-change-in-allele-frequencies-over-time."><span class="header-section-number">10.4.0.1</span> Example plot of change in allele frequencies over time.</h4>
<div class="cell">
<div class="cell-output-display">
<div id="fig-advdom" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-advdom-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="MechEvolSelection_files/figure-html/fig-advdom-1.png" class="img-fluid figure-img" width="672">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-advdom-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;10.2: Change in the frequency of the favored allele A1 when A1 is advantegous dominant allele, making the A2 allele deleterious recessive. In this instance the starting allele frequency for A1 is 0.01.
</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="outcome" class="level4" data-number="10.4.0.2">
<h4 data-number="10.4.0.2" class="anchored" data-anchor-id="outcome"><span class="header-section-number">10.4.0.2</span> Outcome:</h4>
<ul>
<li>selection will act to increase the frequency of the favored dominant allele (decrease the deleterious recessive)</li>
<li>the advantagous dominant allele will increase in frequency inthe popiulation approaching fixation (<span class="math inline">\(p=1.0\)</span>), however, it will never reach fixation</li>
<li>the rate of increase will depend on the level of selection against the deleterious recessive allele; the higher the selection against the deleterious recessive, the faster the advantageous allele will approach <span class="math inline">\(p=1.0\)</span>, however <span class="math inline">\(p\)</span> will assymptote out and the favored allele will never reach <span class="math inline">\(p=1.0\)</span>.</li>
<li>the deleterious recessive allele will remain in the population, even though at a very low frequency</li>
<li>even if at very low frequencies, sometimes the deleterious homozygous genotypes will be formed in a population, which will be acted on by selection</li>
<li>even for a lethal deleterious allele (<span class="math inline">\(s=1.0\)</span>), selection can not remove it from the population</li>
<li>the deleterious allele effectively “hides” in the heterozygote such that selection can not act against it</li>
</ul>
</section>
<section id="evolutionary-implications" class="level4" data-number="10.4.0.3">
<h4 data-number="10.4.0.3" class="anchored" data-anchor-id="evolutionary-implications"><span class="header-section-number">10.4.0.3</span> Evolutionary implications</h4>
<ul>
<li>deleterious recessive alleles can not be removed from natural populations</li>
<li>natural populations may harbor a large number of deleterious recessive alleles</li>
<li>every once in a while, homozygotes for deleterious recessive allele will result from random matings in the population</li>
<li>this may explain the prevalence of many rare congenital diseases in human populations; they are the result of deleterious alleles hiding in heterozygous individuals that get expressed in situation when two hoterozygous parents have children.</li>
</ul>
</section>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-6-contents" aria-controls="callout-6" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Advantageous recessive/deleterious dominant (pure dominance)
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-6" class="callout-6-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>In this model we are considering what happens in a situation where there is pure dominance, except in this case the advantageous allele <span class="math inline">\(A_1\)</span> is <strong><em>recessive</em></strong>. In other words, we have a deleterious dominant allele that can mask the positive effects of the advantageous allele in the heterozygotes. How common this type of allelic relationship is in natural population is unknown. However, there is some evidence that it occurs. For example individuals that have the CCR5- <span class="math inline">\(\Delta32\)</span> allele in the homozygous state tend to be resistant to infection from HIV and not progress to AIDS. There is some evidence, though limited, that heterozygous individuals (CCR5/CCR5- <span class="math inline">\(\Delta32\)</span>) are not resistant to HIV infection, and may have similar survival profiles as homozygous individuals for the regular CCR5 allele.</p>
<blockquote class="blockquote">
<p><strong>Model settings</strong></p>
<ul>
<li><strong>Population Size:</strong> Infinite</li>
<li><strong>Generations:</strong> 500</li>
<li><strong>Starting Frequency of Allele</strong> <span class="math inline">\(A_1\)</span>: 0.01</li>
<li><strong>Fitness array:</strong> <span class="math inline">\(W_{11} &gt; W_{12} = W_{22}\)</span></li>
</ul>
<p><strong>Example fitness values:</strong></p>
<ul>
<li>Fitness Coefficient of <span class="math inline">\(A_1A_1\)</span>: 1.0</li>
<li>Fitness Coefficient of <span class="math inline">\(A_1A_2\)</span>: 0.9</li>
<li>Fitness Coefficient of <span class="math inline">\(A_2A_A2\)</span>: 0.90</li>
</ul>
</blockquote>
<section id="example-plot-of-change-in-allele-frequencies-over-time.-1" class="level4" data-number="10.4.0.4">
<h4 data-number="10.4.0.4" class="anchored" data-anchor-id="example-plot-of-change-in-allele-frequencies-over-time.-1"><span class="header-section-number">10.4.0.4</span> Example plot of change in allele frequencies over time.</h4>
<div class="cell">
<div class="cell-output-display">
<div id="fig-advrec" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-advrec-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="MechEvolSelection_files/figure-html/fig-advrec-1.png" class="img-fluid figure-img" width="672">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-advrec-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;10.3: Change in the frequency of the favored allele A1 when A1 is an advantegous recessive allele, making the A2 allele deleterious dominant. In this instance the starting allele frequency for A1 is 0.05.
</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="outcome-1" class="level4" data-number="10.4.0.5">
<h4 data-number="10.4.0.5" class="anchored" data-anchor-id="outcome-1"><span class="header-section-number">10.4.0.5</span> Outcome:</h4>
<ul>
<li>selection will act to increase the frequency of the favored recessive allele</li>
<li>at first <span class="math inline">\(A1\)</span> increases in frequency slowly; however, eventually it reaches a frequency in the population where a sufficnet number of homozygote <span class="math inline">\(A_1A_1\)</span> genotypes are present to cause selection to really work on the population, leading an exponential increase in the frequency of the allele in the population</li>
<li>the advantageous recessive allele will continue to increase in frequency in the popiulation, eventually becoming fixed (<span class="math inline">\(p=1.0\)</span>)</li>
<li>the rate of increase will depend on the level of selection favoring the advantageous recessive allele and its frequency in the population; the stronger selection is favoring the advantageous recessive, the faster it will approach <span class="math inline">\(p=1.0\)</span></li>
</ul>
</section>
<section id="evolutionary-implications-1" class="level4" data-number="10.4.0.6">
<h4 data-number="10.4.0.6" class="anchored" data-anchor-id="evolutionary-implications-1"><span class="header-section-number">10.4.0.6</span> Evolutionary implications</h4>
<ul>
<li>even though the advantageous recessive allele is masked in the heterozygote state, eventually selection will cause it to increase in frequency in the population and eventually become fixed</li>
</ul>
</section>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-7-contents" aria-controls="callout-7" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Codominance
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-7" class="callout-7-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>In this model we are considering what happens in a situation where there is no dominance/recessive relationship between the alleles. In this particular case, the heterozygote fitness is somewhere between the two homozygotes. As discussed above (<a href="#imp-domrecexpl" class="quarto-xref">Important&nbsp;<span>10.1</span></a>), in this instance we would not consider either allele dominant or recessive, yet historically these models are referred to as codominance (where the heterozygote is half way between the two homozygotes) or incomplete dominance (where the heterozygote is not in the middle, but is closer to one homozygote or another). These two models are actually closely related to each, and can be referred to as a general class of <strong>additive models</strong>. Let’s just focus on the codominant model here for brevity’s sake, and then make reference to the results from other models later.</p>
<blockquote class="blockquote">
<p><strong>Model settings</strong></p>
<ul>
<li><strong>Population Size:</strong> Infinite</li>
<li><strong>Generations:</strong> 500</li>
<li><strong>Starting Frequency of Allele</strong> <span class="math inline">\(A_1\)</span>: 0.05</li>
<li><strong>Fitness array:</strong> <span class="math inline">\(W_{11} &gt; W_{12} &gt; W_{22}\)</span></li>
</ul>
<p><strong>Example fitness values:</strong></p>
<ul>
<li>Fitness Coefficient of <span class="math inline">\(A_1A_1\)</span>: 1.0</li>
<li>Fitness Coefficient of <span class="math inline">\(A_1A_2\)</span>: 0.95</li>
<li>Fitness Coefficient of <span class="math inline">\(A_2A_A2\)</span>: 0.90 where s=0.1; (we will vary this for a couple of different simulations)</li>
</ul>
</blockquote>
<section id="example-plot-of-change-in-allele-frequencies-over-time.-2" class="level4" data-number="10.4.0.7">
<h4 data-number="10.4.0.7" class="anchored" data-anchor-id="example-plot-of-change-in-allele-frequencies-over-time.-2"><span class="header-section-number">10.4.0.7</span> Example plot of change in allele frequencies over time.</h4>
<p>In the following plot, we are actually plotting the results of the model described above where <span class="math inline">\(s=0.1\)</span>, as well as three others: <span class="math inline">\(s=0.02\)</span>, <span class="math inline">\(s=0.05\)</span>, and <span class="math inline">\(s=0.20\)</span>. This range of values of <span class="math inline">\(s\)</span> are plotted to make a point about the effectiveness of selection in changing allele frequencies at a range of values, from relatively low and relatively high. In fact, these are all pretty high values of <span class="math inline">\(s\)</span> generally speaking. Though it is difficult to determine exact selection coefficients, generally they seem to be around much lower than these values, perhaps around <span class="math inline">\(s&lt;0.01\)</span> in most instances. A selection coeficient of <span class="math inline">\(s&gt;0.01\)</span> would be considered pretty intense selection.</p>
<div class="cell">
<div class="cell-output-display">
<div id="fig-codom" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-codom-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="MechEvolSelection_files/figure-html/fig-codom-1.png" class="img-fluid figure-img" width="672">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-codom-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;10.4: Change in the frequency of the favored allele A1 when A1 and A2 are codominant. In this instance showing the results for a range of selection coefficients - 0.02, 0.05, 0.1, 0.2.
</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="outcome-2" class="level4" data-number="10.4.0.8">
<h4 data-number="10.4.0.8" class="anchored" data-anchor-id="outcome-2"><span class="header-section-number">10.4.0.8</span> Outcome:</h4>
<ul>
<li>selection will act to increase the frequency of the favored recessive allele</li>
<li>depending on the intensity of selection, the frequency of the favored allele <span class="math inline">\(A1\)</span> will increase relatively rapidly and eventually reach fixation (<span class="math inline">\(p=1.0\)</span>)</li>
<li>the rate in which the favored allele goes to fixation depends on the intensity of selection; however, even if selection is relatively weak, the favored will still go to fixation; it just might take a while to get there</li>
</ul>
</section>
<section id="evolutionary-implications-2" class="level4" data-number="10.4.0.9">
<h4 data-number="10.4.0.9" class="anchored" data-anchor-id="evolutionary-implications-2"><span class="header-section-number">10.4.0.9</span> Evolutionary implications</h4>
<ul>
<li>in codominant and incomplete dominance, the favored allele will always go to fixation, at a rate dependent on the intensity of selection</li>
<li>even very weak selection will cause the frequency of the favored allele to become fixed over time</li>
<li>the important lesson from this model is that any minor change in a character the increases the fitness of an organism will be acted on by selection, driving it to fixation in a population.</li>
<li>this explains why we so such exquisite adaptations in natural populations, looking as if they were designed to serve a specific purpose; all it takes is a little advantage, and selection will grab onto that variant and drive it to fixation</li>
<li>deleterious alleles will be purged from a populations, referred to as <strong><em>purefying selection</em></strong></li>
</ul>
</section>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-8-contents" aria-controls="callout-8" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Heterozygous advantage
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-8" class="callout-8-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>In the last couple of models we considered cases of when one allele was favored over the other, with the heterozygous genotype either having a fitness somewhere in between the two homozygous geneotypes (incomplete dominance, codominanace, or additive effects), or had the same fitness as one of the homozygous genotypes (dominance). But what happens when the heterozygous genotype has a higher fitness than either homozygous genotype? We refer to this type of model as <strong>heterozygous advantage</strong>, and the type of selection is referred to as <strong>balancing selection</strong> or <strong>overdominance</strong>.</p>
<p>How common is balancing selection? Again, we are not entirely sure, but studies and theory suggest it is pretty common. In fact, at one point in the history of evolutionary thinking, whether balancing selection was common or rare was central in the Neutralist-Selection debate which raged from the mid-1960s to the 1990s, and still continues in some fashion today.</p>
<p>To explore how heterozygous advantage works in theory, or model parameters should be as follows:</p>
<blockquote class="blockquote">
<p><strong>Model settings</strong></p>
<ul>
<li><strong>Population Size:</strong> Infinite</li>
<li><strong>Generations:</strong> 500</li>
<li><strong>Starting Frequency of Allele</strong> <span class="math inline">\(A_1\)</span>: 0.05 (we will vary this for a couple of different simulations)</li>
<li><strong>Fitness array:</strong> <span class="math inline">\(W_{11} &gt; W_{12} &lt; W_{22}\)</span></li>
</ul>
<p><strong>Example fitness values:</strong></p>
<ul>
<li>Fitness Coefficient of <span class="math inline">\(A_1A_1\)</span>: 0.9</li>
<li>Fitness Coefficient of <span class="math inline">\(A_1A_2\)</span>: 1.0</li>
<li>Fitness Coefficient of <span class="math inline">\(A_2A_A2\)</span>: 0.8 where s varies by homozygous genotype; in this instance for <span class="math inline">\(A_1A_1\)</span>, <span class="math inline">\(s=0.1\)</span>; for <span class="math inline">\(A_2A_2\)</span>, <span class="math inline">\(t=0.2\)</span>; this is pretty intense selection.</li>
</ul>
</blockquote>
<section id="example-plot-of-change-in-allele-frequencies-over-time.-3" class="level4" data-number="10.4.0.10">
<h4 data-number="10.4.0.10" class="anchored" data-anchor-id="example-plot-of-change-in-allele-frequencies-over-time.-3"><span class="header-section-number">10.4.0.10</span> Example plot of change in allele frequencies over time.</h4>
<p>In the plot below, we actually ran four simulations that varied by starting frequency of allele <span class="math inline">\(A_1\)</span>: <span class="math inline">\(0.05\)</span>, <span class="math inline">\(0.4\)</span>, <span class="math inline">\(0.8\)</span>, and <span class="math inline">\(0.95\)</span>. We did this accentuate the results, which will be described below.</p>
<div class="cell">
<div class="cell-output-display">
<div id="fig-hetadv" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-hetadv-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="MechEvolSelection_files/figure-html/fig-hetadv-1.png" class="img-fluid figure-img" width="672">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-hetadv-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;10.5: Change in the frequency of allele A1 for heterozygous advantage, where the heterozygous genotype has the highest fitness. In this instance we are plotting a range of starting allele frequencies for A1, 0.05, 0.4, 0.8, and 0.95.
</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="outcome-3" class="level4" data-number="10.4.0.11">
<h4 data-number="10.4.0.11" class="anchored" data-anchor-id="outcome-3"><span class="header-section-number">10.4.0.11</span> Outcome:</h4>
<ul>
<li>frequency of the allele <span class="math inline">\(A1\)</span> will converge on some value regardless of the starting allele frequency; we refer to this frequency as an <strong>equilibrium frequency</strong>, designated as <span class="math inline">\(\hat{p}\)</span>.</li>
<li>the rate in which the starting allele frequency goes to <span class="math inline">\(\hat{p}\)</span> depends on the intensity of selection against the two homozygous genotypes.</li>
<li>once the <span class="math inline">\(p=\hat{p}\)</span>, the allele frequency will remain at <span class="math inline">\(\hat{p}\)</span> even if the system is purturbed (<em>i.e.</em> for some reason the <span class="math inline">\(p\)</span> is driven off <span class="math inline">\(\hat{p}\)</span>); this is referred to as a stable equiibrium frequency.</li>
<li>we can derive an equation for equibrium frequeny, which is <span class="math inline">\(\hat{p}=\frac{t}{s+t}\)</span></li>
<li>that being said, if <span class="math inline">\(s=t\)</span>, regardless of how large <span class="math inline">\(s\)</span> and <span class="math inline">\(t\)</span> are, <span class="math inline">\(\hat{p}=0.5\)</span></li>
</ul>
</section>
<section id="evolutionary-implications-3" class="level4" data-number="10.4.0.12">
<h4 data-number="10.4.0.12" class="anchored" data-anchor-id="evolutionary-implications-3"><span class="header-section-number">10.4.0.12</span> Evolutionary implications</h4>
<ul>
<li>heterozygous advantage, the frequency of the alleles will reach an equilbrium value determined by the level of selection against he two homozygous genotypes</li>
<li>since the allele frequencies reach an equiblrium, both alleles are maintained in the population; in other words, unlike the above models of selection, heterozygous advantage, also known as balancing selection, works to <strong>maintain variation in the population</strong></li>
<li>though selection is acting on the population, the allele frequencies are not changing, so formally the populations is not evolving according our definition of evolution</li>
<li>we have numerous examples of balancing selection acting in natural populations and believe it to be quite common, though it can be difficult to detect unless carefully tested for.</li>
<li>In human populations, the mutation that causes sickle cell anemia has been shown to ne maintained in human populations due to balancing selection between sick cell anemia and the resistance to the parasite that causes malaria.</li>
</ul>
</section>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-9-contents" aria-controls="callout-9" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Heterozygous disadvantage
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-9" class="callout-9-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Our last model builds of the previous (Heterozygous advantage), only now we ask what happens to the frequency of alleles when the heterozygous genotype has the lowest relative fitness compared to either of homozygous genotypes. This model is referred to as <strong>heterozygous disadvantage</strong>, and is also referred to as <strong>underdominance</strong>.</p>
<p>We beleive that underdominance may play a very important role in sympatric speciation due to ecological divergence or sexual selection, topic we will be covering later in the semester.</p>
<p>To explore how heterozygous disadvantage works in theory, or model parameters will be as follows:</p>
<blockquote class="blockquote">
<p><strong>Model settings</strong></p>
<ul>
<li><strong>Population Size:</strong> Infinite</li>
<li><strong>Generations:</strong> 500</li>
<li><strong>Starting Frequency of Allele</strong> <span class="math inline">\(A_1\)</span>: 0.05 (we will vary this for a couple of different simulations)</li>
<li><strong>Fitness array:</strong> <span class="math inline">\(W_{11} &lt; W_{12} &gt; W_{22}\)</span></li>
</ul>
<p><strong>Example fitness values:</strong></p>
<ul>
<li>Fitness Coefficient of <span class="math inline">\(A_1A_1\)</span>: 1.0</li>
<li>Fitness Coefficient of <span class="math inline">\(A_1A_2\)</span>: 0.8</li>
<li>Fitness Coefficient of <span class="math inline">\(A_2A_A2\)</span>: 1.0</li>
</ul>
</blockquote>
<section id="example-plot-of-change-in-allele-frequencies-over-time.-4" class="level4" data-number="10.4.0.13">
<h4 data-number="10.4.0.13" class="anchored" data-anchor-id="example-plot-of-change-in-allele-frequencies-over-time.-4"><span class="header-section-number">10.4.0.13</span> Example plot of change in allele frequencies over time.</h4>
<p>In the plot below, we actually ran four simulations that varied by starting frequency of allele <span class="math inline">\(A_1\)</span>: <span class="math inline">\(0.05\)</span>, <span class="math inline">\(0.4\)</span>, <span class="math inline">\(0.8\)</span>, and <span class="math inline">\(0.95\)</span>. We did this accentuate the results, which will be described below.</p>
<div class="cell">
<div class="cell-output-display">
<div id="fig-hetdisadv" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-hetdisadv-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="MechEvolSelection_files/figure-html/fig-hetdisadv-1.png" class="img-fluid figure-img" width="672">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-hetdisadv-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;10.6: Change in the frequency of allele A1 for heterozygous disadvantage, where the heterozygous genotype has the lowest fitness. In this instance we are plotting a range of starting allele frequencies for A1, 0.5,0.52, 0.49,0.4999, 0.5000001 0.501
</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="outcome-4" class="level4" data-number="10.4.0.14">
<h4 data-number="10.4.0.14" class="anchored" data-anchor-id="outcome-4"><span class="header-section-number">10.4.0.14</span> Outcome:</h4>
<ul>
<li>given the fitness array, 1.0:0.8:1.0, under heterozygous disadvantage, since the fitness coefficients for the two homozygous genotypes are both 1.0, then there exists an equilibrium frequency that occurs when <span class="math inline">\(p-0.5\)</span>, as seen in the orange curve above.</li>
<li>However, it runs out this is an <strong>unstable equilibrium</strong>, unlike the equilibrium seen in heterozygous advantage. Any purturbation that takes the allele frequency off the equilibrium value will cause the change in allele frequency to drive toward either fixation or loss, depending on if the purturbation changes the allele frequency to be greater or less than the equilibrium frequnecy. We can see that for the simulations where the starting allele frequency are 0.49, which leads to the <span class="math inline">\(A_1\)</span> going to <span class="math inline">\(p=0.0\)</span>, and when the starting allele frequency is <span class="math inline">\(p=0.52\)</span>, where the <span class="math inline">\(p\)</span> then approaches and reaches fixation (<span class="math inline">\(p=1.0\)</span>).</li>
<li>in fact, it takes even the slightest change off the equilibrium frequency to drive the system to either boundary, <em>e.g.</em> when starting <span class="math inline">\(p=0.5000001\)</span>, the <span class="math inline">\(p\)</span> approaches and reaches fixation. Likewise, if the starting <span class="math inline">\(p=0.4999\)</span>, then <span class="math inline">\(p\)</span> approaches and reached loss (<span class="math inline">\(p=0.0\)</span>).</li>
<li>if the fitness array is assymetrical, for example 1.0:0.8:0.9 for the three genotypes, then an unstable equilbrium exists but it will not be at <span class="math inline">\(p=0.5\)</span>. The 0.5 is due to selection acting equally on the two homozygous genotypes.</li>
</ul>
</section>
<section id="evolutionary-implications-4" class="level4" data-number="10.4.0.15">
<h4 data-number="10.4.0.15" class="anchored" data-anchor-id="evolutionary-implications-4"><span class="header-section-number">10.4.0.15</span> Evolutionary implications</h4>
<ul>
<li>under heterozygous disadvantage, an equilibrium allele frequency exists, but it is an unstable equilbrium.</li>
<li>any minor change in allele frequency will drive the system off the equilibrium, causing the allele to either become fixed or lost.</li>
<li>since no population is infinite in size, there is always some measure of drift going on, and so this unstable equilibrium is never reached in natural populations; it is a theoretical equilibrium</li>
<li>more importantly, this model shows us that if a heterozygous genotypes has a lower fitness in a population compared to the two homozygous genotypes, then natural can cause the population to effectively split into two separate groups</li>
<li>underdominance then may be an important mechanism in ecological and sympatric speciation, which we will explore later in the semester when we discuss speciation.</li>
<li>there are examples of underdominace leading to variation within and among popualtions in the literature</li>
</ul>
</section>
</div>
</div>
</div>
<hr>
</section>
<section id="drift-and-selection" class="level2" data-number="10.5">
<h2 data-number="10.5" class="anchored" data-anchor-id="drift-and-selection"><span class="header-section-number">10.5</span> Drift and Selection</h2>
<section id="interactions-between-drift-and-selection" class="level3" data-number="10.5.1">
<h3 data-number="10.5.1" class="anchored" data-anchor-id="interactions-between-drift-and-selection"><span class="header-section-number">10.5.1</span> Interactions Between Drift and Selection</h3>
<p>As already discussed (<a href="MechEvolDrift.html" class="quarto-xref"><span>Chapter 9</span></a>), genetic drift can have a large impact on the change in allele frequencies in small populations. When drift is intense, it can tend to complicate the effects of selection, sometimes leading to unexpected and counter-intuitive outcomes, <em>i.e.</em> genetic drift effects the fate of alleles under selection. Drift can cause deleterious mutations to be more common than expected by selection alone, and it can cause beneficial alleles to disappear from the population.</p>
<p>As an example of how drift can impact the effects of selection, consider the out come of the following model:</p>
<div class="cell">
<div class="cell-output-display">
<div id="fig-advdomdrfit" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-advdomdrfit-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="MechEvolSelection_files/figure-html/fig-advdomdrfit-1.png" class="img-fluid figure-img" width="672">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-advdomdrfit-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;10.7: Change in the frequency of the favored allele A1 when A1 is advantegous dominant allele, making the <em>A2 allele deleterious recessive</em>. In this instance the starting allele frequency for A1 is 0.05, and we introduce moderate levels of genetic drift. The inifinite population baseline is shown in black.
</figcaption>
</figure>
</div>
</div>
</div>
<p>Notice how in <a href="#fig-advdomdrfit" class="quarto-xref">Figure&nbsp;<span>10.7</span></a> the change in the frequency of alleles over time generally follow the plot for the infinite population size model we saw earlier, but with a substantial amoung of variation between generations and simulations. This obviously is due to drift acting on the population the same time as selection increases the frequency of <span class="math inline">\(A_1\)</span> as expected. However, there are some very imporant differences. Fist, in several populations, the favored became fixed rather than reaching an asymptote as expected with an infinite population size (<a href="#fig-advdom" class="quarto-xref">Figure&nbsp;<span>10.2</span></a>). In fact, <span class="math inline">\(A_1\)</span> became fixed <span class="math inline">\(p=1.0\)</span> in 7 of the 12 simulations. When drift is introduced into the advantageous dominant allele (deleterious recessive) model, we actually can remove deleterious recessive alleles from the population. If we were to run this simulation with varying intensities of genetic drift, we would find that the probablility of losing the deleterious recessive is some function of the intensity of drift. That is one difference between what we would expect under an infinite population size model and one that incorporates drift, which has obvious consequences from an evolutionary perspective in terms how many deleterious recessive alleles we should find in a natural population.</p>
<p>But there is an even more interesting result, namely that in one of the simulations, we actually <strong><em>lost</em></strong> the advantageous allele. In other words, given sufficient levels of genetic drift, an advantageous dominant allele can be lost, with the deleterious recessive allele becoming fixed in a population. This is not supposed to happen under selection. Selection always acts to increase population fitness, and yet when drift occurs, you can find a decrease in the mean population fitness.</p>
<p>We could repeat these types of simulations varying the intensity of selection and drift for all the various models discussed above and find the similar results, <strong>that populations can lose advantageous alleles due to drift overcoming the effects of selection</strong> and model outcomes can be different from expectations. For example:</p>
<ul>
<li>rate to fixation or loss may be different from models with infinite population size</li>
<li>can lose deleterious recessives under drift</li>
<li>an advantageous allele may be lost under intense drift</li>
<li>conversely, a disadvantageous allele may become fixed</li>
<li>populations may not attain equilibrium allele frequencies (e.g.&nbsp;heterozygous advantage)</li>
<li>in heterozygote disadvantage, populations will diverge since equilibrium is impossible due to drift</li>
</ul>
<p>How does genetic drift impact the effects of selection? In theory, for selection to be effective, the selection coefficient <span class="math inline">\(s\)</span> must be greater than <span class="math inline">\(\frac{1}{2N_e}\)</span>, or formally <span class="math inline">\(s&gt;\frac{1}{2N_e}\)</span>. When <span class="math inline">\(s\)</span> is less than <span class="math inline">\(\frac{1}{2N_e}\)</span>, than drift can overcome the effects of selection, and we can have unexpected outcomes compared to our theoretical models based on an infinite population size.</p>
<p>Looking at the relationship of drift and selection from a different perspective, generally speaking, the fate of alleles subject to selection and drift is dependent on the product of 2<em>N</em><sub>e</sub><em>s</em>, as depicted in <a href="#fig-driftsel" class="quarto-xref">Figure&nbsp;<span>10.8</span></a>. If a new mutation is neutral (<span class="math inline">\(s\)</span>=0, where here we use the nomenclature for <span class="math inline">\(s\)</span> that a favorable allele has is <span class="math inline">\(s&gt;1\)</span> and a deleterious allele <span class="math inline">\(1&gt;s&gt;0\)</span><a href="#fn1" class="footnote-ref" id="fnref1" role="doc-noteref"><sup>1</sup></a>), the probability of fixation is <span class="math inline">\(\frac{1}{2N_e}\)</span> (<span class="math inline">\(\lambda=1\)</span>; dotted line), as described above. If the new mutation is deleterious (<span class="math inline">\(s&lt;0\)</span>), then the likelihood of fixation becomes smaller than what is expected by chance, approaching zero for higher values of <span class="math inline">\(|2N_es|\)</span> (<a href="#fig-driftsel" class="quarto-xref">Figure&nbsp;<span>10.8</span></a>). In contrast, if the new mutation provides a fitness advantage (<span class="math inline">\(s&gt;1\)</span>), then the likelihood of fixation become greater than what is expected by chance under drift alone. For example, for <span class="math inline">\(2N_es = 5\)</span>, the likelihood of fixation for the new mutation is 5 times higher than what would be expected by chance.</p>
<p>The important point here is that the likelihood of fixation is dependent on both the strength of selection and the effective population size. <span class="math inline">\(2N_es\)</span> can be large when selection is strong or when populations are large. When populations are small, selection on novel alleles needs to be comparatively strong for them to have a high likelihood of fixation. If a mutation only provides a minor fitness benefit, drift might cause its loss from the population before it ever has a chance to become common. Conversely, when population sizes are very large, novel alleles with minute fitness benefits can have a high likelihood of fixation. As a rule of thumb, evolution of novel alleles is primarily governed by genetic drift for <span class="math inline">\(2N_es\)</span>-values between -1 and 1 (gray shaded area in <a href="#fig-driftsel" class="quarto-xref">Figure&nbsp;<span>10.8</span></a>. Beyond that range, selection has the upper hand.</p>
<div class="cell">
<div class="cell-output-display">
<div id="fig-driftsel" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-driftsel-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="MechEvolSelection_files/figure-html/fig-driftsel-1.png" class="img-fluid figure-img" width="672">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-driftsel-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;10.8: The likelihood for fixation of a novel allele increases with increasing values for <em>N</em><sub>e</sub> * <em>s</em> (Charlesworth 2009). The dotted line indicates perfect neutrality; the gray-shaded area corresponds to <em>N</em><sub>e</sub> * <em>s</em> values for which novel mutations evolve largely by genetic drift. Note that the relationship depicted assumes <em>N</em> = <em>N</em><sub>e</sub>.
</figcaption>
</figure>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-10-contents" aria-controls="callout-10" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Drift and selection - the math
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-10" class="callout-10-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>In case you are interested in the math underlying Figure <a href="#fig-driftsel" class="quarto-xref">Figure&nbsp;<span>10.8</span></a>: The probability (<span class="math inline">\(Q\)</span>) that a new mutation spreads in a population and eventually becomes fixed is dependent on the effective population size (<span class="math inline">\(N_e\)</span>), the census population size (<span class="math inline">\(N\)</span>), and the selection coefficient (<span class="math inline">\(s\)</span>). <span class="math inline">\(\lambda\)</span> is the fixation probability relative to neutral evolution (<span class="math inline">\(\frac{1}{2N_e}\)</span>).</p>
<p><span class="math display">\[
\begin{aligned}
Q = \frac{N_es}{N} \frac{1}{1-exp(-2N_es)} \\
\lambda = \frac{Q}{1/(2N_e)}
\end{aligned}
\]</span></p>
<p>Note that this relationship assumes that the fitness of the heterozygotes is intermediate between the two homozygotes:</p>
<p><span class="math display">\[
\begin{aligned}
w_{Aa} = \frac{w_{AA}+w_{aa}}{2}
\end{aligned}
\]</span></p>
</div>
</div>
</div>


</section>
</section>
<section id="footnotes" class="footnotes footnotes-end-of-document" role="doc-endnotes">
<hr>
<ol>
<li id="fn1"><p>This use of <span class="math inline">\(s&gt;0&gt;s\)</span> as a way to distinguish advantageous, neutral, or deleterious alleles seems different from how we used <span class="math inline">\(s\)</span> when setting up the fitness arrays for our selection models above. Actually it is different, but still consistent. For our purposes we specifically defined <span class="math inline">\(s\)</span> as always positive such that <span class="math inline">\(W_{ij}=1-s\)</span> made more intuitive sense. It really is a matter of perspective and convenience for the math that defines the models and how we think about selection. Try not to let this throw you for a loop. We are changing how we view <span class="math inline">\(s\)</span> in this discussion to be consistent with how Tobler uses it in his OER.<a href="#fnref1" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
</ol>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="./MechEvolDrift.html" class="pagination-link" aria-label="Genetic Drift">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">Genetic Drift</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="./MechEvolMutation.html" class="pagination-link" aria-label="Mutation">
        <span class="nav-page-text"><span class="chapter-number">11</span>&nbsp; <span class="chapter-title">Mutation</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->




</body></html>